"use strict";(self.webpackChunknirvana_documentation=self.webpackChunknirvana_documentation||[]).push([[156,3130],{3905:function(t,e,n){n.d(e,{Zo:function(){return u},kt:function(){return m}});var a=n(67294);function r(t,e,n){return e in t?Object.defineProperty(t,e,{value:n,enumerable:!0,configurable:!0,writable:!0}):t[e]=n,t}function o(t,e){var n=Object.keys(t);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(t);e&&(a=a.filter((function(e){return Object.getOwnPropertyDescriptor(t,e).enumerable}))),n.push.apply(n,a)}return n}function i(t){for(var e=1;e<arguments.length;e++){var n=null!=arguments[e]?arguments[e]:{};e%2?o(Object(n),!0).forEach((function(e){r(t,e,n[e])})):Object.getOwnPropertyDescriptors?Object.defineProperties(t,Object.getOwnPropertyDescriptors(n)):o(Object(n)).forEach((function(e){Object.defineProperty(t,e,Object.getOwnPropertyDescriptor(n,e))}))}return t}function l(t,e){if(null==t)return{};var n,a,r=function(t,e){if(null==t)return{};var n,a,r={},o=Object.keys(t);for(a=0;a<o.length;a++)n=o[a],e.indexOf(n)>=0||(r[n]=t[n]);return r}(t,e);if(Object.getOwnPropertySymbols){var o=Object.getOwnPropertySymbols(t);for(a=0;a<o.length;a++)n=o[a],e.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(t,n)&&(r[n]=t[n])}return r}var s=a.createContext({}),d=function(t){var e=a.useContext(s),n=e;return t&&(n="function"==typeof t?t(e):i(i({},e),t)),n},u=function(t){var e=d(t.components);return a.createElement(s.Provider,{value:e},t.children)},c={inlineCode:"code",wrapper:function(t){var e=t.children;return a.createElement(a.Fragment,{},e)}},p=a.forwardRef((function(t,e){var n=t.components,r=t.mdxType,o=t.originalType,s=t.parentName,u=l(t,["components","mdxType","originalType","parentName"]),p=d(n),m=r,v=p["".concat(s,".").concat(m)]||p[m]||c[m]||o;return n?a.createElement(v,i(i({ref:e},u),{},{components:n})):a.createElement(v,i({ref:e},u))}));function m(t,e){var n=arguments,r=e&&e.mdxType;if("string"==typeof t||r){var o=n.length,i=new Array(o);i[0]=p;var l={};for(var s in e)hasOwnProperty.call(e,s)&&(l[s]=e[s]);l.originalType=t,l.mdxType="string"==typeof t?t:r,i[1]=l;for(var d=2;d<o;d++)i[d]=n[d];return a.createElement.apply(null,i)}return a.createElement.apply(null,n)}p.displayName="MDXCreateElement"},22529:function(t,e,n){n.r(e),n.d(e,{contentTitle:function(){return s},default:function(){return p},frontMatter:function(){return l},metadata:function(){return d},toc:function(){return u}});var a=n(87462),r=n(63366),o=(n(67294),n(3905)),i=["components"],l={},s=void 0,d={unversionedId:"data-sources/decipher-json",id:"version-3.18/data-sources/decipher-json",title:"decipher-json",description:"| Field                | Type   | Notes                                                           |",source:"@site/versioned_docs/version-3.18/data-sources/decipher-json.md",sourceDirName:"data-sources",slug:"/data-sources/decipher-json",permalink:"/IlluminaAnnotatorDocumentation/3.18/data-sources/decipher-json",editUrl:"https://github.com/Illumina/IlluminaAnnotatorDocumentation/edit/master/versioned_docs/version-3.18/data-sources/decipher-json.md",tags:[],version:"3.18",frontMatter:{}},u=[],c={toc:u};function p(t){var e=t.components,n=(0,r.Z)(t,i);return(0,o.kt)("wrapper",(0,a.Z)({},c,n,{components:e,mdxType:"MDXLayout"}),(0,o.kt)("pre",null,(0,o.kt)("code",{parentName:"pre",className:"language-json"},'"decipher":[\n  {\n    "chromosome":"1",\n    "begin":13516,\n    "end":91073,\n    "numDeletions":27,\n    "deletionFrequency":0.675,\n    "numDuplications":27,\n    "duplicationFrequency":0.675,\n    "sampleSize":40,\n    "reciprocalOverlap": 0.27555,\n    "annotationOverlap": 0.5901\n  }\n],\n')),(0,o.kt)("table",null,(0,o.kt)("thead",{parentName:"table"},(0,o.kt)("tr",{parentName:"thead"},(0,o.kt)("th",{parentName:"tr",align:null},"Field"),(0,o.kt)("th",{parentName:"tr",align:null},"Type"),(0,o.kt)("th",{parentName:"tr",align:null},"Notes"))),(0,o.kt)("tbody",{parentName:"table"},(0,o.kt)("tr",{parentName:"tbody"},(0,o.kt)("td",{parentName:"tr",align:null},"chromosome"),(0,o.kt)("td",{parentName:"tr",align:null},"int"),(0,o.kt)("td",{parentName:"tr",align:null},"Ensembl-style chromosome names")),(0,o.kt)("tr",{parentName:"tbody"},(0,o.kt)("td",{parentName:"tr",align:null},"begin"),(0,o.kt)("td",{parentName:"tr",align:null},"int"),(0,o.kt)("td",{parentName:"tr",align:null},"1-based position")),(0,o.kt)("tr",{parentName:"tbody"},(0,o.kt)("td",{parentName:"tr",align:null},"end"),(0,o.kt)("td",{parentName:"tr",align:null},"int"),(0,o.kt)("td",{parentName:"tr",align:null},"1-based position")),(0,o.kt)("tr",{parentName:"tbody"},(0,o.kt)("td",{parentName:"tr",align:null},"numDeletions"),(0,o.kt)("td",{parentName:"tr",align:null},"int"),(0,o.kt)("td",{parentName:"tr",align:null},"# of observed deletions")),(0,o.kt)("tr",{parentName:"tbody"},(0,o.kt)("td",{parentName:"tr",align:null},"deletionFrequency"),(0,o.kt)("td",{parentName:"tr",align:null},"float"),(0,o.kt)("td",{parentName:"tr",align:null},"deletion frequency")),(0,o.kt)("tr",{parentName:"tbody"},(0,o.kt)("td",{parentName:"tr",align:null},"numDuplications"),(0,o.kt)("td",{parentName:"tr",align:null},"int"),(0,o.kt)("td",{parentName:"tr",align:null},"# of observed duplications")),(0,o.kt)("tr",{parentName:"tbody"},(0,o.kt)("td",{parentName:"tr",align:null},"duplicationFrequency"),(0,o.kt)("td",{parentName:"tr",align:null},"float"),(0,o.kt)("td",{parentName:"tr",align:null},"duplication frequency")),(0,o.kt)("tr",{parentName:"tbody"},(0,o.kt)("td",{parentName:"tr",align:null},"sampleSize"),(0,o.kt)("td",{parentName:"tr",align:null},"int"),(0,o.kt)("td",{parentName:"tr",align:null},"total # of samples")),(0,o.kt)("tr",{parentName:"tbody"},(0,o.kt)("td",{parentName:"tr",align:null},"reciprocalOverlap"),(0,o.kt)("td",{parentName:"tr",align:null},"float"),(0,o.kt)("td",{parentName:"tr",align:null},"Range: 0 - 1. E.g. 0.57 would indicate a 57% reciprocal overlap")),(0,o.kt)("tr",{parentName:"tbody"},(0,o.kt)("td",{parentName:"tr",align:null},"annotationOverlap"),(0,o.kt)("td",{parentName:"tr",align:null},"float"),(0,o.kt)("td",{parentName:"tr",align:null},"Range: 0 - 1. E.g. 0.57 would indicate a 57% annotation overlap")))))}p.isMDXComponent=!0},57429:function(t,e,n){n.r(e),n.d(e,{contentTitle:function(){return d},default:function(){return m},frontMatter:function(){return s},metadata:function(){return u},toc:function(){return c}});var a=n(87462),r=n(63366),o=(n(67294),n(3905)),i=n(22529),l=["components"],s={title:"DECIPHER"},d=void 0,u={unversionedId:"data-sources/decipher",id:"version-3.18/data-sources/decipher",title:"DECIPHER",description:"Overview",source:"@site/versioned_docs/version-3.18/data-sources/decipher.mdx",sourceDirName:"data-sources",slug:"/data-sources/decipher",permalink:"/IlluminaAnnotatorDocumentation/3.18/data-sources/decipher",editUrl:"https://github.com/Illumina/IlluminaAnnotatorDocumentation/edit/master/versioned_docs/version-3.18/data-sources/decipher.mdx",tags:[],version:"3.18",frontMatter:{title:"DECIPHER"},sidebar:"docs",previous:{title:"dbSNP",permalink:"/IlluminaAnnotatorDocumentation/3.18/data-sources/dbsnp"},next:{title:"FusionCatcher",permalink:"/IlluminaAnnotatorDocumentation/3.18/data-sources/fusioncatcher"}},c=[{value:"Overview",id:"overview",children:[{value:"TSV Extraction",id:"tsv-extraction",children:[{value:"Parsing",id:"parsing",children:[],level:4}],level:3}],level:2},{value:"Download URL",id:"download-url",children:[{value:"JSON output",id:"json-output",children:[],level:3}],level:2}],p={toc:c};function m(t){var e=t.components,n=(0,r.Z)(t,l);return(0,o.kt)("wrapper",(0,a.Z)({},p,n,{components:e,mdxType:"MDXLayout"}),(0,o.kt)("h2",{id:"overview"},"Overview"),(0,o.kt)("p",null,(0,o.kt)("a",{parentName:"p",href:"https://www.deciphergenomics.org/"},"DECIPHER")," (DatabasE of genomiC varIation and Phenotype in Humans using Ensembl Resources) is an interactive web-based database which incorporates a suite of tools designed to aid the interpretation of genomic variants."),(0,o.kt)("p",null,"DECIPHER enhances clinical diagnosis by retrieving information from a variety of bioinformatics resources relevant to the variant found in the patient. The patient's variant is displayed in the context of both normal variation and pathogenic variation reported at that locus thereby facilitating interpretation."),(0,o.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,o.kt)("div",{parentName:"div",className:"admonition-heading"},(0,o.kt)("h5",{parentName:"div"},(0,o.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,o.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,o.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"Publication")),(0,o.kt)("div",{parentName:"div",className:"admonition-content"},(0,o.kt)("p",{parentName:"div"},"DECIPHER: Database of Chromosomal Imbalance and Phenotype in Humans using Ensembl Resources. Firth, H.V. et al., 2009. Am.J.Hum.Genet 84, 524-533 (DOI: dx.doi.org/10/1016/j.ajhg.2009.03.010)"))),(0,o.kt)("h3",{id:"tsv-extraction"},"TSV Extraction"),(0,o.kt)("pre",null,(0,o.kt)("code",{parentName:"pre",className:"language-scss"},"#population_cnv_id  chr start   end deletion_observations   deletion_frequency  deletion_standard_error duplication_observations    duplication_frequency   duplication_standard_error  observations    frequency   standard_error  type    sample_size study\n1   1   10529   177368  0   0   1   3   0.075   0.555277708 3   0.075   0.555277708 1   40  42M calls\n2   1   13516   91073   0   0   1   27  0.675   0.109713431 27  0.675   0.109713431 1   40  42M calls\n3   1   18888   35451   0   0   1   2   0.002366864 0.706269473 2   0.002366864 0.706269473 1   845 DDD\n")),(0,o.kt)("h4",{id:"parsing"},"Parsing"),(0,o.kt)("p",null,"We parse the DECIPHER tsv file and extract the following columns:"),(0,o.kt)("ul",null,(0,o.kt)("li",{parentName:"ul"},"chr"),(0,o.kt)("li",{parentName:"ul"},"start"),(0,o.kt)("li",{parentName:"ul"},"end"),(0,o.kt)("li",{parentName:"ul"},"deletion_observations"),(0,o.kt)("li",{parentName:"ul"},"deletion_frequency"),(0,o.kt)("li",{parentName:"ul"},"duplication_observations"),(0,o.kt)("li",{parentName:"ul"},"duplication_frequency"),(0,o.kt)("li",{parentName:"ul"},"sample_size")),(0,o.kt)("h2",{id:"download-url"},"Download URL"),(0,o.kt)("p",null,(0,o.kt)("a",{parentName:"p",href:"https://www.deciphergenomics.org/files/downloads/population_cnv_grch38.txt.gz"},"https://www.deciphergenomics.org/files/downloads/population_cnv_grch38.txt.gz"),"\n",(0,o.kt)("a",{parentName:"p",href:"https://www.deciphergenomics.org/files/downloads/population_cnv_grch37.txt.gz"},"https://www.deciphergenomics.org/files/downloads/population_cnv_grch37.txt.gz")),(0,o.kt)("h3",{id:"json-output"},"JSON output"),(0,o.kt)(i.default,{mdxType:"JSON"}))}m.isMDXComponent=!0}}]);