"use strict";(self.webpackChunknirvana_documentation=self.webpackChunknirvana_documentation||[]).push([[1932,5337,9151,3425,1790,9383,8660],{3905:function(t,e,n){n.d(e,{Zo:function(){return d},kt:function(){return N}});var a=n(67294);function r(t,e,n){return e in t?Object.defineProperty(t,e,{value:n,enumerable:!0,configurable:!0,writable:!0}):t[e]=n,t}function l(t,e){var n=Object.keys(t);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(t);e&&(a=a.filter((function(e){return Object.getOwnPropertyDescriptor(t,e).enumerable}))),n.push.apply(n,a)}return n}function i(t){for(var e=1;e<arguments.length;e++){var n=null!=arguments[e]?arguments[e]:{};e%2?l(Object(n),!0).forEach((function(e){r(t,e,n[e])})):Object.getOwnPropertyDescriptors?Object.defineProperties(t,Object.getOwnPropertyDescriptors(n)):l(Object(n)).forEach((function(e){Object.defineProperty(t,e,Object.getOwnPropertyDescriptor(n,e))}))}return t}function o(t,e){if(null==t)return{};var n,a,r=function(t,e){if(null==t)return{};var n,a,r={},l=Object.keys(t);for(a=0;a<l.length;a++)n=l[a],e.indexOf(n)>=0||(r[n]=t[n]);return r}(t,e);if(Object.getOwnPropertySymbols){var l=Object.getOwnPropertySymbols(t);for(a=0;a<l.length;a++)n=l[a],e.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(t,n)&&(r[n]=t[n])}return r}var p=a.createContext({}),m=function(t){var e=a.useContext(p),n=e;return t&&(n="function"==typeof t?t(e):i(i({},e),t)),n},d=function(t){var e=m(t.components);return a.createElement(p.Provider,{value:e},t.children)},s={inlineCode:"code",wrapper:function(t){var e=t.children;return a.createElement(a.Fragment,{},e)}},g=a.forwardRef((function(t,e){var n=t.components,r=t.mdxType,l=t.originalType,p=t.parentName,d=o(t,["components","mdxType","originalType","parentName"]),g=m(n),N=r,k=g["".concat(p,".").concat(N)]||g[N]||s[N]||l;return n?a.createElement(k,i(i({ref:e},d),{},{components:n})):a.createElement(k,i({ref:e},d))}));function N(t,e){var n=arguments,r=e&&e.mdxType;if("string"==typeof t||r){var l=n.length,i=new Array(l);i[0]=g;var o={};for(var p in e)hasOwnProperty.call(e,p)&&(o[p]=e[p]);o.originalType=t,o.mdxType="string"==typeof t?t:r,i[1]=o;for(var m=2;m<l;m++)i[m]=n[m];return a.createElement.apply(null,i)}return a.createElement.apply(null,n)}g.displayName="MDXCreateElement"},22166:function(t,e,n){n.r(e),n.d(e,{contentTitle:function(){return p},default:function(){return g},frontMatter:function(){return o},metadata:function(){return m},toc:function(){return d}});var a=n(87462),r=n(63366),l=(n(67294),n(3905)),i=["components"],o={},p=void 0,m={unversionedId:"data-sources/1000Genomes-snv-json",id:"version-3.2.5/data-sources/1000Genomes-snv-json",title:"1000Genomes-snv-json",description:"| Field            | Type         | Notes                     |",source:"@site/versioned_docs/version-3.2.5/data-sources/1000Genomes-snv-json.md",sourceDirName:"data-sources",slug:"/data-sources/1000Genomes-snv-json",permalink:"/IlluminaAnnotatorDocumentation/3.2.5/data-sources/1000Genomes-snv-json",editUrl:"https://github.com/Illumina/IlluminaAnnotatorDocumentation/edit/master/versioned_docs/version-3.2.5/data-sources/1000Genomes-snv-json.md",tags:[],version:"3.2.5",frontMatter:{}},d=[],s={toc:d};function g(t){var e=t.components,n=(0,r.Z)(t,i);return(0,l.kt)("wrapper",(0,a.Z)({},s,n,{components:e,mdxType:"MDXLayout"}),(0,l.kt)("pre",null,(0,l.kt)("code",{parentName:"pre",className:"language-json"},'"oneKg":{\n   "allAf":0.200879,\n   "afrAf":0.210287,\n   "amrAf":0.139769,\n   "easAf":0.275794,\n   "eurAf":0.181909,\n   "sasAf":0.173824,\n   "allAn":5008,\n   "afrAn":1322,\n   "amrAn":694,\n   "easAn":1008,\n   "eurAn":1006,\n   "sasAn":978,\n   "allAc":1006,\n   "afrAc":278,\n   "amrAc":97,\n   "easAc":278,\n   "eurAc":183,\n   "sasAc":170\n}\n')),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,l.kt)("th",{parentName:"tr",align:"left"},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"allAf"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele frequency for all populations. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"allAc"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele count for all populations. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"allAn"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele number for all populations. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"afrAf"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele frequency for the African super population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"afrAc"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele count for the African super population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"afrAn"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele number for the African super population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"amrAf"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele frequency for the Ad Mixed American super population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"amrAc"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele count for the Ad Mixed American super population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"amrAn"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele number for the Ad Mixed American super population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"easAf"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele frequency for the East Asian super population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"easAc"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele count for the East Asian super population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"easAn"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele number for the East Asian super population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"eurAf"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele frequency for the European super population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"eurAc"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele count for the European super population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"eurAn"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele number for the European super population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"sasAf"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele frequency for the South Asian super population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"sasAc"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele count for the South Asian super population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"sasAn"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"allele number for the South Asian super population. Non-zero integer.")))))}g.isMDXComponent=!0},50092:function(t,e,n){n.r(e),n.d(e,{contentTitle:function(){return p},default:function(){return g},frontMatter:function(){return o},metadata:function(){return m},toc:function(){return d}});var a=n(87462),r=n(63366),l=(n(67294),n(3905)),i=["components"],o={},p=void 0,m={unversionedId:"data-sources/1000Genomes-sv-json",id:"version-3.2.5/data-sources/1000Genomes-sv-json",title:"1000Genomes-sv-json",description:"| Field             | Type           | Notes                                                                                                                           |",source:"@site/versioned_docs/version-3.2.5/data-sources/1000Genomes-sv-json.md",sourceDirName:"data-sources",slug:"/data-sources/1000Genomes-sv-json",permalink:"/IlluminaAnnotatorDocumentation/3.2.5/data-sources/1000Genomes-sv-json",editUrl:"https://github.com/Illumina/IlluminaAnnotatorDocumentation/edit/master/versioned_docs/version-3.2.5/data-sources/1000Genomes-sv-json.md",tags:[],version:"3.2.5",frontMatter:{}},d=[],s={toc:d};function g(t){var e=t.components,n=(0,r.Z)(t,i);return(0,l.kt)("wrapper",(0,a.Z)({},s,n,{components:e,mdxType:"MDXLayout"}),(0,l.kt)("pre",null,(0,l.kt)("code",{parentName:"pre",className:"language-json"},'"oneKg":[\n   {\n      "chromosome":"1",\n      "begin":1595369,\n      "end":1612441,\n      "variantType": "copy_number_variation",\n      "id": "esv3635753;esv3635754;esv3635755;esv3635756;esv3635757",\n      "allAn": 5008,\n      "allAc": 2702,\n      "allAf": 0.539537,\n      "afrAf": 0.6052,\n      "amrAf": 0.3675,\n      "eurAf": 0.5357,\n      "easAf": 0.5368,\n      "sasAf": 0.5797,\n      "reciprocalOverlap": 0.07555\n   }\n],\n')),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:null},"Field"),(0,l.kt)("th",{parentName:"tr",align:null},"Type"),(0,l.kt)("th",{parentName:"tr",align:null},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"chromosome"),(0,l.kt)("td",{parentName:"tr",align:null},"string"),(0,l.kt)("td",{parentName:"tr",align:null})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"begin"),(0,l.kt)("td",{parentName:"tr",align:null},"integer"),(0,l.kt)("td",{parentName:"tr",align:null})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"end"),(0,l.kt)("td",{parentName:"tr",align:null},"integer"),(0,l.kt)("td",{parentName:"tr",align:null})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"variantType"),(0,l.kt)("td",{parentName:"tr",align:null},"string"),(0,l.kt)("td",{parentName:"tr",align:null})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"id"),(0,l.kt)("td",{parentName:"tr",align:null},"string"),(0,l.kt)("td",{parentName:"tr",align:null})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"allAn"),(0,l.kt)("td",{parentName:"tr",align:null},"floating point"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for all populations. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"allAc"),(0,l.kt)("td",{parentName:"tr",align:null},"floating point"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for all populations. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"allAf"),(0,l.kt)("td",{parentName:"tr",align:null},"floating point"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for all populations. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"afrAf"),(0,l.kt)("td",{parentName:"tr",align:null},"floating point"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the African super population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"amrAf"),(0,l.kt)("td",{parentName:"tr",align:null},"floating point"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the Ad Mixed American super population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"eurAf"),(0,l.kt)("td",{parentName:"tr",align:null},"floating point"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the European super population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"easAf"),(0,l.kt)("td",{parentName:"tr",align:null},"integer"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the East Asian super population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"sasAf"),(0,l.kt)("td",{parentName:"tr",align:null},"integer"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the South Asian super population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"reciprocalOverlap"),(0,l.kt)("td",{parentName:"tr",align:null},"floating point"),(0,l.kt)("td",{parentName:"tr",align:null},"range: 0 - 1.")))))}g.isMDXComponent=!0},86631:function(t,e,n){n.r(e),n.d(e,{contentTitle:function(){return p},default:function(){return g},frontMatter:function(){return o},metadata:function(){return m},toc:function(){return d}});var a=n(87462),r=n(63366),l=(n(67294),n(3905)),i=["components"],o={},p=void 0,m={unversionedId:"data-sources/clinvar-json",id:"version-3.2.5/data-sources/clinvar-json",title:"clinvar-json",description:"| Field            | Type         | Notes                     |",source:"@site/versioned_docs/version-3.2.5/data-sources/clinvar-json.md",sourceDirName:"data-sources",slug:"/data-sources/clinvar-json",permalink:"/IlluminaAnnotatorDocumentation/3.2.5/data-sources/clinvar-json",editUrl:"https://github.com/Illumina/IlluminaAnnotatorDocumentation/edit/master/versioned_docs/version-3.2.5/data-sources/clinvar-json.md",tags:[],version:"3.2.5",frontMatter:{}},d=[],s={toc:d};function g(t){var e=t.components,n=(0,r.Z)(t,i);return(0,l.kt)("wrapper",(0,a.Z)({},s,n,{components:e,mdxType:"MDXLayout"}),(0,l.kt)("pre",null,(0,l.kt)("code",{parentName:"pre",className:"language-json"},'"clinvar":[\n   {\n      "id":"RCV000030258.4",\n      "reviewStatus":"reviewed by expert panel",\n      "alleleOrigins":[\n         "germline"\n      ],\n      "refAllele":"G",\n      "altAllele":"A",\n      "phenotypes":[\n         "Lynch syndrome"\n      ],\n      "medGenIds":[\n         "C1333990"\n      ],\n      "omimIds":[\n         "120435"\n      ],\n      "significance":[\n         "benign"\n      ],\n      "lastUpdatedDate":"2017-05-01",\n      "isAlleleSpecific":true\n   }\n]\n')),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,l.kt)("th",{parentName:"tr",align:"left"},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"id"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"ClinVar ID")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"reviewStatus"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"see possible values below")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"alleleOrigins"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"see possible values below")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"refAllele"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"altAllele"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"phenotypes"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"medGenIds"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"MedGen IDs")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"omimIds"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"OMIM IDs")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"orphanetIds"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"Orphanet IDs")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"significance"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"see possible values below")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"lastUpdatedDate"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"yyyy-MM-dd")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"pubMedIds"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"PubMed IDs")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"isAlleleSpecific"),(0,l.kt)("td",{parentName:"tr",align:"center"},"bool"),(0,l.kt)("td",{parentName:"tr",align:"left"},"true when the current variant alternate allele matches the ClinVar alternate allele")))),(0,l.kt)("p",null,(0,l.kt)("strong",{parentName:"p"},"reviewStatus:")),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},"no assertion provided"),(0,l.kt)("li",{parentName:"ul"},"no assertion criteria provided"),(0,l.kt)("li",{parentName:"ul"},"criteria provided, single submitter"),(0,l.kt)("li",{parentName:"ul"},"practice guideline"),(0,l.kt)("li",{parentName:"ul"},"classified by multiple submitters"),(0,l.kt)("li",{parentName:"ul"},"criteria provided, conflicting interpretations"),(0,l.kt)("li",{parentName:"ul"},"criteria provided, multiple submitters, no conflicts"),(0,l.kt)("li",{parentName:"ul"},"no interpretation for the single variant")),(0,l.kt)("p",null,(0,l.kt)("strong",{parentName:"p"},"alleleOrigins:")),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},"unknown"),(0,l.kt)("li",{parentName:"ul"},"other"),(0,l.kt)("li",{parentName:"ul"},"germline"),(0,l.kt)("li",{parentName:"ul"},"somatic"),(0,l.kt)("li",{parentName:"ul"},"inherited"),(0,l.kt)("li",{parentName:"ul"},"paternal"),(0,l.kt)("li",{parentName:"ul"},"maternal"),(0,l.kt)("li",{parentName:"ul"},"de-novo"),(0,l.kt)("li",{parentName:"ul"},"biparental"),(0,l.kt)("li",{parentName:"ul"},"uniparental"),(0,l.kt)("li",{parentName:"ul"},"not-tested"),(0,l.kt)("li",{parentName:"ul"},"tested-inconclusive")),(0,l.kt)("p",null,(0,l.kt)("strong",{parentName:"p"},"significance:")),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},"uncertain significance"),(0,l.kt)("li",{parentName:"ul"},"not provided"),(0,l.kt)("li",{parentName:"ul"},"benign"),(0,l.kt)("li",{parentName:"ul"},"likely benign"),(0,l.kt)("li",{parentName:"ul"},"likely pathogenic"),(0,l.kt)("li",{parentName:"ul"},"pathogenic"),(0,l.kt)("li",{parentName:"ul"},"drug response"),(0,l.kt)("li",{parentName:"ul"},"histocompatibility"),(0,l.kt)("li",{parentName:"ul"},"association"),(0,l.kt)("li",{parentName:"ul"},"risk factor"),(0,l.kt)("li",{parentName:"ul"},"protective"),(0,l.kt)("li",{parentName:"ul"},"affects"),(0,l.kt)("li",{parentName:"ul"},"conflicting data from submitters"),(0,l.kt)("li",{parentName:"ul"},"other"),(0,l.kt)("li",{parentName:"ul"},"no interpretation for the single variant"),(0,l.kt)("li",{parentName:"ul"},"conflicting interpretations of pathogenicity")))}g.isMDXComponent=!0},59999:function(t,e,n){n.r(e),n.d(e,{contentTitle:function(){return p},default:function(){return g},frontMatter:function(){return o},metadata:function(){return m},toc:function(){return d}});var a=n(87462),r=n(63366),l=(n(67294),n(3905)),i=["components"],o={},p=void 0,m={unversionedId:"data-sources/dbsnp-json",id:"version-3.2.5/data-sources/dbsnp-json",title:"dbsnp-json",description:"| Field            | Type         | Notes                     |",source:"@site/versioned_docs/version-3.2.5/data-sources/dbsnp-json.md",sourceDirName:"data-sources",slug:"/data-sources/dbsnp-json",permalink:"/IlluminaAnnotatorDocumentation/3.2.5/data-sources/dbsnp-json",editUrl:"https://github.com/Illumina/IlluminaAnnotatorDocumentation/edit/master/versioned_docs/version-3.2.5/data-sources/dbsnp-json.md",tags:[],version:"3.2.5",frontMatter:{}},d=[],s={toc:d};function g(t){var e=t.components,n=(0,r.Z)(t,i);return(0,l.kt)("wrapper",(0,a.Z)({},s,n,{components:e,mdxType:"MDXLayout"}),(0,l.kt)("pre",null,(0,l.kt)("code",{parentName:"pre",className:"language-json"},'"dbsnp":[\n   "rs1042821"\n]\n')),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,l.kt)("th",{parentName:"tr",align:"left"},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"dbsnp"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"dbSNP rsIDs")))))}g.isMDXComponent=!0},92811:function(t,e,n){n.r(e),n.d(e,{contentTitle:function(){return p},default:function(){return g},frontMatter:function(){return o},metadata:function(){return m},toc:function(){return d}});var a=n(87462),r=n(63366),l=(n(67294),n(3905)),i=["components"],o={},p=void 0,m={unversionedId:"data-sources/gnomad-exomes-small-variants-json",id:"version-3.2.5/data-sources/gnomad-exomes-small-variants-json",title:"gnomad-exomes-small-variants-json",description:"| Field               | Type   | Notes                                                                                            |",source:"@site/versioned_docs/version-3.2.5/data-sources/gnomad-exomes-small-variants-json.md",sourceDirName:"data-sources",slug:"/data-sources/gnomad-exomes-small-variants-json",permalink:"/IlluminaAnnotatorDocumentation/3.2.5/data-sources/gnomad-exomes-small-variants-json",editUrl:"https://github.com/Illumina/IlluminaAnnotatorDocumentation/edit/master/versioned_docs/version-3.2.5/data-sources/gnomad-exomes-small-variants-json.md",tags:[],version:"3.2.5",frontMatter:{}},d=[],s={toc:d};function g(t){var e=t.components,n=(0,r.Z)(t,i);return(0,l.kt)("wrapper",(0,a.Z)({},s,n,{components:e,mdxType:"MDXLayout"}),(0,l.kt)("pre",null,(0,l.kt)("code",{parentName:"pre",className:"language-json"},'"gnomadExome":{ \n   "coverage":20,\n   "allAf":0.190317,\n   "afrAf":0.222876,\n   "amrAf":0.121394,\n   "easAf":0.239802,\n   "finAf":0.136833,\n   "nfeAf":0.181282,\n   "asjAf":0.258278,\n   "othAf":0.186094,\n   "allAn":30796,\n   "afrAn":8664,\n   "amrAn":832,\n   "easAn":1618,\n   "finAn":3486,\n   "nfeAn":14916,\n   "asjAn":302,\n   "othAn":978,\n   "allAc":5861,\n   "afrAc":1931,\n   "amrAc":101,\n   "easAc":388,\n   "finAc":477,\n   "nfeAc":2704,\n   "asjAc":78,\n   "othAc":182,\n   "allHc":561,\n   "afrHc":208,\n   "amrHc":6,\n   "easHc":42,\n   "finHc":31,\n   "nfeHc":242,\n   "asjHc":13,\n   "othHc":19,\n   "failedFilter":true\n}\n')),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:null},"Field"),(0,l.kt)("th",{parentName:"tr",align:null},"Type"),(0,l.kt)("th",{parentName:"tr",align:null},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"coverage"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"average coverage (non-negative integer values)")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"allAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for all populations. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"allAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for all populations. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"allAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for all populations. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"allHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for all populations. Non-negative integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"afrAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the African / African American population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"afrAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the African / African American population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"afrAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the African / African American population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"afrHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for African / African American population. Non-negative integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"amrAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the Latino population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"amrAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the Latino population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"amrAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the Latino population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"amrHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for Latino population. Non-negative integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"easAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the East Asian population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"easAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the East Asian population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"easAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the East Asian population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"easHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for East Asian population. Non-negative integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"finAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the Finnish population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"finAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the Finnish population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"finAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the Finnish population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"finHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for Finnish population. Non-negative integer")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"nfeAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the Non-Finnish European population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"nfeAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the Non-Finnish European population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"nfeAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the Non-Finnish European population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"nfeHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for Non-Finnish European population. Non-negative integer")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"othAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the Other population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"othAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the Other population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"othAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the Other population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"othHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for Other population. Non-negative integer")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"asjAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the Ashkenazi Jewish population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"asjAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the Ashkenazi Jewish population Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"asjAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the Ashkenazi Jewish population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"asjHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for the Ashkenazi Jewish population. Non-negative integer")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"sasAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the South Asian population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"sasAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the South Asian population Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"sasAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the South Asian population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"sasHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for the South Asian population. Non-negative integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"failedFilter"),(0,l.kt)("td",{parentName:"tr",align:null},"bool"),(0,l.kt)("td",{parentName:"tr",align:null},"True if this variant failed any filters (Note: we do not list the failed filters)")))))}g.isMDXComponent=!0},64501:function(t,e,n){n.r(e),n.d(e,{contentTitle:function(){return p},default:function(){return g},frontMatter:function(){return o},metadata:function(){return m},toc:function(){return d}});var a=n(87462),r=n(63366),l=(n(67294),n(3905)),i=["components"],o={},p=void 0,m={unversionedId:"data-sources/gnomad-genomes-small-variants-json",id:"version-3.2.5/data-sources/gnomad-genomes-small-variants-json",title:"gnomad-genomes-small-variants-json",description:"| Field               | Type   | Notes                                                                                            |",source:"@site/versioned_docs/version-3.2.5/data-sources/gnomad-genomes-small-variants-json.md",sourceDirName:"data-sources",slug:"/data-sources/gnomad-genomes-small-variants-json",permalink:"/IlluminaAnnotatorDocumentation/3.2.5/data-sources/gnomad-genomes-small-variants-json",editUrl:"https://github.com/Illumina/IlluminaAnnotatorDocumentation/edit/master/versioned_docs/version-3.2.5/data-sources/gnomad-genomes-small-variants-json.md",tags:[],version:"3.2.5",frontMatter:{}},d=[],s={toc:d};function g(t){var e=t.components,n=(0,r.Z)(t,i);return(0,l.kt)("wrapper",(0,a.Z)({},s,n,{components:e,mdxType:"MDXLayout"}),(0,l.kt)("pre",null,(0,l.kt)("code",{parentName:"pre",className:"language-json"},'"gnomad":{ \n   "coverage":20,\n   "allAf":0.190317,\n   "afrAf":0.222876,\n   "amrAf":0.121394,\n   "easAf":0.239802,\n   "finAf":0.136833,\n   "nfeAf":0.181282,\n   "asjAf":0.258278,\n   "othAf":0.186094,\n   "allAn":30796,\n   "afrAn":8664,\n   "amrAn":832,\n   "easAn":1618,\n   "finAn":3486,\n   "nfeAn":14916,\n   "asjAn":302,\n   "othAn":978,\n   "allAc":5861,\n   "afrAc":1931,\n   "amrAc":101,\n   "easAc":388,\n   "finAc":477,\n   "nfeAc":2704,\n   "asjAc":78,\n   "othAc":182,\n   "allHc":561,\n   "afrHc":208,\n   "amrHc":6,\n   "easHc":42,\n   "finHc":31,\n   "nfeHc":242,\n   "asjHc":13,\n   "othHc":19,\n   "failedFilter":true\n}\n')),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:null},"Field"),(0,l.kt)("th",{parentName:"tr",align:null},"Type"),(0,l.kt)("th",{parentName:"tr",align:null},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"coverage"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"average coverage (non-negative integer values)")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"allAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for all populations. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"allAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for all populations. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"allAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for all populations. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"allHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for all populations. Non-negative integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"afrAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the African / African American population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"afrAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the African / African American population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"afrAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the African / African American population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"afrHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for African / African American population. Non-negative integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"amrAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the Latino population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"amrAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the Latino population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"amrAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the Latino population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"amrHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for Latino population. Non-negative integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"easAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the East Asian population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"easAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the East Asian population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"easAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the East Asian population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"easHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for East Asian population. Non-negative integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"finAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the Finnish population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"finAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the Finnish population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"finAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the Finnish population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"finHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for Finnish population. Non-negative integer")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"nfeAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the Non-Finnish European population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"nfeAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the Non-Finnish European population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"nfeAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the Non-Finnish European population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"nfeHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for Non-Finnish European population. Non-negative integer")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"othAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the Other population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"othAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the Other population. Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"othAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the Other population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"othHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for Other population. Non-negative integer")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"asjAf"),(0,l.kt)("td",{parentName:"tr",align:null},"float"),(0,l.kt)("td",{parentName:"tr",align:null},"allele frequency for the Ashkenazi Jewish population. Range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"asjAc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele count for the Ashkenazi Jewish population Integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"asjAn"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"allele number for the Ashkenazi Jewish population. Non-zero integer.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"asjHc"),(0,l.kt)("td",{parentName:"tr",align:null},"int"),(0,l.kt)("td",{parentName:"tr",align:null},"count of homozygous individuals for the Ashkenazi Jewish population. Non-negative integer")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:null},"failedFilter"),(0,l.kt)("td",{parentName:"tr",align:null},"bool"),(0,l.kt)("td",{parentName:"tr",align:null},"True if this variant failed any filters (Note: we do not list the failed filters)")))))}g.isMDXComponent=!0},16712:function(t,e,n){n.r(e),n.d(e,{contentTitle:function(){return k},default:function(){return h},frontMatter:function(){return N},metadata:function(){return u},toc:function(){return c}});var a=n(87462),r=n(63366),l=(n(67294),n(3905)),i=n(86631),o=n(59999),p=n(64501),m=n(92811),d=n(22166),s=n(50092),g=["components"],N={title:"Nirvana JSON File Format"},k=void 0,u={unversionedId:"file-formats/nirvana-json-file-format",id:"version-3.2.5/file-formats/nirvana-json-file-format",title:"Nirvana JSON File Format",description:"Overview",source:"@site/versioned_docs/version-3.2.5/file-formats/nirvana-json-file-format.mdx",sourceDirName:"file-formats",slug:"/file-formats/nirvana-json-file-format",permalink:"/IlluminaAnnotatorDocumentation/3.2.5/file-formats/nirvana-json-file-format",editUrl:"https://github.com/Illumina/IlluminaAnnotatorDocumentation/edit/master/versioned_docs/version-3.2.5/file-formats/nirvana-json-file-format.mdx",tags:[],version:"3.2.5",frontMatter:{title:"Nirvana JSON File Format"},sidebar:"version-3.2.5/docs",previous:{title:"gnomAD",permalink:"/IlluminaAnnotatorDocumentation/3.2.5/data-sources/gnomad"},next:{title:"Variant IDs",permalink:"/IlluminaAnnotatorDocumentation/3.2.5/core-functionality/variant-ids"}},c=[{value:"Overview",id:"overview",children:[{value:"Conventions",id:"conventions",children:[],level:3},{value:"JSON Layout",id:"json-layout",children:[],level:3}],level:2},{value:"Header",id:"header",children:[{value:"Data Source",id:"data-source",children:[],level:4},{value:"Genome Assemblies",id:"genome-assemblies",children:[],level:4}],level:2},{value:"Positions",id:"positions",children:[{value:"1000 Genomes (SV)",id:"1000-genomes-sv",children:[],level:3}],level:2},{value:"Samples",id:"samples",children:[],level:2},{value:"Variants",id:"variants",children:[{value:"Transcripts",id:"transcripts",children:[{value:"PolyPhen",id:"polyphen",children:[],level:4},{value:"SIFT",id:"sift",children:[],level:4},{value:"Gene Fusions",id:"gene-fusions",children:[],level:4},{value:"Fusion",id:"fusion",children:[],level:4}],level:3},{value:"Regulatory Regions",id:"regulatory-regions",children:[{value:"Regulatory Types",id:"regulatory-types",children:[],level:4},{value:"Regulatory Consequences",id:"regulatory-consequences",children:[],level:4}],level:3},{value:"ClinVar",id:"clinvar",children:[],level:3},{value:"1000 Genomes",id:"1000-genomes",children:[],level:3},{value:"gnomAD (genomes)",id:"gnomad-genomes",children:[],level:3},{value:"gnomAD (exomes)",id:"gnomad-exomes",children:[],level:3},{value:"dbSNP",id:"dbsnp",children:[],level:3}],level:2}],f={toc:c};function h(t){var e=t.components,N=(0,r.Z)(t,g);return(0,l.kt)("wrapper",(0,a.Z)({},f,N,{components:e,mdxType:"MDXLayout"}),(0,l.kt)("h2",{id:"overview"},"Overview"),(0,l.kt)("h3",{id:"conventions"},"Conventions"),(0,l.kt)("p",null,"In the Nirvana JSON representation, we try to maximize the amount of useful information that is relayed in the output file. As such, we have several conventions that are useful to know about:"),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},"With boolean key/value pairs, we only output the keys that have a true value. I.e. there's no reason to display ",(0,l.kt)("inlineCode",{parentName:"li"},'"isStructuralVariant":false')," a few million times when annotating a small variant VCF."),(0,l.kt)("li",{parentName:"ul"},"When transferring data from the VCF file to the JSON (e.g. for allele depths (AD)), it is common to use a period (.) as a placeholder for missing data in the VCF file. Nirvana treats periods like empty or null strings and therefore will not output those entries.")),(0,l.kt)("h3",{id:"json-layout"},"JSON Layout"),(0,l.kt)("p",null,(0,l.kt)("img",{src:n(33658).Z})),(0,l.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,l.kt)("div",{parentName:"div",className:"admonition-heading"},(0,l.kt)("h5",{parentName:"div"},(0,l.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,l.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,l.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"info")),(0,l.kt)("div",{parentName:"div",className:"admonition-content"},(0,l.kt)("p",{parentName:"div"},"In general, each position corresponds to a row in the original VCF file."),(0,l.kt)("p",{parentName:"div"},"For each gene that was referenced in the transcripts found in the positions section, there will be additional gene-level annotation in the gene section."))),(0,l.kt)("h2",{id:"header"},"Header"),(0,l.kt)("pre",null,(0,l.kt)("code",{parentName:"pre",className:"language-json"},'{ \n   "header":{ \n      "annotator":"Nirvana 3.2.5",\n      "creationTime":"2022-12-05 16:43:41",\n      "genomeAssembly":"GRCh37",\n      "schemaVersion":6,\n      "dataVersion":"91.26.50",\n      "dataSources":[ \n         { \n            "name":"VEP",\n            "version":"91",\n            "description":"RefSeq",\n            "releaseDate":"2018-03-05"\n         },\n         { \n            "name":"ClinVar",\n            "version":"20190204",\n            "description":"A freely accessible, public archive of reports of the relationships among human variations and phenotypes, with supporting evidence",\n            "releaseDate":"2019-02-04"\n         }\n      ],\n      "samples":[ \n         "NA12878",\n         "NA12891",\n         "NA12892"\n      ]\n   },\n')),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,l.kt)("th",{parentName:"tr",align:"left"},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"annotator"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"the name of the annotator and the current version")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"creationTime"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"yyyy-MM-dd hh:mm:ss")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"genomeAssembly"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"see ",(0,l.kt)("a",{parentName:"td",href:"#genome-assemblies"},"possible values below"))),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"schemaVersion"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer"),(0,l.kt)("td",{parentName:"tr",align:"left"},"incremented whenever the core structure of the JSON file introduces breaking changes")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"dataVersion"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"dataSources"),(0,l.kt)("td",{parentName:"tr",align:"center"},"object array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"see ",(0,l.kt)("a",{parentName:"td",href:"#data-source"},"Data Source entry below"))),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"samples"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"the order of these sample names will be used throughout the JSON file when enumerating samples")))),(0,l.kt)("h4",{id:"data-source"},"Data Source"),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,l.kt)("th",{parentName:"tr",align:"left"},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"name"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"version"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"description"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"optional description of the data source")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"releaseDate"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"yyyy-MM-dd")))),(0,l.kt)("h4",{id:"genome-assemblies"},"Genome Assemblies"),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},"GRCh37"),(0,l.kt)("li",{parentName:"ul"},"GRCh38"),(0,l.kt)("li",{parentName:"ul"},"hg19")),(0,l.kt)("h2",{id:"positions"},"Positions"),(0,l.kt)("pre",null,(0,l.kt)("code",{parentName:"pre",className:"language-json"},'"positions":[ \n   { \n      "chromosome":"chr2",\n      "position":48010488,\n      "repeatUnit":"GGCCCC",\n      "refRepeatCount":3,\n      "svEnd":48020488,\n      "refAllele":"G",\n      "altAlleles":[ \n         "A",\n         "GT"\n      ],\n      "quality":461,\n      "filters":[ \n         "PASS"\n      ],\n      "ciPos":[ \n         -170,\n         170\n      ],\n      "ciEnd":[ \n         -175,\n         175\n      ],\n      "svLength":1000,\n      "strandBias":1.23,\n      "jointSomaticNormalQuality":29,\n      "cytogeneticBand":"2p16.3",\n')),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Variant Type"),(0,l.kt)("th",{parentName:"tr",align:"left"},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"chromosome"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"center"},"all"),(0,l.kt)("td",{parentName:"tr",align:"left"},"exactly as displayed in the vcf")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"postion"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer"),(0,l.kt)("td",{parentName:"tr",align:"center"},"all"),(0,l.kt)("td",{parentName:"tr",align:"left"},"exactly as displayed in the vcf (1-based notation). Range: 1 - 250 million")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"repeatUnit"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"center"},"STR"),(0,l.kt)("td",{parentName:"tr",align:"left"},"provided by ExpansionHunter")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"refRepeatCount"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer"),(0,l.kt)("td",{parentName:"tr",align:"center"},"STR"),(0,l.kt)("td",{parentName:"tr",align:"left"},"provided by ExpansionHunter")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"svEnd"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer"),(0,l.kt)("td",{parentName:"tr",align:"center"},"SV"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"refAllele"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"center"},"all"),(0,l.kt)("td",{parentName:"tr",align:"left"},"exactly as displayed in the vcf")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"altAllele"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"center"},"all"),(0,l.kt)("td",{parentName:"tr",align:"left"},"exactly as displayed in the vcf")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"quality"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"center"},"all"),(0,l.kt)("td",{parentName:"tr",align:"left"},"exactly as displayed in the vcf (Normally an integer, but some variant callers using floating point. Has been observed as high as 500k)")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"filters"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"center"},"all"),(0,l.kt)("td",{parentName:"tr",align:"left"},"exactly as displayed in the vcf")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"ciPos"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer array"),(0,l.kt)("td",{parentName:"tr",align:"center"},"SV"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"ciEnd"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer array"),(0,l.kt)("td",{parentName:"tr",align:"center"},"SV"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"svLength"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer"),(0,l.kt)("td",{parentName:"tr",align:"center"},"SV"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"strandBias"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"center"},"small variant"),(0,l.kt)("td",{parentName:"tr",align:"left"},"provided by GATK (from SB)")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"jointSomaticNormalQuality"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer"),(0,l.kt)("td",{parentName:"tr",align:"center"},"SV"),(0,l.kt)("td",{parentName:"tr",align:"left"},"provided by the Manta variant caller (SOMATICSCORE)")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"cytogeneticBand"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"center"},"all"),(0,l.kt)("td",{parentName:"tr",align:"left"},"e.g. 17p13.1")))),(0,l.kt)("h3",{id:"1000-genomes-sv"},"1000 Genomes (SV)"),(0,l.kt)(s.default,{mdxType:"ThousandGenomesSV"}),(0,l.kt)("h2",{id:"samples"},"Samples"),(0,l.kt)("pre",null,(0,l.kt)("code",{parentName:"pre",className:"language-json"},'"samples":[\n   {\n      "genotype":"0/1",\n      "variantFrequencies":[\n         0.333,\n         0.5\n      ],\n      "totalDepth":57,\n      "genotypeQuality":12,\n      "copyNumber":3,\n      "repeatUnitCounts":[\n         10,\n         20\n      ],\n      "alleleDepths":[\n         10,\n         20,\n         30\n      ],\n      "failedFilter":true,\n      "splitReadCounts":[\n         10,\n         20\n      ],\n      "pairedEndReadCounts":[\n         10,\n         20\n      ],\n      "diseaseAffectedStatuses":[\n         "-"\n      ],\n      "artifactAdjustedQualityScore":89.3,\n      "likelihoodRatioQualityScore":78.2\n   }\n]\n')),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,l.kt)("th",{parentName:"tr",align:"left"},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"genotype"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"repeatNumbers"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"ExpansionHunter-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"repeatNumberSpans"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"ExpansionHunter-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"variantFrequencies"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"range: 0 - 1.0. One value per alternate allele")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"totalDepth"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer"),(0,l.kt)("td",{parentName:"tr",align:"left"},"non-negative integer values")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"genotypeQuality"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer"),(0,l.kt)("td",{parentName:"tr",align:"left"},"non-negative integer values. Typically maxes out at 99")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"copyNumber"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer"),(0,l.kt)("td",{parentName:"tr",align:"left"},"non-negative integer values")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"alleleDepths"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"non-negative integer values")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"failedFilter"),(0,l.kt)("td",{parentName:"tr",align:"center"},"bool"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"splitReadCounts"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"Manta-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"pairedEndReadCounts"),(0,l.kt)("td",{parentName:"tr",align:"center"},"integer array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"Manta-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"lossOfHeterozygosity"),(0,l.kt)("td",{parentName:"tr",align:"center"},"bool"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"deNovoQuality"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"mpileupAlleleDepths"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"SMN1-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"silentCarrierHaplotype"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"SMN1-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"paralogousEntrezGeneIds"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"SMN1-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"paralogousGeneCopyNumbers"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"SMN1-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"diseaseClassificationSources"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"SMN1-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"diseaseIds"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"SMN1-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"diseaseAffectedStatuses"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"SMN1-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"proteinAlteringVariantPositions"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"SMN1-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"isCompoundHetCompatible"),(0,l.kt)("td",{parentName:"tr",align:"center"},"bool"),(0,l.kt)("td",{parentName:"tr",align:"left"},"SMN1-specific")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"artifactAdjustedQualityScore"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"left"},"PEPE-specific. Range: 0 - 100.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"likelihoodRatioQualityScore"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"left"},"PEPE-specific. Range: 0 - 100.0")))),(0,l.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,l.kt)("div",{parentName:"div",className:"admonition-heading"},(0,l.kt)("h5",{parentName:"div"},(0,l.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,l.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,l.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"Empty Samples")),(0,l.kt)("div",{parentName:"div",className:"admonition-content"},(0,l.kt)("p",{parentName:"div"},"If a sample does not contain any entries, we will create a sample object that contains the ",(0,l.kt)("inlineCode",{parentName:"p"},"isEmpty")," key. This ensures that sample ordering is preserved while indicating that a sample is intentionally empty."),(0,l.kt)("pre",{parentName:"div"},(0,l.kt)("code",{parentName:"pre",className:"language-json"},'"samples":[ \n   { \n      "isEmpty":true\n   }\n],\n')))),(0,l.kt)("h2",{id:"variants"},"Variants"),(0,l.kt)("pre",null,(0,l.kt)("code",{parentName:"pre",className:"language-json"},'"variants":[ \n   { \n      "vid":"2:48010488:A",\n      "chromosome":"chr2",\n      "begin":48010488,\n      "end":48010488,\n      "isReferenceMinorAllele":true,\n      "isStructuralVariant":true,\n      "refAllele":"G",\n      "altAllele":"A",\n      "variantType":"SNV",\n      "isDecomposedVariant":true,\n      "isRecomposedVariant":true,\n      "hgvsg":"NC_000002.11:g.48010488G>A",\n      "phylopScore":0.459\n')),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,l.kt)("th",{parentName:"tr",align:"left"},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"vid"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"see ",(0,l.kt)("a",{parentName:"td",href:"../core-functionality/variant-ids"},"Variant Identifiers"))),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"chromosome"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"begin"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"1-based non-negative integer values. Range: 1 - 250 million")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"end"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"1-based non-negative integer values. Range: 1 - 250 million")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"isReferenceMinorAllele"),(0,l.kt)("td",{parentName:"tr",align:"center"},"bool"),(0,l.kt)("td",{parentName:"tr",align:"left"},"true when this is a reference minor allele")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"isStructuralVariant"),(0,l.kt)("td",{parentName:"tr",align:"center"},"bool"),(0,l.kt)("td",{parentName:"tr",align:"left"},"true when the variant is a structural variant")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"refAllele"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"parsimonious representation of the reference allele")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"altAllele"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"parsimonious representation of the alternate allele.")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"variantType"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"uses\xa0",(0,l.kt)("a",{parentName:"td",href:"http://www.sequenceontology.org/browser/current_svn/term/SO:0001059"},"Sequence Ontology sequence alterations"))),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"isDecomposedVariant"),(0,l.kt)("td",{parentName:"tr",align:"center"},"bool"),(0,l.kt)("td",{parentName:"tr",align:"left"},"true when the decomposed variant has been used to create another recomposed variant")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"isRecomposedVariant"),(0,l.kt)("td",{parentName:"tr",align:"center"},"bool"),(0,l.kt)("td",{parentName:"tr",align:"left"},"true when the variant is recomposed from two or more decomposed variants")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"hgvsg"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"HGVS g. notation")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"phylopScore"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"left"},"phyloP conservation score. Range: -14.08 to 6.424")))),(0,l.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,l.kt)("div",{parentName:"div",className:"admonition-heading"},(0,l.kt)("h5",{parentName:"div"},(0,l.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,l.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,l.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"Reference Minor Alleles")),(0,l.kt)("div",{parentName:"div",className:"admonition-content"},(0,l.kt)("p",{parentName:"div"},"Nirvana supports annotating reference minor alleles. In such a case, ",(0,l.kt)("inlineCode",{parentName:"p"},"refAllele")," will be replaced by the global major allele and ",(0,l.kt)("inlineCode",{parentName:"p"},"altAllele")," will be replaced with the original reference allele."))),(0,l.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,l.kt)("div",{parentName:"div",className:"admonition-heading"},(0,l.kt)("h5",{parentName:"div"},(0,l.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,l.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,l.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"Flagging Decomposed & Recomposed Variants")),(0,l.kt)("div",{parentName:"div",className:"admonition-content"},(0,l.kt)("p",{parentName:"div"},"When two or more decomposed variants are recomposed into an MNV, the decomposed variants will be marked with ",(0,l.kt)("inlineCode",{parentName:"p"},'"isDecomposedVariant":true'),"."),(0,l.kt)("p",{parentName:"div"},"Similarly, the recomposed variant will be shown as a new VCF position. This recomposed variant will be flagged with ",(0,l.kt)("inlineCode",{parentName:"p"},'"isRecomposedVariant":true'),"."))),(0,l.kt)("h3",{id:"transcripts"},"Transcripts"),(0,l.kt)("pre",null,(0,l.kt)("code",{parentName:"pre",className:"language-json"},'"transcripts":[\n   {\n      "transcript":"ENST00000445503.1",\n      "source":"Ensembl",\n      "bioType":"nonsense_mediated_decay",\n      "codons":"gGg/gAg",\n      "aminoAcids":"G/E",\n      "cdnaPos":"268",\n      "cdsPos":"116",\n      "exons":"1/9",\n      "introns":"1/8",\n      "proteinPos":"39",\n      "geneId":"ENSG00000116062",\n      "hgnc":"MSH6",\n      "consequence":[\n         "missense_variant",\n         "NMD_transcript_variant"\n      ],\n      "hgvsc":"ENST00000445503.1:c.116G>A",\n      "hgvsp":"ENSP00000405294.1:p.(Gly39Glu)",\n      "geneFusion":{\n         "exon":6,\n         "intron":5,\n         "fusions":[\n            {\n               "hgvsc":"ETV6{ENST00000396373.4}:c.1_1009+3402_RUNX1{ENST00000437180.1}:c.58+568_1443",\n               "exon":3,\n               "intron":2\n            },\n            {\n               "hgvsc":"ETV6{ENST00000396373.4}:c.1_1009+3402_RUNX1{ENST00000300305.3}:c.58+568_1443",\n               "exon":2,\n               "intron":1\n            }\n         ]\n      },\n      "isCanonical":true,\n      "polyPhenScore":0.95,\n      "polyPhenPrediction":"probably damaging",\n      "proteinId":"ENSP00000405294.1",\n      "siftScore":0.61,\n      "siftPrediction":"tolerated",\n      "completeOverlap":true\n   }\n]\n')),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,l.kt)("th",{parentName:"tr",align:"left"},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"transcript"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"transcript ID. e.g. ENST00000445503.1")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"source"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"RefSeq / Ensembl")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"bioType"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"descriptions of the ",(0,l.kt)("a",{parentName:"td",href:"https://uswest.ensembl.org/info/genome/genebuild/biotypes.html"},"biotypes from Ensembl"))),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"codons"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"aminoAcids"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"cdnaPos"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"cdsPos"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"exons"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"exons affected by the variant")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"introns"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"introns affected by the variant")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"proteinPos"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"geneId"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"gene ID. e.g. ENSG00000116062")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"hgnc"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"gene symbol. e.g. MSH6")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"consequence"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:"left"},(0,l.kt)("a",{parentName:"td",href:"http://www.sequenceontology.org/index.html"},"Sequence Ontology Consequences"))),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"hgvsc"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"HGVS coding nomenclature")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"hgvsp"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"HGVS protein nomenclature")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"geneFusion"),(0,l.kt)("td",{parentName:"tr",align:"center"},"object"),(0,l.kt)("td",{parentName:"tr",align:"left"},"see ",(0,l.kt)("a",{parentName:"td",href:"#gene-fusions"},"Gene Fusions entry below"))),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"isCanonical"),(0,l.kt)("td",{parentName:"tr",align:"center"},"bool"),(0,l.kt)("td",{parentName:"tr",align:"left"},"true when this is a canonical transcript")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"polyPhenScore"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"left"},"range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"polyPhenPrediction"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"see ",(0,l.kt)("a",{parentName:"td",href:"#polyphen"},"possible values below"))),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"proteinId"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"protein ID. E.g. ENSP00000405294.1")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"siftScore"),(0,l.kt)("td",{parentName:"tr",align:"center"},"float"),(0,l.kt)("td",{parentName:"tr",align:"left"},"range: 0 - 1.0")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"siftPrediction"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"see ",(0,l.kt)("a",{parentName:"td",href:"#sift"},"possible values below"))),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"completeOverlap"),(0,l.kt)("td",{parentName:"tr",align:"center"},"bool"),(0,l.kt)("td",{parentName:"tr",align:"left"},"true when this transcript is completely overlapped by the variant")))),(0,l.kt)("h4",{id:"polyphen"},"PolyPhen"),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},"probably damaging"),(0,l.kt)("li",{parentName:"ul"},"possibly damaging"),(0,l.kt)("li",{parentName:"ul"},"benign"),(0,l.kt)("li",{parentName:"ul"},"unknown")),(0,l.kt)("h4",{id:"sift"},"SIFT"),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},"tolerated"),(0,l.kt)("li",{parentName:"ul"},"deleterious"),(0,l.kt)("li",{parentName:"ul"},"tolerated - low confidence"),(0,l.kt)("li",{parentName:"ul"},"deleterious - low confidence")),(0,l.kt)("h4",{id:"gene-fusions"},"Gene Fusions"),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,l.kt)("th",{parentName:"tr",align:"left"},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"exon"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"actual exon where the breakpoint was located")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"intron"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"actual intron where the breakpoint was located")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"fusions"),(0,l.kt)("td",{parentName:"tr",align:"center"},"object array"),(0,l.kt)("td",{parentName:"tr",align:"left"},"see ",(0,l.kt)("a",{parentName:"td",href:"#fusion"},"Fusion entry below"))))),(0,l.kt)("h4",{id:"fusion"},"Fusion"),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,l.kt)("th",{parentName:"tr",align:"left"},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"exon"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"actual exon where the other breakpoint was located")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"intron"),(0,l.kt)("td",{parentName:"tr",align:"center"},"int"),(0,l.kt)("td",{parentName:"tr",align:"left"},"actual intron where the other breakpoint was located")),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"hgvsc"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:"left"},"HGVS coding nomenclature describing the two genes and the transcripts that are fused along with")))),(0,l.kt)("h3",{id:"regulatory-regions"},"Regulatory Regions"),(0,l.kt)("pre",null,(0,l.kt)("code",{parentName:"pre",className:"language-json"},'"regulatoryRegions":[ \n   { \n      "id":"ENSR00001542175",\n      "type":"promoter",\n      "consequence":[ \n         "regulatory_region_variant"\n      ]\n   }\n]\n')),(0,l.kt)("table",null,(0,l.kt)("thead",{parentName:"table"},(0,l.kt)("tr",{parentName:"thead"},(0,l.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,l.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,l.kt)("th",{parentName:"tr",align:null},"Notes"))),(0,l.kt)("tbody",{parentName:"table"},(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"id"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:null})),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"type"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string"),(0,l.kt)("td",{parentName:"tr",align:null},"see ",(0,l.kt)("a",{parentName:"td",href:"#regulatory-types"},"possible values below"))),(0,l.kt)("tr",{parentName:"tbody"},(0,l.kt)("td",{parentName:"tr",align:"left"},"consequence"),(0,l.kt)("td",{parentName:"tr",align:"center"},"string array"),(0,l.kt)("td",{parentName:"tr",align:null},"see ",(0,l.kt)("a",{parentName:"td",href:"#regulatory-consequences"},"possible values below"))))),(0,l.kt)("h4",{id:"regulatory-types"},"Regulatory Types"),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},"CTCF_binding_site"),(0,l.kt)("li",{parentName:"ul"},"enhancer"),(0,l.kt)("li",{parentName:"ul"},"open_chromatin_region"),(0,l.kt)("li",{parentName:"ul"},"promoter"),(0,l.kt)("li",{parentName:"ul"},"promoter_flanking_region"),(0,l.kt)("li",{parentName:"ul"},"TF_binding_site")),(0,l.kt)("h4",{id:"regulatory-consequences"},"Regulatory Consequences"),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},"regulatory_region_variant"),(0,l.kt)("li",{parentName:"ul"},"regulatory_region_ablation"),(0,l.kt)("li",{parentName:"ul"},"regulatory_region_amplification"),(0,l.kt)("li",{parentName:"ul"},"regulatory_region_truncation")),(0,l.kt)("h3",{id:"clinvar"},"ClinVar"),(0,l.kt)(i.default,{mdxType:"ClinVar"}),(0,l.kt)("h3",{id:"1000-genomes"},"1000 Genomes"),(0,l.kt)(d.default,{mdxType:"ThousandGenomesSmall"}),(0,l.kt)("h3",{id:"gnomad-genomes"},"gnomAD (genomes)"),(0,l.kt)(p.default,{mdxType:"GnomadGenomesSmall"}),(0,l.kt)("h3",{id:"gnomad-exomes"},"gnomAD (exomes)"),(0,l.kt)(m.default,{mdxType:"GnomadExomesSmall"}),(0,l.kt)("h3",{id:"dbsnp"},"dbSNP"),(0,l.kt)(o.default,{mdxType:"DbSNP"}))}h.isMDXComponent=!0},33658:function(t,e,n){e.Z=n.p+"assets/images/JSON-Layout-fc8e5c0cf4c8428981cd206fe9b6feac.svg"}}]);