"use strict";(self.webpackChunknirvana_documentation=self.webpackChunknirvana_documentation||[]).push([[9143,9836],{3905:(e,n,t)=>{t.d(n,{Zo:()=>d,kt:()=>m});var a=t(7294);function r(e,n,t){return n in e?Object.defineProperty(e,n,{value:t,enumerable:!0,configurable:!0,writable:!0}):e[n]=t,e}function o(e,n){var t=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);n&&(a=a.filter((function(n){return Object.getOwnPropertyDescriptor(e,n).enumerable}))),t.push.apply(t,a)}return t}function i(e){for(var n=1;n<arguments.length;n++){var t=null!=arguments[n]?arguments[n]:{};n%2?o(Object(t),!0).forEach((function(n){r(e,n,t[n])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(t)):o(Object(t)).forEach((function(n){Object.defineProperty(e,n,Object.getOwnPropertyDescriptor(t,n))}))}return e}function s(e,n){if(null==e)return{};var t,a,r=function(e,n){if(null==e)return{};var t,a,r={},o=Object.keys(e);for(a=0;a<o.length;a++)t=o[a],n.indexOf(t)>=0||(r[t]=e[t]);return r}(e,n);if(Object.getOwnPropertySymbols){var o=Object.getOwnPropertySymbols(e);for(a=0;a<o.length;a++)t=o[a],n.indexOf(t)>=0||Object.prototype.propertyIsEnumerable.call(e,t)&&(r[t]=e[t])}return r}var l=a.createContext({}),c=function(e){var n=a.useContext(l),t=n;return e&&(t="function"==typeof e?e(n):i(i({},n),e)),t},d=function(e){var n=c(e.components);return a.createElement(l.Provider,{value:n},e.children)},u={inlineCode:"code",wrapper:function(e){var n=e.children;return a.createElement(a.Fragment,{},n)}},p=a.forwardRef((function(e,n){var t=e.components,r=e.mdxType,o=e.originalType,l=e.parentName,d=s(e,["components","mdxType","originalType","parentName"]),p=c(t),m=r,h=p["".concat(l,".").concat(m)]||p[m]||u[m]||o;return t?a.createElement(h,i(i({ref:n},d),{},{components:t})):a.createElement(h,i({ref:n},d))}));function m(e,n){var t=arguments,r=n&&n.mdxType;if("string"==typeof e||r){var o=t.length,i=new Array(o);i[0]=p;var s={};for(var l in n)hasOwnProperty.call(n,l)&&(s[l]=n[l]);s.originalType=e,s.mdxType="string"==typeof e?e:r,i[1]=s;for(var c=2;c<o;c++)i[c]=t[c];return a.createElement.apply(null,i)}return a.createElement.apply(null,t)}p.displayName="MDXCreateElement"},9679:(e,n,t)=>{t.r(n),t.d(n,{contentTitle:()=>i,default:()=>d,frontMatter:()=>o,metadata:()=>s,toc:()=>l});var a=t(7462),r=(t(7294),t(3905));const o={},i=void 0,s={unversionedId:"data-sources/amino-acid-conservation-json",id:"version-3.16/data-sources/amino-acid-conservation-json",title:"amino-acid-conservation-json",description:"| Field                 | Type         | Notes                     |",source:"@site/versioned_docs/version-3.16/data-sources/amino-acid-conservation-json.md",sourceDirName:"data-sources",slug:"/data-sources/amino-acid-conservation-json",permalink:"/NirvanaDocumentation/3.16/data-sources/amino-acid-conservation-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/versioned_docs/version-3.16/data-sources/amino-acid-conservation-json.md",tags:[],version:"3.16",frontMatter:{}},l=[],c={toc:l};function d(e){let{components:n,...t}=e;return(0,r.kt)("wrapper",(0,a.Z)({},c,t,{components:n,mdxType:"MDXLayout"}),(0,r.kt)("pre",null,(0,r.kt)("code",{parentName:"pre",className:"language-json"},'"aminoAcidConservation": {\n    "scores": [0.34]\n}  \n')),(0,r.kt)("table",null,(0,r.kt)("thead",{parentName:"table"},(0,r.kt)("tr",{parentName:"thead"},(0,r.kt)("th",{parentName:"tr",align:"left"},"Field"),(0,r.kt)("th",{parentName:"tr",align:"center"},"Type"),(0,r.kt)("th",{parentName:"tr",align:"left"},"Notes"))),(0,r.kt)("tbody",{parentName:"table"},(0,r.kt)("tr",{parentName:"tbody"},(0,r.kt)("td",{parentName:"tr",align:"left"},"aminoAcidConservation"),(0,r.kt)("td",{parentName:"tr",align:"center"},"object"),(0,r.kt)("td",{parentName:"tr",align:"left"})),(0,r.kt)("tr",{parentName:"tbody"},(0,r.kt)("td",{parentName:"tr",align:"left"},"scores"),(0,r.kt)("td",{parentName:"tr",align:"center"},"object array of doubles"),(0,r.kt)("td",{parentName:"tr",align:"left"},"percent conserved with respect to human amino acid residue. Range: 0.01 - 1.00")))))}d.isMDXComponent=!0},1867:(e,n,t)=>{t.r(n),t.d(n,{contentTitle:()=>s,default:()=>u,frontMatter:()=>i,metadata:()=>l,toc:()=>c});var a=t(7462),r=(t(7294),t(3905)),o=t(9679);const i={title:"Amino Acid Conservation"},s=void 0,l={unversionedId:"data-sources/amino-acid-conservation",id:"version-3.16/data-sources/amino-acid-conservation",title:"Amino Acid Conservation",description:"Overview",source:"@site/versioned_docs/version-3.16/data-sources/amino-acid-conservation.mdx",sourceDirName:"data-sources",slug:"/data-sources/amino-acid-conservation",permalink:"/NirvanaDocumentation/3.16/data-sources/amino-acid-conservation",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/versioned_docs/version-3.16/data-sources/amino-acid-conservation.mdx",tags:[],version:"3.16",frontMatter:{title:"Amino Acid Conservation"},sidebar:"version-3.16/docs",previous:{title:"1000 Genomes",permalink:"/NirvanaDocumentation/3.16/data-sources/1000Genomes"},next:{title:"ClinGen",permalink:"/NirvanaDocumentation/3.16/data-sources/clingen"}},c=[{value:"Overview",id:"overview",children:[],level:2},{value:"FASTA File",id:"fasta-file",children:[],level:2},{value:"Parsing FASTA",id:"parsing-fasta",children:[],level:2},{value:"Assigning scores to Nirvana transcripts",id:"assigning-scores-to-nirvana-transcripts",children:[{value:"GRCh37",id:"grch37",children:[],level:3},{value:"GRCh38",id:"grch38",children:[],level:3}],level:2},{value:"Download URL",id:"download-url",children:[],level:2},{value:"JSON Output",id:"json-output",children:[],level:2}],d={toc:c};function u(e){let{components:n,...t}=e;return(0,r.kt)("wrapper",(0,a.Z)({},d,t,{components:n,mdxType:"MDXLayout"}),(0,r.kt)("h2",{id:"overview"},"Overview"),(0,r.kt)("p",null,"Amino acid conservation scores are obtained from multiple alignments of vertebrate exomes to the human ones. The score indicate the frequency with which a particular AA is observed in Humans."),(0,r.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,r.kt)("div",{parentName:"div",className:"admonition-heading"},(0,r.kt)("h5",{parentName:"div"},(0,r.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,r.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,r.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"Publication")),(0,r.kt)("div",{parentName:"div",className:"admonition-content"},(0,r.kt)("p",{parentName:"div"},"Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K, Clawson H, Spieth J, Hillier LW, Richards S, et al. Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. ",(0,r.kt)("strong",{parentName:"p"},"Genome Res. 2005")," Aug;15(8):1034-50.  (",(0,r.kt)("a",{parentName:"p",href:"http://www.genome.org/cgi/doi/10.1101/gr.3715005"},"http://www.genome.org/cgi/doi/10.1101/gr.3715005"),")"))),(0,r.kt)("h2",{id:"fasta-file"},"FASTA File"),(0,r.kt)("p",null,"The exon alignments are provided in FASTA files as follows:"),(0,r.kt)("pre",null,(0,r.kt)("code",{parentName:"pre",className:"language-scss"},">ENST00000641515.2_hg38_1_2 3 0 0 chr1:65565-65573+\nMKK\n>ENST00000641515.2_panTro4_1_2 3 0 0 chrUn_GL393541:146907-146915+\nMKK\n>ENST00000641515.2_gorGor3_1_2 3 0 0\n---\n>ENST00000641515.2_ponAbe2_1_2 3 0 0 chr15:99141417-99141425-\nMKK\n>ENST00000641515.2_hg38_2_2 324 0 0 chr1:69037-70008+\nVTAEAISWNESTSETNNSMVTEFIFLGLSDSQELQTFLFMLFFVFYGGIVFGNLLIVITVVSDSHLHSPMYFLLANLSLIDLSLSSVTAPKMITDFFSQRKVISFKGCLVQIFLLHFFGGSEMVILIAMGFDRYIAICKPLHYTTIMCGNACVGIMAVTWGIGFLHSVSQLAFAVHLLFCGPNEVDSFYCDLPRVIKLACTDTYRLDIMVIANSGVLTVCSFVLLIISYTIILMTIQHRPLDKSSKALSTLTAHITVVLLFFGPCVFIYAWPFPIKSLDKFLAVFYSVITPLLNPIIYTLRNKDMKTAIRQLRKWDAHSSVKFZ\n>ENST00000641515.2_panTro4_2_2 324 0 0 chrUn_GL393541:151333-152303+\n")),(0,r.kt)("h2",{id:"parsing-fasta"},"Parsing FASTA"),(0,r.kt)("p",null,"For each Ensembl transcript, we will need to aggregate all the exons together for each of the 100 species. From there, we should get a full alignment that can be used to determine conservation. For example, for ENST00000641515.2 we have:"),(0,r.kt)("pre",null,(0,r.kt)("code",{parentName:"pre",className:"language-scss"},"Human (hg38) MKKVTAEAISWNESTSETNNSMVTEFIFLGLSDSQELQTFLFMLFFVFYGGIVFGNLLIVITVVSDSHLHSPMYFLLANLSLIDLSLSSVTAPKMITDFFSQRKVISFKGCLVQIFLL\nChimp        MKKVTAEAISWNESTSETNNSMVTEFIFLGLSDSQELQTFL-MLFFVFYGGIVFGNLLIVRIVVSDSHLHSPMYFLLANLSLIDLSLCSVTAPKMITDFFSQRKVISFKGCLVQIFLL\nGorilla      ----------------------------------------------------------------------------------------------------------------------\nOrangutan    MKKVTAEAISWNESTSKTNNSVVTEFIFLGLSDSQELQTFLFMLFFVFYGGIVFGNLLIVIIVVSDSHLHSPMYFLLANLSLIDLSLSSVTAPKMITDFFSQRKVISFKGCLVQIFLL\nGibbon       ----------------------------------------------------------------------------------------------------------------------\nRhesus       MKKVTEAAISWNESTSETNNSIVTEFIFLGLSDSQELQIFLFVLFLVFYGGIVFGNLLIVITVVSDSHLHSPMYLLLANLSVVDLSLSSVTAPKMITDFFSQRKAISFKGCLVQIFLL\nMacaque      MKKVTEAAISWNESTSETNNSIVTEFIFLGLSDSQELQIFLFVLFLVFYGGIVFGNLLIVITVVSDSHLHSPMYLLLANLSVIDLSLSSVTAPKMITDFFSQRKAISFKGCLVQIFLL\n")),(0,r.kt)("p",null,"If we look at position 6, we see that humans have an Alanine (A) residue. This residue is shared by Chimp and Orangutan. However, Rhesus and Macaque have a Glutamic acid (E) residue at that position. Moreover, Gorilla and Gibbon don't even have data for that transcript.\nFor position 6, we would say that we have 43% conservation (3/7) since three organisms share the same residue as humans."),(0,r.kt)("h2",{id:"assigning-scores-to-nirvana-transcripts"},"Assigning scores to Nirvana transcripts"),(0,r.kt)("p",null,"The source FASTA file comes with Ensembl/UCSC transcript ids of the transcripts used for alignments. The Nirvana cache has RefSeq and Ensembl transcripts and our first attempt was to map the given Ensembl/UCSC ids to their equivalent RefSeq/Ensembl ids. This attempt was unsuccessful since UCSC Table Browser provided mapping without version numbers. So we proceeded as follows:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},"Take proteins which have a unique mapping (and hence one set of conservation scores). For ones that mapped to both ChrX and ChrY, we accepted the one from ChrX."),(0,r.kt)("li",{parentName:"ul"},"A Nirvana transcript having an exact peptide sequence match with a uniquely aligned protein is assigned the corresponding conservation scores.")),(0,r.kt)("p",null,"Unfortunately this left us with a very small number of transcripts having conservation scores."),(0,r.kt)("h3",{id:"grch37"},"GRCh37"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},"Source FASTA contained 41957 protein alignments."),(0,r.kt)("li",{parentName:"ul"},"38165 proteins had unique scores."),(0,r.kt)("li",{parentName:"ul"},"88 aligned proteins existed in Nirvana cache."),(0,r.kt)("li",{parentName:"ul"},"118 transcripts had conservation scores.")),(0,r.kt)("h3",{id:"grch38"},"GRCh38"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},"Source FASTA contained 110024 protein alignments."),(0,r.kt)("li",{parentName:"ul"},"88961 proteins had unique scores."),(0,r.kt)("li",{parentName:"ul"},"11688 aligned proteins existed in Nirvana cache."),(0,r.kt)("li",{parentName:"ul"},"12098 transcripts had conservation scores.")),(0,r.kt)("h2",{id:"download-url"},"Download URL"),(0,r.kt)("p",null,"GRCh37: ",(0,r.kt)("a",{parentName:"p",href:"http://hgdownload.soe.ucsc.edu/goldenPath/hg19/multiz100way/alignments/knownGene.exonAA.fa.gz"},"http://hgdownload.soe.ucsc.edu/goldenPath/hg19/multiz100way/alignments/knownGene.exonAA.fa.gz")),(0,r.kt)("p",null,"GRCh38: ",(0,r.kt)("a",{parentName:"p",href:"http://hgdownload.soe.ucsc.edu/goldenPath/hg38/multiz100way/alignments/knownGene.exonAA.fa.gz"},"http://hgdownload.soe.ucsc.edu/goldenPath/hg38/multiz100way/alignments/knownGene.exonAA.fa.gz")),(0,r.kt)("h2",{id:"json-output"},"JSON Output"),(0,r.kt)("p",null,"Conservation scores are reported in the transcript section. One score is reported for each alt allele"),(0,r.kt)(o.default,{mdxType:"JSON"}))}u.isMDXComponent=!0}}]);