"use strict";(self.webpackChunknirvana_documentation=self.webpackChunknirvana_documentation||[]).push([[6116],{24807:e=>{e.exports=JSON.parse('{"pluginId":"default","version":"3.2.5","label":"3.2.5","banner":"unmaintained","badge":true,"className":"docs-version-3.2.5","isLast":false,"docsSidebars":{"version-3.2.5/docs":[{"collapsed":true,"type":"category","label":"Introduction","items":[{"type":"link","label":"Introduction","href":"/NirvanaDocumentation/3.2.5/","docId":"introduction/introduction"},{"type":"link","label":"Dependencies","href":"/NirvanaDocumentation/3.2.5/introduction/dependencies","docId":"introduction/dependencies"},{"type":"link","label":"Getting Started","href":"/NirvanaDocumentation/3.2.5/introduction/getting-started","docId":"introduction/getting-started"}],"collapsible":true},{"collapsed":true,"type":"category","label":"Data Sources","items":[{"type":"link","label":"1000 Genomes","href":"/NirvanaDocumentation/3.2.5/data-sources/1000Genomes","docId":"data-sources/1000Genomes"},{"type":"link","label":"ClinVar","href":"/NirvanaDocumentation/3.2.5/data-sources/clinvar","docId":"data-sources/clinvar"},{"type":"link","label":"dbSNP","href":"/NirvanaDocumentation/3.2.5/data-sources/dbsnp","docId":"data-sources/dbsnp"},{"type":"link","label":"gnomAD","href":"/NirvanaDocumentation/3.2.5/data-sources/gnomad","docId":"data-sources/gnomad"}],"collapsible":true},{"collapsed":true,"type":"category","label":"File Formats","items":[{"type":"link","label":"Nirvana JSON File Format","href":"/NirvanaDocumentation/3.2.5/file-formats/nirvana-json-file-format","docId":"file-formats/nirvana-json-file-format"}],"collapsible":true},{"collapsed":true,"type":"category","label":"Core Functionality","items":[{"type":"link","label":"Variant IDs","href":"/NirvanaDocumentation/3.2.5/core-functionality/variant-ids","docId":"core-functionality/variant-ids"},{"type":"link","label":"Gene Fusion Detection","href":"/NirvanaDocumentation/3.2.5/core-functionality/gene-fusions","docId":"core-functionality/gene-fusions"}],"collapsible":true}]},"docs":{"core-functionality/gene-fusions":{"id":"core-functionality/gene-fusions","title":"Gene Fusion Detection","description":"Overview","sidebar":"version-3.2.5/docs"},"core-functionality/variant-ids":{"id":"core-functionality/variant-ids","title":"Variant IDs","description":"Overview","sidebar":"version-3.2.5/docs"},"data-sources/1000Genomes":{"id":"data-sources/1000Genomes","title":"1000 Genomes","description":"Overview","sidebar":"version-3.2.5/docs"},"data-sources/1000Genomes-snv-json":{"id":"data-sources/1000Genomes-snv-json","title":"1000Genomes-snv-json","description":"| Field            | Type         | Notes                     |"},"data-sources/1000Genomes-sv-json":{"id":"data-sources/1000Genomes-sv-json","title":"1000Genomes-sv-json","description":"| Field             | Type           | Notes                                                                                                                           |"},"data-sources/clinvar":{"id":"data-sources/clinvar","title":"ClinVar","description":"Overview","sidebar":"version-3.2.5/docs"},"data-sources/clinvar-json":{"id":"data-sources/clinvar-json","title":"clinvar-json","description":"| Field            | Type         | Notes                     |"},"data-sources/dbsnp":{"id":"data-sources/dbsnp","title":"dbSNP","description":"Overview","sidebar":"version-3.2.5/docs"},"data-sources/dbsnp-json":{"id":"data-sources/dbsnp-json","title":"dbsnp-json","description":"| Field            | Type         | Notes                     |"},"data-sources/gnomad":{"id":"data-sources/gnomad","title":"gnomAD","description":"Overview","sidebar":"version-3.2.5/docs"},"data-sources/gnomad-exomes-small-variants-json":{"id":"data-sources/gnomad-exomes-small-variants-json","title":"gnomad-exomes-small-variants-json","description":"| Field               | Type   | Notes                                                                                            |"},"data-sources/gnomad-genomes-small-variants-json":{"id":"data-sources/gnomad-genomes-small-variants-json","title":"gnomad-genomes-small-variants-json","description":"| Field               | Type   | Notes                                                                                            |"},"file-formats/nirvana-json-file-format":{"id":"file-formats/nirvana-json-file-format","title":"Nirvana JSON File Format","description":"Overview","sidebar":"version-3.2.5/docs"},"introduction/dependencies":{"id":"introduction/dependencies","title":"Dependencies","description":"All of the following dependencies have been included in this repository.","sidebar":"version-3.2.5/docs"},"introduction/getting-started":{"id":"introduction/getting-started","title":"Getting Started","description":"Nirvana is written in C# using .NET Core (an amazing runtime environment that currently runs on Windows, Linux, Mac OS X, and in Docker images). Once .NET Core has been downloaded, all you need to do is grab the source, compile it, and grab the data files.","sidebar":"version-3.2.5/docs"},"introduction/introduction":{"id":"introduction/introduction","title":"Introduction","description":"Clinical-grade variant annotation","sidebar":"version-3.2.5/docs"}}}')}}]);