---
title: ClinVar
---

import JSON from './clinvar-json.md'

## RCV Files

### Example
```xml
<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<ReleaseSet Dated="2016-07-04" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Type="full"
            xsi:noNamespaceSchemaLocation="http://ftp.ncbi.nlm.nih.gov/pub/clinvar/xsd_public/clinvar_public_1.35.xsd">

    <ClinVarSet ID="47064274">
        <RecordStatus>current</RecordStatus>
        <Title>NM_000235.4(LIPA):c.894+1G&gt;A AND Wolman disease</Title>
        <ReferenceClinVarAssertion DateCreated="2012-08-13" DateLastUpdated="2019-11-02" ID="57604">
            <ClinVarAccession Acc="RCV000000101" Version="3" Type="RCV" DateUpdated="2019-11-02"/>
            <RecordStatus>current</RecordStatus>
            <ClinicalSignificance DateLastEvaluated="1996-04-01">
                <ReviewStatus>no assertion criteria provided</ReviewStatus>
                <Description>Pathogenic</Description>
            </ClinicalSignificance>
            <Assertion Type="variation to disease"/>
            <ObservedIn>
                <Sample>
                    <Origin>germline</Origin>
                    <Species TaxonomyId="9606">human</Species>
                    <AffectedStatus>not provided</AffectedStatus>
                </Sample>
                <Method>
                    <MethodType>literature only</MethodType>
                </Method>
                <ObservedData ID="47897969">
                    <Attribute Type="Description">In 2 sibs with Wolman disease (278000) from a consanguineous family,
                        Aslanidis et al. (1996) detected homozygosity for a G-to-A mutation at position +1 of the splice
                        donor site following exon 8 of the LIPA gene. Both children died within the first year of life.
                        The parents, who were heterozygous for the mutation, had reduced enzymatic activity, while no
                        enzymatic activity was detectable in fibroblasts from the affected children. Although the same
                        donor splice site is involved as in the mutation reported in CESD (934G-A, 613497.0002), the
                        nucleotide at position +1 was changed in the Wolman disease mutation while the nucleotide at
                        position -1 was changed in the CESD mutation. Both mutations result in deletion of the same 24
                        amino acids (exon 8), but the effects are dramatically different: the -1 mutation allowed some
                        correct splicing (3% of total LIPA RNA), but the +1 splice site mutation, which affects one of
                        the invariable nucleotides of the splice consensus sequences, permits no correct splicing.
                        Aslanidis et al. (1996) suggested that the residual activity in CESD patients compared to Wolman
                        patients may result either from a partially active enzyme with the internal deletion of 24 amino
                        acids (skipping of exon 8) or from the production of low amounts of the full size of the protein
                        due to inefficient exon exclusion from the mutated allele.
                    </Attribute>
                    <Citation Type="general">
                        <ID Source="PubMed">8617513</ID>
                    </Citation>
                </ObservedData>
            </ObservedIn>
            <MeasureSet Type="Variant" ID="81" Acc="VCV000000081" Version="1">
                <Measure Type="single nucleotide variant" ID="15120">
                    <Name>
                        <ElementValue Type="Preferred">NM_000235.4(LIPA):c.894+1G&gt;A</ElementValue>
                    </Name>
                    <AttributeSet>
                        <Attribute Accession="NM_001288979" Version="1" Change="c.546+1G&gt;A"
                                   Type="HGVS, coding, RefSeq">NM_001288979.1:c.546+1G&gt;A
                        </Attribute>
                    </AttributeSet>
                    <AttributeSet>
                        <Attribute Accession="NM_000235" Version="4" Change="c.894+1G&gt;A" Type="HGVS, coding, RefSeq">
                            NM_000235.4:c.894+1G&gt;A
                        </Attribute>
                    </AttributeSet>
                    <AttributeSet>
                        <Attribute Accession="NM_001127605" Version="3" Change="c.894+1G&gt;A"
                                   Type="HGVS, coding, RefSeq">NM_001127605.3:c.894+1G&gt;A
                        </Attribute>
                    </AttributeSet>
                    <AttributeSet>
                        <Attribute Accession="NG_008194" Version="1" Change="g.34394G&gt;A"
                                   Type="HGVS, genomic, RefSeqGene">NG_008194.1:g.34394G&gt;A
                        </Attribute>
                    </AttributeSet>
                    <AttributeSet>
                        <Attribute Accession="NC_000010" Version="11" Change="g.89222510C&gt;T"
                                   Type="HGVS, genomic, top level" integerValue="38">NC_000010.11:g.89222510C&gt;T
                        </Attribute>
                    </AttributeSet>
                    <AttributeSet>
                        <Attribute Accession="NC_000010" Version="10" Change="g.90982267C&gt;T"
                                   Type="HGVS, genomic, top level, previous" integerValue="37">NC_000010.10:g.90982267C&gt;T
                        </Attribute>
                    </AttributeSet>
                    <AttributeSet>
                        <Attribute Accession="NM_000235" Version="3" Change="c.894+1G&gt;A" Type="HGVS, previous">
                            NM_000235.3:c.894+1G&gt;A
                        </Attribute>
                    </AttributeSet>
                    <AttributeSet>
                        <Attribute Type="MolecularConsequence">splice donor variant</Attribute>
                        <XRef ID="SO:0001575" DB="Sequence Ontology"/>
                        <XRef ID="NM_000235.4:c.894+1G&gt;A" DB="RefSeq"/>
                    </AttributeSet>
                    <AttributeSet>
                        <Attribute Type="MolecularConsequence">splice donor variant</Attribute>
                        <XRef ID="SO:0001575" DB="Sequence Ontology"/>
                        <XRef ID="NM_001127605.3:c.894+1G&gt;A" DB="RefSeq"/>
                    </AttributeSet>
                    <AttributeSet>
                        <Attribute Type="MolecularConsequence">splice donor variant</Attribute>
                        <XRef ID="SO:0001575" DB="Sequence Ontology"/>
                        <XRef ID="NM_001288979.1:c.546+1G&gt;A" DB="RefSeq"/>
                    </AttributeSet>
                    <AttributeSet>
                        <Attribute Type="nucleotide change">IVS8, G-A, +1</Attribute>
                    </AttributeSet>
                    <CytogeneticLocation>10q23.31</CytogeneticLocation>
                    <SequenceLocation Assembly="GRCh38" AssemblyAccessionVersion="GCF_000001405.38"
                                      AssemblyStatus="current" Chr="10" Accession="NC_000010.11" start="89222510"
                                      stop="89222510" display_start="89222510" display_stop="89222510" variantLength="1"
                                      positionVCF="89222510" referenceAlleleVCF="C" alternateAlleleVCF="T"/>
                    <SequenceLocation Assembly="GRCh37" AssemblyAccessionVersion="GCF_000001405.25"
                                      AssemblyStatus="previous" Chr="10" Accession="NC_000010.10" start="90982267"
                                      stop="90982267" display_start="90982267" display_stop="90982267" variantLength="1"
                                      positionVCF="90982267" referenceAlleleVCF="C" alternateAlleleVCF="T"/>
                    <MeasureRelationship Type="within single gene">
                        <Name>
                            <ElementValue Type="Preferred">lipase A, lysosomal acid type</ElementValue>
                        </Name>
                        <Symbol>
                            <ElementValue Type="Preferred">LIPA</ElementValue>
                        </Symbol>
                        <SequenceLocation Assembly="GRCh38" AssemblyAccessionVersion="GCF_000001405.38"
                                          AssemblyStatus="current" Chr="10" Accession="NC_000010.11" start="89213569"
                                          stop="89252039" display_start="89213569" display_stop="89252039" Strand="-"/>
                        <SequenceLocation Assembly="GRCh37" AssemblyAccessionVersion="GCF_000001405.25"
                                          AssemblyStatus="previous" Chr="10" Accession="NC_000010.10" start="90973325"
                                          stop="91011659" display_start="90973325" display_stop="91011659"
                                          variantLength="38335" Strand="-"/>
                        <XRef ID="3988" DB="Gene"/>
                        <XRef Type="MIM" ID="613497" DB="OMIM"/>
                        <XRef ID="HGNC:6617" DB="HGNC"/>
                    </MeasureRelationship>
                    <XRef Type="Allelic variant" ID="613497.0005" DB="OMIM"/>
                    <XRef Type="rs" ID="1564751995" DB="dbSNP"/>
                    <Comment DataSource="ClinGen" Type="public">ClinGen staff contributed the HGVS expression for this
                        variant.
                    </Comment>
                </Measure>
                <Name>
                    <ElementValue Type="Preferred">NM_000235.4(LIPA):c.894+1G&gt;A</ElementValue>
                </Name>
            </MeasureSet>
            <TraitSet Type="Disease" ID="40">
                <Trait ID="9048" Type="Disease">
                    <Name>
                        <ElementValue Type="Preferred">Wolman disease</ElementValue>
                        <XRef ID="7899" DB="Office of Rare Diseases"/>
                    </Name>
                    <XRef ID="CN438428" DB="MedGen"/>
                </Trait>
            </TraitSet>
        </ReferenceClinVarAssertion>
        <ClinVarAssertion ID="20244">
            <ClinVarSubmissionID localKey="613497.0005_WOLMAN DISEASE" submitter="OMIM" submitterDate="2017-12-22"
                                 title="LIPA, IVS8, G-A, +1_WOLMAN DISEASE"/>
            <ClinVarAccession Acc="SCV000020244" Version="2" Type="SCV" OrgID="3" OrganizationCategory="resource"
                              OrgType="primary" DateUpdated="2019-03-31"/>
            <RecordStatus>current</RecordStatus>
            <ClinicalSignificance DateLastEvaluated="1996-04-01">
                <ReviewStatus>no assertion criteria provided</ReviewStatus>
                <Description>Pathogenic</Description>
            </ClinicalSignificance>
            <Assertion Type="variation to disease"/>
            <ExternalID DB="OMIM" ID="613497.0005" Type="Allelic variant"/>
            <ObservedIn>
                <Sample>
                    <Origin>germline</Origin>
                    <Species>human</Species>
                    <AffectedStatus>not provided</AffectedStatus>
                </Sample>
                <Method>
                    <MethodType>literature only</MethodType>
                </Method>
                <ObservedData>
                    <Attribute Type="Description">In 2 sibs with Wolman disease (278000) from a consanguineous family,
                        Aslanidis et al. (1996) detected homozygosity for a G-to-A mutation at position +1 of the splice
                        donor site following exon 8 of the LIPA gene. Both children died within the first year of life.
                        The parents, who were heterozygous for the mutation, had reduced enzymatic activity, while no
                        enzymatic activity was detectable in fibroblasts from the affected children. Although the same
                        donor splice site is involved as in the mutation reported in CESD (934G-A, 613497.0002), the
                        nucleotide at position +1 was changed in the Wolman disease mutation while the nucleotide at
                        position -1 was changed in the CESD mutation. Both mutations result in deletion of the same 24
                        amino acids (exon 8), but the effects are dramatically different: the -1 mutation allowed some
                        correct splicing (3% of total LIPA RNA), but the +1 splice site mutation, which affects one of
                        the invariable nucleotides of the splice consensus sequences, permits no correct splicing.
                        Aslanidis et al. (1996) suggested that the residual activity in CESD patients compared to Wolman
                        patients may result either from a partially active enzyme with the internal deletion of 24 amino
                        acids (skipping of exon 8) or from the production of low amounts of the full size of the protein
                        due to inefficient exon exclusion from the mutated allele.
                    </Attribute>
                    <Citation>
                        <ID Source="PubMed">8617513</ID>
                    </Citation>
                    <XRef DB="OMIM" ID="278000" Type="MIM"/>
                </ObservedData>
            </ObservedIn>
            <MeasureSet Type="Variant">
                <Measure Type="Variation">
                    <Name>
                        <ElementValue Type="Preferred">LIPA, IVS8, G-A, +1</ElementValue>
                    </Name>
                    <AttributeSet>
                        <Attribute Type="NonHGVS">IVS8, G-A, +1</Attribute>
                    </AttributeSet>
                    <MeasureRelationship Type="variant in gene">
                        <Symbol>
                            <ElementValue Type="Preferred">LIPA</ElementValue>
                        </Symbol>
                    </MeasureRelationship>
                    <XRef DB="OMIM" ID="613497.0005" Type="Allelic variant"/>
                </Measure>
            </MeasureSet>
            <TraitSet Type="Disease">
                <Trait Type="Disease">
                    <Name>
                        <ElementValue Type="Preferred">WOLMAN DISEASE</ElementValue>
                    </Name>
                </Trait>
            </TraitSet>
        </ClinVarAssertion>
    </ClinVarSet>

</ReleaseSet>
```
### Parsing

In the following section, we discuss which field of the XML was used to extract information that is presented in the JSON output.

**ID**

```xml {3}
<ClinVarSet>
   <ReferenceClinVarAssertion>    
       <ClinVarAccession Acc="RCV000000001" Version="2">
</ClinVarSet>
```
The Acc and Version fields are merged to form the ID (RCV000000001.2)

**LastUpdatedDate**

```xml {2}
<ClinVarSet>
   <ReferenceClinVarAssertion DateCreated="2012-08-13" DateLastUpdated="2016-02-17" ID="57604" >
</ClinVarSet>
```

**Significance**

```xml {5}
<ClinVarSet>
   <ReferenceClinVarAssertion>
       <ClinicalSignificance DateLastEvaluated="1996-04-01">        
           <ReviewStatus>no assertion criteria provided</ReviewStatus>              
           <Description>Pathogenic</Description>            
       </ClinicalSignificance>
</ClinVarSet>
```

**ReviewStatus**

```xml {4}
<ClinVarSet>
   <ReferenceClinVarAssertion>    
       <ClinicalSignificance DateLastEvaluated="1996-04-01">        
           <ReviewStatus>no assertion criteria provided</ReviewStatus>             
           <Description>Pathogenic</Description>            
       </ClinicalSignificance>
</ClinVarSet>
```

**Phenotypes**

```xml {2-8}
<ReferenceClinVarAssertion>
   <TraitSet Type="Disease" ID="62">
       <Trait Type="Disease">
            <Name>
               <ElementValue Type="Preferred">Joubert syndrome 9</ElementValue>
            </Name>
        </Trait>
   </TraitSet>
</ReferenceClinVarAssertion>
```
We only use the field with Type="Preferred". Multiple phenotypes may be reported

**Location and Variant Id**

```xml {3,5-12}
<ReferenceClinVarAssertion>
<GenotypeSet Type="CompoundHeterozygote" ID="424709">
   <MeasureSet Type="Variant" ID="81">
       <Measure Type="single nucleotide variant" ID="15120">
        <SequenceLocation Assembly="GRCh38" AssemblyAccessionVersion="GCF_000001405.38"  
          AssemblyStatus="current" Chr="10" Accession="NC_000010.11" start="89222510"  
          stop="89222510" display_start="89222510" display_stop="89222510" variantLength="1"  
          positionVCF="89222510" referenceAlleleVCF="C" alternateAlleleVCF="T"/>  
        <SequenceLocation Assembly="GRCh37" AssemblyAccessionVersion="GCF_000001405.25"  
          AssemblyStatus="previous" Chr="10" Accession="NC_000010.10" start="90982267"  
          stop="90982267" display_start="90982267" display_stop="90982267" variantLength="1"  
          positionVCF="90982267" referenceAlleleVCF="C" alternateAlleleVCF="T"/>
       </Measure>
   </MeasureSet>
 </GenotypeSet>
</ReferenceClinVarAssertion>
```

- The variant position is extracted from the fields for their respective assemblies.
- Updated records contain positionVCF, referenceAlleleVCF and alternateAlleleVCF fields and when present, we use them to create the variant.
- For older records, since "start' and "stop" fields are not always available, we use the "display_start" and "display_end" fields.
- If a required allele is not available, we extract it from the reference sequence.
- Only variants having a dbSNP id are extracted.
- Note that a ClinVar accession may have multiple variants associated with it (possible in different locations)
- VariantId is extracted from the MeasureSet attributes.

**MedGen, OMIM, Orphanet IDs**

```xml {4-7}
<ReferenceClinVarAssertion>
   <TraitSet Type="Disease" ID="175">
       <Trait ID="3036" Type="Disease">
           <XRef ID="C0086651" DB="MedGen"/>
           <XRef ID="309297" DB="Orphanet"/>
           <XRef ID="582" DB="Orphanet"/>
           <XRef Type="MIM" ID="253000" DB="OMIM"/>
       </Trait>
   </TraitSet>
</ReferenceClinVarAssertion>
```

**AlleleOrigins**

```xml {2}
<ClinVarAssertion>
     <Origin>germline</Origin>
</ClinVarAssertion>
```

We only extract all Allele Origins from Submissions (SCV) entries.

**PubMedIds**

```xml {4,10,16,21}
<ClinVarAssertion>
   <ClinicalSignificance DateLastEvaluated="1996-04-01">
       <Citation Type="general">
           <ID Source="PubMed">12114475</ID>
       </Citation>
   </ClinicalSignificance>
   <AttributeSet>
       <Attribute Type="AssertionMethod">LMM Criteria</Attribute>
       <Citation>
            <ID Source="PubMed">24033266</ID>
       </Citation>
   </AttributeSet>
   <ObservedIn>
       <ObservedData ID="9727445">
           <Citation Type="general">
               <ID Source="PubMed">9113933</ID>
           </Citation>
       </ObservedData>
   </ObservedIn>
   <Citation Type="general">
       <ID Source="PubMed">23757202</ID>
   </Citation>
</ClinVarAssertion>
```

We only extract all Pubmed Ids from Submissions (SCV) entries.

## VCV Files
### Example
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ClinVarVariationRelease xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://ftp.ncbi.nlm.nih.gov/pub/clinvar/xsd_public/clinvar_variation/variation_archive_1.4.xsd" ReleaseDate="2019-12-31">
<VariationArchive VariationID="431749" VariationName="GRCh37/hg19 1p36.31(chr1:6051187-6158763)" VariationType="copy number gain" DateCreated="2017-08-12" DateLastUpdated="2019-09-10" Accession="VCV000431749" Version="1" RecordType="included" NumberOfSubmissions="0" NumberOfSubmitters="0">
  <RecordStatus>current</RecordStatus>
  <Species>Homo sapiens</Species>
  <IncludedRecord>
    <SimpleAllele AlleleID="425239" VariationID="431749">
      <GeneList>
        <Gene Symbol="KCNAB2" FullName="potassium voltage-gated channel subfamily A regulatory beta subunit 2" GeneID="8514" HGNC_ID="HGNC:6229" Source="calculated" RelationshipType="genes overlapped by variant">
          <Location>
            <CytogeneticLocation>1p36.31</CytogeneticLocation>
            <SequenceLocation Assembly="GRCh38" AssemblyAccessionVersion="GCF_000001405.38" AssemblyStatus="current" Chr="1" Accession="NC_000001.11" start="5992639" stop="6101186" display_start="5992639" display_stop="6101186" Strand="+"/>
            <SequenceLocation Assembly="GRCh37" AssemblyAccessionVersion="GCF_000001405.25" AssemblyStatus="previous" Chr="1" Accession="NC_000001.10" start="6052357" stop="6161252" display_start="6052357" display_stop="6161252" Strand="+"/>
          </Location>
          <OMIM>601142</OMIM>
        </Gene>
        <Gene Symbol="NPHP4" FullName="nephrocystin 4" GeneID="261734" HGNC_ID="HGNC:19104" Source="calculated" RelationshipType="genes overlapped by variant">
          <Location>
            <CytogeneticLocation>1p36.31</CytogeneticLocation>
            <SequenceLocation Assembly="GRCh38" AssemblyAccessionVersion="GCF_000001405.38" AssemblyStatus="current" Chr="1" Accession="NC_000001.11" start="5862810" stop="5992425" display_start="5862810" display_stop="5992425" Strand="-"/>
            <SequenceLocation Assembly="GRCh37" AssemblyAccessionVersion="GCF_000001405.25" AssemblyStatus="previous" Chr="1" Accession="NC_000001.10" start="5922869" stop="6052532" display_start="5922869" display_stop="6052532" Strand="-"/>
          </Location>
          <OMIM>607215</OMIM>
        </Gene>
      </GeneList>
      <Name>GRCh37/hg19 1p36.31(chr1:6051187-6158763)</Name>
      <VariantType>copy number gain</VariantType>
      <Location>
        <CytogeneticLocation>1p36.31</CytogeneticLocation>
        <SequenceLocation Assembly="GRCh37" AssemblyAccessionVersion="GCF_000001405.25" forDisplay="true" AssemblyStatus="previous" Chr="1" Accession="NC_000001.10" start="6051187" stop="6158763" display_start="6051187" display_stop="6158763"/>      </Location>
      <Interpretations>
        <Interpretation NumberOfSubmissions="0" NumberOfSubmitters="0" Type="Clinical significance">
          <Description>no interpretation for the single variant</Description>
        </Interpretation>
      </Interpretations>
      <XRefList>
        <XRef Type="Interpreted" ID="431733" DB="ClinVar"/>
      </XRefList>
    </SimpleAllele>
    <ReviewStatus>no interpretation for the single variant</ReviewStatus>
    <Interpretations>
      <Interpretation NumberOfSubmissions="0" NumberOfSubmitters="0" Type="Clinical significance">
        <Description>no interpretation for the single variant</Description>
      </Interpretation>
    </Interpretations>
    <SubmittedInterpretationList>
      <SCV Title="SUB1895145" Accession="SCV000296057" Version="1"/>
    </SubmittedInterpretationList>
    <InterpretedVariationList>
      <InterpretedVariation VariationID="431733" Accession="VCV000431733" Version="1"/>
    </InterpretedVariationList>
  </IncludedRecord>
</VariationArchive>
</ClinVarVariationRelease>
```
### Parsing

In the following section, we discuss which field of the XML was used to extract information that is presented in the JSON output.

**id**
```xml
<VariationArchive VariationID="431749" VariationName="GRCh37/hg19 1p36.31(chr1:6051187-6158763)" VariationType="copy number gain" DateCreated="2017-08-12" DateLastUpdated="2019-09-10" Accession="VCV000431749" Version="1" RecordType="included" NumberOfSubmissions="0" NumberOfSubmitters="0">
```
The Acc and Version fields are merged to form the ID (RCV000000001.2)

**significance**
```xml {7}
<ClinVarVariationRelease>
  <VariationArchive>
    <IncludedRecord>
      <SimpleAllele>
        <Interpretations>
          <Interpretation NumberOfSubmissions="0" NumberOfSubmitters="0" Type="Clinical significance">
            <Description>no interpretation for the single variant</Description>
          </Interpretation>
        </Interpretations>
      </SimpleAllele>
    </IncludedRecord>
  </VariationArchive>
</ClinVarVariationRelease>
```

May have multiple significances listed.

**reviewStatus**
```xml {4}
<ClinVarVariationRelease>
  <VariationArchive>
    <IncludedRecord>
      <ReviewStatus>no interpretation for the single variant</ReviewStatus>
    </IncludedRecord>
  </VariationArchive>
</ClinVarVariationRelease>
```

## Parsing Significance

Extracting significance(s) may involve parsing multiple fields. Take the following snippets into consideration.

```xml {3,8,13-14}
<ClinicalSignificance DateLastEvaluated="1996-04-01">
    <ReviewStatus>no assertion criteria provided</ReviewStatus>
    <Description>Pathogenic</Description>
</ClinicalSignificance>

<ClinicalSignificance DateLastEvaluated="2016-10-13">
    <ReviewStatus>criteria provided, multiple submitters, no conflicts</ReviewStatus>
    <Description>Pathogenic/Likely pathogenic</Description>
</ClinicalSignificance>

<ClinicalSignificance DateLastEvaluated="2012-06-07">
    <ReviewStatus>no assertion criteria provided</ReviewStatus>
    <Description>Conflicting interpretations of pathogenicity</Description>
    <Explanation DataSource="ClinVar" Type="public">Pathogenic(1);Uncertain significance(1)</Explanation>
</ClinicalSignificance>
```

Given the evidence, we converted significance field into an array of string which may be parsed out of the Descriptions or Explanation fields. Note that the delimiters in each field may vary. Currently, the delimiters for Description are ',' and '/'. For, Explanation we use ';' and '/'.

## Known Issues

:::caution Known Issues
* The XML file contains ~1k more entries (out of 162K) than the VCF file
* The XML file does not have a field indicating that a record is associated with the reference base - something that was present in VCF
* The XML file contains entries (e.g. RCV000016645 version=1) which have IUPAC ambiguous bases ("R", "Y", "H", 
etc.) as their alternate allele
:::

## Download URL

[ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/ClinVarFullRelease_00-latest.xml.gz](ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/ClinVarFullRelease_00-latest.xml.gz)

## JSON Output

<JSON />
