---
title: Primate AI
---

import JSON from './primate-ai-json.md'

## Overview

Primate AI is a deep residual neural network for classifying the pathogenicity of missense mutations. The method is described in these publications:

:::info Publication
1. Hong Gao et al. ,The landscape of tolerated genetic variation in humans and primates. *Science* **380**, eabn8153 (2023). [https://doi.org/10.1126/science.abn8197](https://doi.org/10.1126/science.abn8197)
2. Sundaram, L., Gao, H., Padigepati, S.R. et al. Predicting the clinical impact of human mutation with deep neural networks. *Nat Genet* **50**, 1161â€“1170 (2018). [https://doi.org/10.1038/s41588-018-0167-z](https://doi.org/10.1038/s41588-018-0167-z)
:::


## CSV File

### Example

```scss
,chr,pos,non_flipped_ref,non_flipped_alt,gene_name,change_position_1based,ref_aa,alt_aa,score_PAI3D,percentile_PAI3D,refseq
0,chr1,69094,G,A,ENST00000335137.4,2,V,M,0.6169436463713646,0.5200308441794135,NM_001005484.1
1,chr1,69094,G,C,ENST00000335137.4,2,V,L,0.5557043975591658,0.4271457250214688,NM_001005484.1
2,chr1,69094,G,T,ENST00000335137.4,2,V,L,0.5557043975591658,0.4271457391722522,NM_001005484.1
```


### Parsing

From the CSV file, following columns are parsed:
* `chr`
* `pos`
* `ref`
* `alt`
* `gene_name`
* `score_PAI3D`
* `percentile_PAI3D`
* `refseq`

The fields `gene_name` and `refseq` define the Ensembl and RefSeq transcript IDs respectively.
These fields are mapped to Gene Symbol in the output Json.


## Parsing Run
```text
---------------------------------------------------------------------------
SAUtils                                             (c) 2023 Illumina, Inc.
Stromberg, Roy, Platzer, Siddiqui, Ouyang, et al                     3.21.0
---------------------------------------------------------------------------

Chromosome 1 completed in 00:02:47.9
Chromosome 10 completed in 00:01:07.2
Chromosome 11 completed in 00:01:37.4
Chromosome 12 completed in 00:01:25.8
Chromosome 13 completed in 00:00:28.6
Chromosome 14 completed in 00:00:51.0
Chromosome 15 completed in 00:01:06.4
Chromosome 16 completed in 00:01:17.8
Chromosome 17 completed in 00:01:48.8
Chromosome 18 completed in 00:00:29.5
Chromosome 19 completed in 00:02:00.0
Chromosome 2 completed in 00:02:15.2
Chromosome 20 completed in 00:00:45.5
Chromosome 21 completed in 00:00:18.7
Chromosome 22 completed in 00:00:39.2
Chromosome 3 completed in 00:01:50.6
Chromosome 4 completed in 00:01:17.8
Chromosome 5 completed in 00:01:28.8
Chromosome 6 completed in 00:01:34.4
Chromosome 7 completed in 00:01:30.7
Chromosome 8 completed in 00:01:04.5
Chromosome 9 completed in 00:01:17.2
Chromosome X completed in 00:01:10.7
Chromosome Y completed in 00:00:04.7
Maximum bp shifted for any variant:0
{
  "totalLines": 70667466,
  "totalValid": 69853187,
  "unrecognizedTranscriptIds": 814279,
  "scoreMinimum": 0.0534,
  "scoreMaximum": 0.9988
}


Time: 00:30:24.2

Process finished with exit code 0.
```
About 0.392% of the transcripts were not recognized in GRCh38 assembly.


## Known Issues

:::caution Known Issues
Some transcript IDs defined in the data file are obsolete, retired, or updated.
In such a case, they cannot be mapped onto a gene and are ignored.
:::

### Example:
This entry is skipped because a gene symbol cannot be assigned.

**ENST00000643905.1** transcript is retired according to [Ensembl](https://useast.ensembl.org/Homo_sapiens/Transcript/Idhistory?db=core;t=ENST00000643905)

**NM_182838.2** transcript is removed because it is a pseudo-gene according to [RefSeq](https://www.ncbi.nlm.nih.gov/nuccore/NM_182838.3)


## Download URL

[https://primad.basespace.illumina.com/](https://primad.basespace.illumina.com/)


## JSON Output

<JSON />
