(window.webpackJsonp=window.webpackJsonp||[]).push([[14],{75:function(e,t,n){"use strict";n.r(t),n.d(t,"frontMatter",(function(){return o})),n.d(t,"metadata",(function(){return c})),n.d(t,"rightToc",(function(){return l})),n.d(t,"default",(function(){return s}));var a=n(2),i=n(6),r=(n(0),n(92)),o={title:"Annotating COVID-19"},c={unversionedId:"introduction/covid19",id:"introduction/covid19",isDocsHomePage:!1,title:"Annotating COVID-19",description:"The Nirvana development team is mainly focused on providing annotations for the human genome. This focus allows us maximizing our resources towards understanding human health.",source:"@site/docs/introduction/covid19.md",slug:"/introduction/covid19",permalink:"/NirvanaDocumentation/introduction/covid19",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/introduction/covid19.md",version:"current",sidebar:"docs",previous:{title:"Getting Started",permalink:"/NirvanaDocumentation/introduction/getting-started"},next:{title:"ClinVar",permalink:"/NirvanaDocumentation/data-sources/clinvar"}},l=[{value:"Getting Nirvana",id:"getting-nirvana",children:[]},{value:"Downloading the COVID-19 data files",id:"downloading-the-covid-19-data-files",children:[]},{value:"Download a COVID-19 VCF file",id:"download-a-covid-19-vcf-file",children:[]},{value:"Running Nirvana",id:"running-nirvana",children:[]}],u={rightToc:l};function s(e){var t=e.components,n=Object(i.a)(e,["components"]);return Object(r.b)("wrapper",Object(a.a)({},u,n,{components:t,mdxType:"MDXLayout"}),Object(r.b)("p",null,"The Nirvana development team is mainly focused on providing annotations for the human genome. This focus allows us maximizing our resources towards understanding human health."),Object(r.b)("p",null,"However, nothing in our architecture prevents us from supporting other genomes. Earlier this year, we had an opportunity to put that statement to the test - we added support for annotating the ",Object(r.b)("strong",{parentName:"p"},"SARS-CoV-2")," genome, the virus that causes the ",Object(r.b)("strong",{parentName:"p"},"COVID-19")," disease."),Object(r.b)("h2",{id:"getting-nirvana"},"Getting Nirvana"),Object(r.b)("p",null,"If you don't have Nirvana already, please consult our ",Object(r.b)("a",Object(a.a)({parentName:"p"},{href:"getting-started"}),"Getting Started")," page first."),Object(r.b)("h2",{id:"downloading-the-covid-19-data-files"},"Downloading the COVID-19 data files"),Object(r.b)("p",null,"Here's ",Object(r.b)("a",Object(a.a)({parentName:"p"},{href:"https://illumina.github.io/NirvanaDocumentation/files/Covid19Data.zip"}),"a data zip file")," containing new gene models, reference, and external data sources for SARS-CoV-2:"),Object(r.b)("p",null,"Just go to the directory that contains your Nirvana ",Object(r.b)("inlineCode",{parentName:"p"},"Data")," directory."),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-bash"}),"cd ~/Nirvana\ncurl -O https://illumina.github.io/NirvanaDocumentation/files/Covid19Data.zip\nunzip Covid19Data.zip\n")),Object(r.b)("h2",{id:"download-a-covid-19-vcf-file"},"Download a COVID-19 VCF file"),Object(r.b)("p",null,"Here's ",Object(r.b)("a",Object(a.a)({parentName:"p"},{href:"https://illumina.github.io/NirvanaDocumentation/files/Covid19Mutations.vcf.gz"}),"a COVID-19 VCF file")," you can play around with:"),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-bash"}),"curl -O https://illumina.github.io/NirvanaDocumentation/files/Covid19Mutations.vcf.gz\n")),Object(r.b)("h2",{id:"running-nirvana"},"Running Nirvana"),Object(r.b)("p",null,"Once you have downloaded the data sets, use the following command to annotate your VCF:"),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-bash"}),"dotnet bin/Release/netcoreapp2.1/Nirvana.dll \\\n     -c Data/Cache/SARS-CoV-2/SARS-CoV-2 \\\n     --sd Data/SupplementaryAnnotation/SARS-CoV-2 \\\n     -r Data/References/SARS-CoV-2.ASM985889v3.dat \\\n     -i Covid19Mutations.vcf.gz \\\n     -o Covid19Mutations\n")),Object(r.b)("ul",null,Object(r.b)("li",{parentName:"ul"},"the ",Object(r.b)("inlineCode",{parentName:"li"},"-c")," argument specifies the cache prefix"),Object(r.b)("li",{parentName:"ul"},"the ",Object(r.b)("inlineCode",{parentName:"li"},"--sd")," argument specifies the supplementary annotation directory"),Object(r.b)("li",{parentName:"ul"},"the ",Object(r.b)("inlineCode",{parentName:"li"},"-r")," argument specifies the compressed reference path"),Object(r.b)("li",{parentName:"ul"},"the ",Object(r.b)("inlineCode",{parentName:"li"},"-i")," argument specifies the input VCF path"),Object(r.b)("li",{parentName:"ul"},"the ",Object(r.b)("inlineCode",{parentName:"li"},"-o")," argument specifies the output filename prefix")),Object(r.b)("p",null,"When running Nirvana, performance metrics are shown as it evaluates each chromosome in the input VCF file:"),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-bash"}),"")),Object(r.b)("p",null,"The output will be a JSON file called ",Object(r.b)("inlineCode",{parentName:"p"},"Covid19Mutations.json.gz"),". Here's ",Object(r.b)("a",Object(a.a)({parentName:"p"},{href:"https://illumina.github.io/NirvanaDocumentation/files/Covid19Mutations.json.gz"}),"the full JSON file"),"."))}s.isMDXComponent=!0},92:function(e,t,n){"use strict";n.d(t,"a",(function(){return p})),n.d(t,"b",(function(){return m}));var a=n(0),i=n.n(a);function r(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}function o(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function c(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?o(Object(n),!0).forEach((function(t){r(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):o(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}function l(e,t){if(null==e)return{};var n,a,i=function(e,t){if(null==e)return{};var n,a,i={},r=Object.keys(e);for(a=0;a<r.length;a++)n=r[a],t.indexOf(n)>=0||(i[n]=e[n]);return i}(e,t);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);for(a=0;a<r.length;a++)n=r[a],t.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(e,n)&&(i[n]=e[n])}return i}var u=i.a.createContext({}),s=function(e){var t=i.a.useContext(u),n=t;return e&&(n="function"==typeof e?e(t):c(c({},t),e)),n},p=function(e){var t=s(e.components);return i.a.createElement(u.Provider,{value:t},e.children)},d={inlineCode:"code",wrapper:function(e){var t=e.children;return i.a.createElement(i.a.Fragment,{},t)}},b=i.a.forwardRef((function(e,t){var n=e.components,a=e.mdxType,r=e.originalType,o=e.parentName,u=l(e,["components","mdxType","originalType","parentName"]),p=s(n),b=a,m=p["".concat(o,".").concat(b)]||p[b]||d[b]||r;return n?i.a.createElement(m,c(c({ref:t},u),{},{components:n})):i.a.createElement(m,c({ref:t},u))}));function m(e,t){var n=arguments,a=t&&t.mdxType;if("string"==typeof e||a){var r=n.length,o=new Array(r);o[0]=b;var c={};for(var l in t)hasOwnProperty.call(t,l)&&(c[l]=t[l]);c.originalType=e,c.mdxType="string"==typeof e?e:a,o[1]=c;for(var u=2;u<r;u++)o[u]=n[u];return i.a.createElement.apply(null,o)}return i.a.createElement.apply(null,n)}b.displayName="MDXCreateElement"}}]);