(window.webpackJsonp=window.webpackJsonp||[]).push([[61,4,13,16,20,21,23,36,47,50,51,57,63,70,73,74,77,80,88],{148:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return A})),a.d(t,"metadata",(function(){return w})),a.d(t,"toc",(function(){return D})),a.d(t,"default",(function(){return S}));var n=a(3),r=a(7),b=(a(0),a(179)),c=a(85),l=a(86),i=a(82),o=a(83),j=a(84),O=a(64),p=a(67),d=a(69),m=a(65),s=a(57),u=a(71),g=a(88),N=a(87),f=(a(89),a(90)),y=a(70),h=a(76),v=a(73),A={title:"Nirvana JSON File Format"},w={unversionedId:"file-formats/nirvana-json-file-format",id:"file-formats/nirvana-json-file-format",isDocsHomePage:!1,title:"Nirvana JSON File Format",description:"Overview",source:"@site/docs/file-formats/nirvana-json-file-format.mdx",slug:"/file-formats/nirvana-json-file-format",permalink:"/NirvanaDocumentation/file-formats/nirvana-json-file-format",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/file-formats/nirvana-json-file-format.mdx",version:"current",sidebar:"docs",previous:{title:"TOPMed",permalink:"/NirvanaDocumentation/data-sources/topmed"},next:{title:"Custom Annotations",permalink:"/NirvanaDocumentation/file-formats/custom-annotations"}},D=[{value:"Overview",id:"overview",children:[{value:"Conventions",id:"conventions",children:[]},{value:"JSON Layout",id:"json-layout",children:[]},{value:"Parsing",id:"parsing",children:[]}]},{value:"Header",id:"header",children:[]},{value:"Positions",id:"positions",children:[{value:"ClinGen",id:"clingen",children:[]},{value:"1000 Genomes (SV)",id:"1000-genomes-sv",children:[]},{value:"MITOMAP (SV)",id:"mitomap-sv",children:[]}]},{value:"Samples",id:"samples",children:[]},{value:"Variants",id:"variants",children:[{value:"Transcripts",id:"transcripts",children:[]},{value:"Regulatory Regions",id:"regulatory-regions",children:[]},{value:"ClinVar",id:"clinvar",children:[]},{value:"1000 Genomes",id:"1000-genomes",children:[]},{value:"gnomAD",id:"gnomad",children:[]},{value:"dbSNP",id:"dbsnp",children:[]},{value:"MITOMAP",id:"mitomap",children:[]},{value:"PhyloP",id:"phylop",children:[]},{value:"Primate AI",id:"primate-ai",children:[]},{value:"REVEL",id:"revel",children:[]},{value:"Splice AI",id:"splice-ai",children:[]},{value:"TOPMed",id:"topmed",children:[]}]},{value:"Genes",id:"genes",children:[{value:"OMIM",id:"omim",children:[]},{value:"gnomAD LoF Gene Metrics",id:"gnomad-lof-gene-metrics",children:[]},{value:"ClinGen Disease Validity",id:"clingen-disease-validity",children:[]}]}],M={toc:D};function S(e){var t=e.components,A=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},M,A,{components:t,mdxType:"MDXLayout"}),Object(b.b)("h2",{id:"overview"},"Overview"),Object(b.b)("h3",{id:"conventions"},"Conventions"),Object(b.b)("p",null,"In the Nirvana JSON representation, we try to maximize the amount of useful information that is relayed in the output file. As such, we have several conventions that are useful to know about:"),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"With boolean key/value pairs, we only output the keys that have a true value. I.e. there's no reason to display ",Object(b.b)("inlineCode",{parentName:"li"},'"isStructuralVariant":false')," a few million times when annotating a small variant VCF."),Object(b.b)("li",{parentName:"ul"},"When transferring data from the VCF file to the JSON (e.g. for allele depths (AD)), it is common to use a period (.) as a placeholder for missing data in the VCF file. Nirvana treats periods like empty or null strings and therefore will not output those entries.")),Object(b.b)("h3",{id:"json-layout"},"JSON Layout"),Object(b.b)("p",null,Object(b.b)("img",{src:a(210).default})),Object(b.b)("div",{className:"admonition admonition-info alert alert--info"},Object(b.b)("div",Object(n.a)({parentName:"div"},{className:"admonition-heading"}),Object(b.b)("h5",{parentName:"div"},Object(b.b)("span",Object(n.a)({parentName:"h5"},{className:"admonition-icon"}),Object(b.b)("svg",Object(n.a)({parentName:"span"},{xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"}),Object(b.b)("path",Object(n.a)({parentName:"svg"},{fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"})))),"info")),Object(b.b)("div",Object(n.a)({parentName:"div"},{className:"admonition-content"}),Object(b.b)("p",{parentName:"div"},"In general, each position corresponds to a row in the original VCF file."),Object(b.b)("p",{parentName:"div"},"For each gene that was referenced in the transcripts found in the positions section, there will be additional gene-level annotation in the gene section."))),Object(b.b)("h3",{id:"parsing"},"Parsing"),Object(b.b)("div",{className:"admonition admonition-info alert alert--info"},Object(b.b)("div",Object(n.a)({parentName:"div"},{className:"admonition-heading"}),Object(b.b)("h5",{parentName:"div"},Object(b.b)("span",Object(n.a)({parentName:"h5"},{className:"admonition-icon"}),Object(b.b)("svg",Object(n.a)({parentName:"span"},{xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"}),Object(b.b)("path",Object(n.a)({parentName:"svg"},{fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"})))),"info")),Object(b.b)("div",Object(n.a)({parentName:"div"},{className:"admonition-content"}),Object(b.b)("p",{parentName:"div"},"We've put together a ",Object(b.b)("a",Object(n.a)({parentName:"p"},{href:"../introduction/parsing-json"}),"new section that discusses how to parse our JSON files")," easily using examples in a ",Object(b.b)("a",Object(n.a)({parentName:"p"},{href:"https://github.com/Illumina/NirvanaDocumentation/blob/master/static/files/parse-nirvana-json-python.ipynb"}),"Python Jupyter notebook")," and a ",Object(b.b)("a",Object(n.a)({parentName:"p"},{href:"https://github.com/Illumina/NirvanaDocumentation/blob/master/static/files/parse-nirvana-json-r.ipynb"}),"R version")," as well. In addition, we have information about how to quickly dump content from our JSON file using a tabix-like utility called JASIX."))),Object(b.b)("h2",{id:"header"},"Header"),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'{\n   "header":{\n      "annotator":"Nirvana 3.0.0-alpha.5+g6c52e247",\n      "creationTime":"2017-06-14 15:53:13",\n      "genomeAssembly":"GRCh37",\n      "dataSources":[\n         {\n            "name":"OMIM",\n            "version":"unknown",\n            "description":"An Online Catalog of Human Genes and Genetic Disorders",\n            "releaseDate":"2017-05-03"\n         },\n         {\n            "name":"VEP",\n            "version":"84",\n            "description":"BothRefSeqAndEnsembl",\n            "releaseDate":"2017-01-16"\n         },\n         {\n            "name":"ClinVar",\n            "version":"20170503",\n            "description":"A freely accessible, public archive of reports of the relationships among human variations and phenotypes, with supporting evidence",\n            "releaseDate":"2017-05-03"\n         },\n         {\n            "name":"phyloP",\n            "version":"hg19",\n            "description":"46 way conservation score between humans and 45 other vertebrates",\n            "releaseDate":"2009-11-10"\n         }\n      ],\n      "samples":[\n         "NA12878",\n         "NA12891",\n         "NA12892"\n      ]\n   },\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"annotator"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"the name of the annotator and the current version")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"creationTime"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"yyyy-MM-dd hh:mm:ss")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"genomeAssembly"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"#genome-assemblies"}),"possible values below"))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"schemaVersion"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"incremented whenever the core structure of the JSON file introduces breaking changes")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"dataVersion"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"dataSources"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"object array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"#data-source"}),"Data Source entry below"))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"samples"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"the order of these sample names will be used throughout the JSON file when enumerating samples")))),Object(b.b)("h4",{id:"data-source"},"Data Source"),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"name"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"version"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"description"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"optional description of the data source")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"releaseDate"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"yyyy-MM-dd")))),Object(b.b)("h4",{id:"genome-assemblies"},"Genome Assemblies"),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"GRCh37"),Object(b.b)("li",{parentName:"ul"},"GRCh38"),Object(b.b)("li",{parentName:"ul"},"hg19"),Object(b.b)("li",{parentName:"ul"},"SARSCoV2")),Object(b.b)("h2",{id:"positions"},"Positions"),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"positions":[\n   {\n      "chromosome":"chr2",\n      "position":48010488,\n      "repeatUnit":"GGCCCC",\n      "refRepeatCount":3,\n      "svEnd":48020488,\n      "refAllele":"G",\n      "altAlleles":[\n         "A",\n         "GT"\n      ],\n      "quality":461,\n      "filters":[\n         "PASS"\n      ],\n      "ciPos":[\n         -170,\n         170\n      ],\n      "ciEnd":[\n         -175,\n         175\n      ],\n      "svLength":1000,\n      "strandBias":1.23,\n      "jointSomaticNormalQuality":29,\n      "cytogeneticBand":"2p16.3",\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Variant Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"chromosome"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"all"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"exactly as displayed in the vcf")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"position"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"all"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"exactly as displayed in the vcf (1-based notation). Range: 1 - 250 million")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"repeatUnit"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"STR"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"provided by ExpansionHunter")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"refRepeatCount"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"STR"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"provided by ExpansionHunter")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"svEnd"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"SV"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"refAllele"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"all"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"exactly as displayed in the vcf")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"altAllele"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"all"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"exactly as displayed in the vcf")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"quality"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"all"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"exactly as displayed in the vcf (Normally an integer, but some variant callers using floating point. Has been observed as high as 500k)")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"filters"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"all"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"exactly as displayed in the vcf")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"ciPos"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"SV"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"ciEnd"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"SV"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"svLength"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"SV"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"strandBias"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"small variant"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"provided by GATK (from SB)")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"jointSomaticNormalQuality"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"SV"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"provided by the Manta variant caller (SOMATICSCORE)")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"cytogeneticBand"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"all"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"e.g. 17p13.1")))),Object(b.b)("h3",{id:"clingen"},"ClinGen"),Object(b.b)(i.default,{mdxType:"ClinGen"}),Object(b.b)(o.default,{mdxType:"ClinGenDosage"}),Object(b.b)("h3",{id:"1000-genomes-sv"},"1000 Genomes (SV)"),Object(b.b)(y.default,{mdxType:"ThousandGenomesSV"}),Object(b.b)("h3",{id:"mitomap-sv"},"MITOMAP (SV)"),Object(b.b)(u.default,{mdxType:"MitoMapSV"}),Object(b.b)("h2",{id:"samples"},"Samples"),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"samples":[\n   {\n      "genotype":"0/1",\n      "variantFrequencies":[\n         0.333,\n         0.5\n      ],\n      "totalDepth":57,\n      "genotypeQuality":12,\n      "copyNumber":3,\n      "repeatUnitCounts":[\n         10,\n         20\n      ],\n      "alleleDepths":[\n         10,\n         20,\n         30\n      ],\n      "failedFilter":true,\n      "splitReadCounts":[\n         10,\n         20\n      ],\n      "pairedEndReadCounts":[\n         10,\n         20\n      ],\n      "isDeNovo":true,\n      "diseaseAffectedStatuses":[\n         "-"\n      ],\n      "artifactAdjustedQualityScore":89.3,\n      "likelihoodRatioQualityScore":78.2,\n      "heteroplasmyPercentile":[\n         23.13,\n         12.65\n      ]\n   }\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"genotype"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"variantFrequencies"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"range: 0 - 1.0. One value per alternate allele")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"totalDepth"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"non-negative integer values")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"genotypeQuality"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"non-negative integer values. Typically maxes out at 99")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"copyNumber"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"non-negative integer values")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"repeatUnitCounts"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"ExpansionHunter-specific")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"alleleDepths"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"non-negative integer values")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"failedFilter"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"splitReadCounts"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"Manta-specific")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"pairedEndReadCounts"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"Manta-specific")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"isDeNovo"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"diseaseAffectedStatuses"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"ExpansionHunter-specific")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"artifactAdjustedQualityScore"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"PEPE-specific. Range: 0 - 100.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"likelihoodRatioQualityScore"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"PEPE-specific. Range: 0 - 100.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"heteroplasmyPercentile"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"range: 0 - 100. 2 decimal places. One value per alternate allele")))),Object(b.b)("div",{className:"admonition admonition-info alert alert--info"},Object(b.b)("div",Object(n.a)({parentName:"div"},{className:"admonition-heading"}),Object(b.b)("h5",{parentName:"div"},Object(b.b)("span",Object(n.a)({parentName:"h5"},{className:"admonition-icon"}),Object(b.b)("svg",Object(n.a)({parentName:"span"},{xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"}),Object(b.b)("path",Object(n.a)({parentName:"svg"},{fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"})))),"Empty Samples")),Object(b.b)("div",Object(n.a)({parentName:"div"},{className:"admonition-content"}),Object(b.b)("p",{parentName:"div"},"If a sample does not contain any entries, we will create a sample object that contains the ",Object(b.b)("inlineCode",{parentName:"p"},"isEmpty")," key. This ensures that sample ordering is preserved while indicating that a sample is intentionally empty."),Object(b.b)("pre",{parentName:"div"},Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"samples":[\n   {\n      "isEmpty":true\n   }\n],\n')))),Object(b.b)("h2",{id:"variants"},"Variants"),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"variants":[\n   {\n      "vid":"2:48010488:A",\n      "chromosome":"chr2",\n      "begin":48010488,\n      "end":48010488,\n      "isReferenceMinorAllele":true,\n      "isStructuralVariant":true,\n      "refAllele":"G",\n      "altAllele":"A",\n      "variantType":"SNV",\n      "isDecomposedVariant":true,\n      "isRecomposedVariant":true,\n      "linkedVids":["2:48010488:GTA:ATC"],\n      "hgvsg":"NC_000002.11:g.48010488G>A",\n      "phylopScore":0.459\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"vid"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"../core-functionality/variant-ids"}),"Variant Identifiers"))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"chromosome"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"begin"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"1-based non-negative integer values. Range: 1 - 250 million")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"end"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"1-based non-negative integer values. Range: 1 - 250 million")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"isReferenceMinorAllele"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"true when this is a reference minor allele")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"isStructuralVariant"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"true when the variant is a structural variant")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"inLowComplexityRegion"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"true when the variant lies in a low complexity region (gnomAD low complexity regions)")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"refAllele"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"parsimonious representation of the reference allele")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"altAllele"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"parsimonious representation of the alternate allele.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"variantType"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"uses\xa0",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"http://www.sequenceontology.org/browser/current_svn/term/SO:0001059"}),"Sequence Ontology sequence alterations"))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"isDecomposedVariant"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"true when the decomposed variant has been used to create another recomposed variant")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"isRecomposedVariant"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"true when the variant is recomposed from two or more decomposed variants")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"linkedVids"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"list of ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"../core-functionality/variant-ids"}),"VIDs")," for variants connecting decomposed and recomposed variants")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"hgvsg"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"HGVS g. notation")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"phylopScore"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"phyloP conservation score. Range: -14.08 to 6.424")))),Object(b.b)("div",{className:"admonition admonition-info alert alert--info"},Object(b.b)("div",Object(n.a)({parentName:"div"},{className:"admonition-heading"}),Object(b.b)("h5",{parentName:"div"},Object(b.b)("span",Object(n.a)({parentName:"h5"},{className:"admonition-icon"}),Object(b.b)("svg",Object(n.a)({parentName:"span"},{xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"}),Object(b.b)("path",Object(n.a)({parentName:"svg"},{fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"})))),"Reference Minor Alleles")),Object(b.b)("div",Object(n.a)({parentName:"div"},{className:"admonition-content"}),Object(b.b)("p",{parentName:"div"},"Nirvana supports annotating reference minor alleles. In such a case, ",Object(b.b)("inlineCode",{parentName:"p"},"refAllele")," will be replaced by the global major allele and ",Object(b.b)("inlineCode",{parentName:"p"},"altAllele")," will be replaced with the original reference allele."))),Object(b.b)("div",{className:"admonition admonition-info alert alert--info"},Object(b.b)("div",Object(n.a)({parentName:"div"},{className:"admonition-heading"}),Object(b.b)("h5",{parentName:"div"},Object(b.b)("span",Object(n.a)({parentName:"h5"},{className:"admonition-icon"}),Object(b.b)("svg",Object(n.a)({parentName:"span"},{xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"}),Object(b.b)("path",Object(n.a)({parentName:"svg"},{fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"})))),"Flagging Decomposed & Recomposed Variants")),Object(b.b)("div",Object(n.a)({parentName:"div"},{className:"admonition-content"}),Object(b.b)("p",{parentName:"div"},"When two or more decomposed variants are recomposed into an MNV, the decomposed variants will be marked with ",Object(b.b)("inlineCode",{parentName:"p"},'"isDecomposedVariant":true'),"."),Object(b.b)("p",{parentName:"div"},"Similarly, the recomposed variant will be shown as a new VCF position. This recomposed variant will be flagged with ",Object(b.b)("inlineCode",{parentName:"p"},'"isRecomposedVariant":true'),"."))),Object(b.b)("h3",{id:"transcripts"},"Transcripts"),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"transcripts":[\n   {\n      "transcript":"ENST00000445503.1",\n      "source":"Ensembl",\n      "bioType":"nonsense_mediated_decay",\n      "codons":"gGg/gAg",\n      "aminoAcids":"G/E",\n      "cdnaPos":"268",\n      "cdsPos":"116",\n      "exons":"1/9",\n      "introns":"1/8",\n      "proteinPos":"39",\n      "geneId":"ENSG00000116062",\n      "hgnc":"MSH6",\n      "consequence":[\n         "missense_variant",\n         "NMD_transcript_variant"\n      ],\n      "hgvsc":"ENST00000445503.1:c.116G>A",\n      "hgvsp":"ENSP00000405294.1:p.(Gly39Glu)",\n      "geneFusion":{\n         "exon":6,\n         "intron":5,\n         "fusions":[\n            {\n               "hgvsc":"ETV6{ENST00000396373.4}:c.1_1009+3402_RUNX1{ENST00000437180.1}:c.58+568_1443",\n               "exon":3,\n               "intron":2\n            },\n            {\n               "hgvsc":"ETV6{ENST00000396373.4}:c.1_1009+3402_RUNX1{ENST00000300305.3}:c.58+568_1443",\n               "exon":2,\n               "intron":1\n            }\n         ]\n      },\n      "isCanonical":true,\n      "polyPhenScore":0.95,\n      "polyPhenPrediction":"probably damaging",\n      "proteinId":"ENSP00000405294.1",\n      "siftScore":0.61,\n      "siftPrediction":"tolerated",\n      "completeOverlap":true\n   }\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"transcript"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"transcript ID. e.g. ENST00000445503.1")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"source"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"RefSeq / Ensembl")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"bioType"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"descriptions of the ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"https://uswest.ensembl.org/info/genome/genebuild/biotypes.html"}),"biotypes from Ensembl"))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"codons"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"aminoAcids"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"cdnaPos"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"cdsPos"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"exons"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"exons affected by the variant")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"introns"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"introns affected by the variant")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"proteinPos"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"geneId"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"gene ID. e.g. ENSG00000116062")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"hgnc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"gene symbol. e.g. MSH6")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"consequence"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"http://www.sequenceontology.org/browser/obob.cgi"}),"Sequence Ontology Consequences"))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"hgvsc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"HGVS coding nomenclature")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"hgvsp"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"HGVS protein nomenclature")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"geneFusion"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"object"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"#gene-fusions"}),"Gene Fusions entry below"))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"isCanonical"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"true when this is a canonical transcript")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"polyPhenScore"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"polyPhenPrediction"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"#polyphen"}),"possible values below"))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"proteinId"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"protein ID. E.g. ENSP00000405294.1")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"siftScore"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"siftPrediction"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"#sift"}),"possible values below"))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"completeOverlap"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"true when this transcript is completely overlapped by the variant")))),Object(b.b)("h4",{id:"polyphen"},"PolyPhen"),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"probably damaging"),Object(b.b)("li",{parentName:"ul"},"possibly damaging"),Object(b.b)("li",{parentName:"ul"},"benign"),Object(b.b)("li",{parentName:"ul"},"unknown")),Object(b.b)("h4",{id:"sift"},"SIFT"),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"tolerated"),Object(b.b)("li",{parentName:"ul"},"deleterious"),Object(b.b)("li",{parentName:"ul"},"tolerated - low confidence"),Object(b.b)("li",{parentName:"ul"},"deleterious - low confidence")),Object(b.b)("h4",{id:"amino-acid-conservation"},"Amino Acid Conservation"),Object(b.b)(c.default,{mdxType:"AminoAcidConservation"}),Object(b.b)("h4",{id:"gene-fusions"},"Gene Fusions"),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"exon"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"actual exon where the breakpoint was located")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"intron"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"actual intron where the breakpoint was located")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"fusions"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"object array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"#fusion"}),"Fusion entry below"))))),Object(b.b)("h4",{id:"fusion"},"Fusion"),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"exon"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"actual exon where the other breakpoint was located")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"intron"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"actual intron where the other breakpoint was located")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"hgvsc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"HGVS coding nomenclature describing the two genes and the transcripts that are fused along with")))),Object(b.b)("h3",{id:"regulatory-regions"},"Regulatory Regions"),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"regulatoryRegions":[\n   {\n      "id":"ENSR00001542175",\n      "type":"promoter",\n      "consequence":[\n         "regulatory_region_variant"\n      ]\n   }\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"id"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"type"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"#regulatory-types"}),"possible values below"))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"consequence"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"#regulatory-consequences"}),"possible values below"))))),Object(b.b)("h4",{id:"regulatory-types"},"Regulatory Types"),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"CTCF_binding_site"),Object(b.b)("li",{parentName:"ul"},"enhancer"),Object(b.b)("li",{parentName:"ul"},"open_chromatin_region"),Object(b.b)("li",{parentName:"ul"},"promoter"),Object(b.b)("li",{parentName:"ul"},"promoter_flanking_region"),Object(b.b)("li",{parentName:"ul"},"TF_binding_site")),Object(b.b)("h4",{id:"regulatory-consequences"},"Regulatory Consequences"),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"regulatory_region_variant"),Object(b.b)("li",{parentName:"ul"},"regulatory_region_ablation"),Object(b.b)("li",{parentName:"ul"},"regulatory_region_amplification"),Object(b.b)("li",{parentName:"ul"},"regulatory_region_truncation")),Object(b.b)("h3",{id:"clinvar"},"ClinVar"),Object(b.b)(l.default,{mdxType:"ClinVar"}),Object(b.b)("h3",{id:"1000-genomes"},"1000 Genomes"),Object(b.b)(f.default,{mdxType:"ThousandGenomesSmall"}),Object(b.b)("h3",{id:"gnomad"},"gnomAD"),Object(b.b)(g.default,{mdxType:"GnomadSmall"}),Object(b.b)("h3",{id:"dbsnp"},"dbSNP"),Object(b.b)(O.default,{mdxType:"DbSNP"}),Object(b.b)("h3",{id:"mitomap"},"MITOMAP"),Object(b.b)(s.default,{mdxType:"MitoMapSmall"}),Object(b.b)("h3",{id:"phylop"},"PhyloP"),Object(b.b)("phylop",null),Object(b.b)("h3",{id:"primate-ai"},"Primate AI"),Object(b.b)(p.default,{mdxType:"PrimateAI"}),Object(b.b)("h3",{id:"revel"},"REVEL"),Object(b.b)(d.default,{mdxType:"REVEL"}),Object(b.b)("h3",{id:"splice-ai"},"Splice AI"),Object(b.b)(m.default,{mdxType:"SpliceAI"}),Object(b.b)("h3",{id:"topmed"},"TOPMed"),Object(b.b)(v.default,{mdxType:"TOPMed"}),Object(b.b)("h2",{id:"genes"},"Genes"),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"genes":[\n   {\n      "name":"MSH6",\n      "hgncId":7329,\n      "summary":"This gene encodes a member of the DNA mismatch repair MutS family. In E. coli, the MutS protein helps in the recognition of mismatched nucleotides prior to their repair. A highly conserved region of approximately 150 aa, called the Walker-A adenine nucleotide binding motif, exists in MutS homologs. The encoded protein heterodimerizes with MSH2 to form a mismatch recognition complex that functions as a bidirectional molecular switch that exchanges ADP and ATP as DNA mismatches are bound and dissociated. Mutations in this gene may be associated with hereditary nonpolyposis colon cancer, colorectal cancer, and endometrial cancer. Transcripts variants encoding different isoforms have been described. [provided by RefSeq, Jul 2013]",\n      /* this is where gene-level data sources can be found e.g. OMIM */\n   }\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"name"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"HGNC gene symbol")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"hgncId"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"HGNC ID")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"summary"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"short description of the gene from ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"https://www.omim.org/"}),"OMIM"))))),Object(b.b)("h3",{id:"omim"},"OMIM"),Object(b.b)(h.default,{mdxType:"Omim"}),Object(b.b)("h3",{id:"gnomad-lof-gene-metrics"},"gnomAD LoF Gene Metrics"),Object(b.b)(N.default,{mdxType:"GnomadGeneLof"}),Object(b.b)("h3",{id:"clingen-disease-validity"},"ClinGen Disease Validity"),Object(b.b)(j.default,{mdxType:"ClinGenDiseaseValidity"}))}S.isMDXComponent=!0},179:function(e,t,a){"use strict";a.d(t,"a",(function(){return O})),a.d(t,"b",(function(){return m}));var n=a(0),r=a.n(n);function b(e,t,a){return t in e?Object.defineProperty(e,t,{value:a,enumerable:!0,configurable:!0,writable:!0}):e[t]=a,e}function c(e,t){var a=Object.keys(e);if(Object.getOwnPropertySymbols){var n=Object.getOwnPropertySymbols(e);t&&(n=n.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),a.push.apply(a,n)}return a}function l(e){for(var t=1;t<arguments.length;t++){var a=null!=arguments[t]?arguments[t]:{};t%2?c(Object(a),!0).forEach((function(t){b(e,t,a[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(a)):c(Object(a)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(a,t))}))}return e}function i(e,t){if(null==e)return{};var a,n,r=function(e,t){if(null==e)return{};var a,n,r={},b=Object.keys(e);for(n=0;n<b.length;n++)a=b[n],t.indexOf(a)>=0||(r[a]=e[a]);return r}(e,t);if(Object.getOwnPropertySymbols){var b=Object.getOwnPropertySymbols(e);for(n=0;n<b.length;n++)a=b[n],t.indexOf(a)>=0||Object.prototype.propertyIsEnumerable.call(e,a)&&(r[a]=e[a])}return r}var o=r.a.createContext({}),j=function(e){var t=r.a.useContext(o),a=t;return e&&(a="function"==typeof e?e(t):l(l({},t),e)),a},O=function(e){var t=j(e.components);return r.a.createElement(o.Provider,{value:t},e.children)},p={inlineCode:"code",wrapper:function(e){var t=e.children;return r.a.createElement(r.a.Fragment,{},t)}},d=r.a.forwardRef((function(e,t){var a=e.components,n=e.mdxType,b=e.originalType,c=e.parentName,o=i(e,["components","mdxType","originalType","parentName"]),O=j(a),d=n,m=O["".concat(c,".").concat(d)]||O[d]||p[d]||b;return a?r.a.createElement(m,l(l({ref:t},o),{},{components:a})):r.a.createElement(m,l({ref:t},o))}));function m(e,t){var a=arguments,n=t&&t.mdxType;if("string"==typeof e||n){var b=a.length,c=new Array(b);c[0]=d;var l={};for(var i in t)hasOwnProperty.call(t,i)&&(l[i]=t[i]);l.originalType=e,l.mdxType="string"==typeof e?e:n,c[1]=l;for(var o=2;o<b;o++)c[o]=a[o];return r.a.createElement.apply(null,c)}return r.a.createElement.apply(null,a)}d.displayName="MDXCreateElement"},210:function(e,t,a){"use strict";a.r(t),t.default=a.p+"assets/images/JSON-Layout-fc8e5c0cf4c8428981cd206fe9b6feac.svg"},57:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/mitomap-small-variants-json",id:"data-sources/mitomap-small-variants-json",isDocsHomePage:!1,title:"mitomap-small-variants-json",description:"`json",source:"@site/docs/data-sources/mitomap-small-variants-json.md",slug:"/data-sources/mitomap-small-variants-json",permalink:"/NirvanaDocumentation/data-sources/mitomap-small-variants-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/mitomap-small-variants-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"mitomap":[ \n   { \n      "refAllele":"G",\n      "altAllele":"A",\n      "diseases":[ \n         "Bipolar disorder",\n         "Melanoma"\n      ],\n      "hasHomoplasmy":false,\n      "hasHeteroplasmy":true,\n      "status":"Reported",\n      "clinicalSignificance":"confirmed pathogenic",\n      "scorePercentile":83.30,\n      "numGenBankFullLengthSeqs":2,\n      "pubMedIds":["2316527","6299878","6301949"],\n      "isAlleleSpecific":true\n   }\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"refAllele"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"altAllele"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"diseases"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"associated diseases")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"hasHomoplasmy"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"boolean"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"hasHeteroplasmy"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"boolean"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"status"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"record status")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"clinicalSignificance"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"predicted pathogenicity")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"scorePercentile"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"MitoTIP score")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"numGenBankFullLengthSeqs"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"# of GenBank full-length sequences")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"pubMedIds"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"isAlleleSpecific"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"boolean"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"true when the current variant alternate allele matches the MITOMAP alternate allele")))))}j.isMDXComponent=!0},64:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/dbsnp-json",id:"data-sources/dbsnp-json",isDocsHomePage:!1,title:"dbsnp-json",description:"`json",source:"@site/docs/data-sources/dbsnp-json.md",slug:"/data-sources/dbsnp-json",permalink:"/NirvanaDocumentation/data-sources/dbsnp-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/dbsnp-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"dbsnp":[\n   "rs1042821"\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"dbsnp"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"dbSNP rsIDs")))))}j.isMDXComponent=!0},65:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/splice-ai-json",id:"data-sources/splice-ai-json",isDocsHomePage:!1,title:"splice-ai-json",description:"`json",source:"@site/docs/data-sources/splice-ai-json.md",slug:"/data-sources/splice-ai-json",permalink:"/NirvanaDocumentation/data-sources/splice-ai-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/splice-ai-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"spliceAI":[ \n   {\n      "hgnc":"BLCAP",\n      "acceptorGainDistance":-3,\n      "acceptorGainScore":0.3,\n      "donorLossDistance":7,\n      "donorLossScore":0.9\n   },\n   { \n      "hgnc":"NNAT",\n      "acceptorGainDistance":-1,\n      "acceptorGainScore":0.2,\n      "donorGainDistance":-2,\n      "donorGainScore":0.3\n   }\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"hgnc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"HGNC gene symbol")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"acceptorGainDistance"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"\xb1 bp from current position")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"acceptorGainScore"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"range: 0 - 1.0. 1 decimal place")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"acceptorLossDistance"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"\xb1 bp from current position")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"acceptorLossScore"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"range: 0 - 1.0. 1 decimal place")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"donorGainDistance"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"\xb1 bp from current position")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"donorGainScore"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"range: 0 - 1.0. 1 decimal place")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"donorLossDistance"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"\xb1 bp from current position")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"donorLossScore"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"range: 0 - 1.0. 1 decimal place")))))}j.isMDXComponent=!0},67:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/primate-ai-json",id:"data-sources/primate-ai-json",isDocsHomePage:!1,title:"primate-ai-json",description:"`json",source:"@site/docs/data-sources/primate-ai-json.md",slug:"/data-sources/primate-ai-json",permalink:"/NirvanaDocumentation/data-sources/primate-ai-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/primate-ai-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"primateAI":[\n   {\n      "hgnc":"TP53",\n      "scorePercentile":0.3,\n   }\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"hgnc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"scorePercentile"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"range: 0 - 1.0")))))}j.isMDXComponent=!0},69:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/revel-json",id:"data-sources/revel-json",isDocsHomePage:!1,title:"revel-json",description:"`json",source:"@site/docs/data-sources/revel-json.md",slug:"/data-sources/revel-json",permalink:"/NirvanaDocumentation/data-sources/revel-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/revel-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"revel":{ \n   "score":0.027\n}\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"score"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"Range: 0 - 1.0")))))}j.isMDXComponent=!0},70:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/1000Genomes-sv-json",id:"data-sources/1000Genomes-sv-json",isDocsHomePage:!1,title:"1000Genomes-sv-json",description:"`json",source:"@site/docs/data-sources/1000Genomes-sv-json.md",slug:"/data-sources/1000Genomes-sv-json",permalink:"/NirvanaDocumentation/data-sources/1000Genomes-sv-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/1000Genomes-sv-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"oneKg":[\n   {\n      "chromosome":"1",\n      "begin":1595369,\n      "end":1612441,\n      "variantType": "copy_number_variation",\n      "id": "esv3635753;esv3635754;esv3635755;esv3635756;esv3635757",\n      "allAn": 5008,\n      "allAc": 2702,\n      "allAf": 0.539537,\n      "afrAf": 0.6052,\n      "amrAf": 0.3675,\n      "eurAf": 0.5357,\n      "easAf": 0.5368,\n      "sasAf": 0.5797,\n      "reciprocalOverlap": 0.07555\n   }\n],\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"chromosome"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"begin"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"end"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"variantType"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"id"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"floating point"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for all populations. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"floating point"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for all populations. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"floating point"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for all populations. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"afrAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"floating point"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the African super population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"amrAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"floating point"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the Ad Mixed American super population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"eurAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"floating point"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the European super population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"easAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the East Asian super population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"sasAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the South Asian super population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"reciprocalOverlap"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"floating point"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"range: 0 - 1.")))))}j.isMDXComponent=!0},71:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/mitomap-structural-variants-json",id:"data-sources/mitomap-structural-variants-json",isDocsHomePage:!1,title:"mitomap-structural-variants-json",description:"`json",source:"@site/docs/data-sources/mitomap-structural-variants-json.md",slug:"/data-sources/mitomap-structural-variants-json",permalink:"/NirvanaDocumentation/data-sources/mitomap-structural-variants-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/mitomap-structural-variants-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"mitomap":[ \n   { \n      "chromosome":"MT",\n      "begin":3166,\n      "end":14152,\n      "variantType":"deletion",\n      "reciprocalOverlap":0.18068,\n      "annotationOverlap":0.42405\n   }\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"chromosome"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"begin"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"end"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"variantType"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"reciprocalOverlap"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"Range: 0 - 1. Specified up to 5 decimal places")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"annotationOverlap"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"Range: 0 - 1. Specified up to 5 decimal places")))))}j.isMDXComponent=!0},73:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/topmed-json",id:"data-sources/topmed-json",isDocsHomePage:!1,title:"topmed-json",description:"`json",source:"@site/docs/data-sources/topmed-json.md",slug:"/data-sources/topmed-json",permalink:"/NirvanaDocumentation/data-sources/topmed-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/topmed-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"topmed":{ \n   "allAc":20,\n   "allAn":125568,\n   "allAf":0.000159,\n   "allHc":0,\n   "failedFilter":true\n}\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"TOPMed allele count")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"TOPMed allele number. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"TOPMed allele frequency (computed by Nirvana)")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allHc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"TOPMed homozygous count")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"failedFilter"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"True if this variant failed any filters")))))}j.isMDXComponent=!0},76:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/omim-json",id:"data-sources/omim-json",isDocsHomePage:!1,title:"omim-json",description:"`json",source:"@site/docs/data-sources/omim-json.md",slug:"/data-sources/omim-json",permalink:"/NirvanaDocumentation/data-sources/omim-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/omim-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"omim":[ \n   { \n      "mimNumber":600678,\n      "geneName":"MutS, E. coli, homolog of, 6",\n      "description":"The transcription factor p53 responds to diverse cellular stresses to regulate target genes that induce cell cycle arrest, apoptosis, senescence, DNA repair, or changes in metabolism. In addition, p53 appears to induce apoptosis through nontranscriptional cytoplasmic processes. In unstressed cells, p53 is kept inactive essentially through the actions of the ubiquitin ligase MDM2, which inhibits p53 transcriptional activity and ubiquitinates p53 to promote its degradation. Numerous posttranslational modifications modulate p53 activity, most notably phosphorylation and acetylation. Several less abundant p53 isoforms also modulate p53 activity. Activity of p53 is ubiquitously lost in human cancer either by mutation of the p53 gene itself or by loss of cell signaling upstream or downstream of p53 (Toledo and Wahl, 2006; Bourdon, 2007; Vousden and Lane, 2007)",\n      "phenotypes":[ \n         { \n            "mimNumber":614350,\n            "phenotype":"Colorectal cancer, hereditary nonpolyposis, type 5",\n            "description":"Hereditary nonpolyposis colorectal cancer type 5 is a cancer predisposition syndrome ...",\n            "mapping":"molecular basis of the disorder is known",\n            "inheritances":[ \n               "Autosomal dominant"\n            ]\n         },\n         { \n            "mimNumber":608089,\n            "phenotype":"Endometrial cancer, familial",\n            "mapping":"molecular basis of the disorder is known"\n         },\n         { \n            "mimNumber":276300,\n            "phenotype":"Mismatch repair cancer syndrome",\n            "description":"Constitutional mismatch repair deficiency is a rare childhood cancer predisposition syndrome ...",\n            "mapping":"molecular basis of the disorder is known",\n            "inheritances":[ \n               "Autosomal recessive"\n            ],\n            "comments"     : [\n                "contribute to susceptibility to multifactorial disorders or to susceptibility to infection",\n                "unconfirmed or possibly spurious mapping"\n            ]\n         }\n      ]\n   }\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"mimNumber"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"OMIM ID for gene")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"geneName"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"gene name")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"description"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"phenotypes"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"object array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"#phenotype"}),"Phenotype entry below"))))),Object(b.b)("h4",{id:"phenotype"},"Phenotype"),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"mimNumber"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"phenotype"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"description"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"mapping"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"#mapping"}),"possible values below"))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"inheritance"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"#inheritance"}),"possible values below"))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"comments"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see ",Object(b.b)("a",Object(n.a)({parentName:"td"},{href:"#comments"}),"possible values below"))))),Object(b.b)("h4",{id:"mapping"},"Mapping"),Object(b.b)("ol",null,Object(b.b)("li",{parentName:"ol"},"disorder was positioned by mapping of the wild type gene"),Object(b.b)("li",{parentName:"ol"},"disease phenotype itself was mapped"),Object(b.b)("li",{parentName:"ol"},"molecular basis of the disorder is known"),Object(b.b)("li",{parentName:"ol"},"disorder is a chromosome deletion or duplication syndrome")),Object(b.b)("h4",{id:"inheritance"},"Inheritance"),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"autosomal recessive"),Object(b.b)("li",{parentName:"ul"},"autosomal dominant")),Object(b.b)("h4",{id:"comments"},"Comments"),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"contributes to the susceptibility to multifactorial disorders"),Object(b.b)("li",{parentName:"ul"},"variations that lead to apparently abnormal laboratory test values"),Object(b.b)("li",{parentName:"ul"},"unconfirmed mapping")))}j.isMDXComponent=!0},82:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/clingen-json",id:"data-sources/clingen-json",isDocsHomePage:!1,title:"clingen-json",description:"`json",source:"@site/docs/data-sources/clingen-json.md",slug:"/data-sources/clingen-json",permalink:"/NirvanaDocumentation/data-sources/clingen-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/clingen-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"clingen":[\n   {\n      "chromosome":"17",\n      "begin":525,\n      "end":14667519,\n      "variantType":"copy_number_gain",\n      "id":"nsv996083",\n      "clinicalInterpretation":"pathogenic",\n      "observedGains":1,\n      "validated":true,\n      "phenotypes":[\n         "Intrauterine growth retardation"\n      ],\n      "phenotypeIds":[\n         "HP:0001511",\n         "MedGen:C1853481"\n      ],\n      "reciprocalOverlap":0.00131\n   },\n   {\n      "chromosome":"17",\n      "begin":45835,\n      "end":7600330,\n      "variantType":"copy_number_loss",\n      "id":"nsv869419",\n      "clinicalInterpretation":"pathogenic",\n      "observedLosses":1,\n      "validated":true,\n      "phenotypes":[\n         "Developmental delay AND/OR other significant developmental or morphological phenotypes"\n      ],\n      "reciprocalOverlap":0.00254\n   }\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"clingen"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"object array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"chromosome"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"Ensembl-style chromosome names")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"begin"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"1-based position")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"end"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"1-based position")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"variantType"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"Any of the\xa0sequence alterations defined here.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"id"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"Identifier from the data source. Alternatively a VID")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"clinicalInterpretation"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"see possible values below")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"observedGains"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"Range: 0 - (2",Object(b.b)("sup",null,"31"),"\xa0- 1). Only used if copy_number_variation, copy_number_loss, or copy_number_gain.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"observedLosses"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"Range: 0 - (2",Object(b.b)("sup",null,"31"),"\xa0- 1). Only used if copy_number_variation, copy_number_loss, or copy_number_gain.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"validated"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"boolean"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"phenotypes"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"Description of the phenotype.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"phenotypeIds"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"Description of the phenotype IDs.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"reciprocalOverlap"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"floating point"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"Range: 0 - 1. E.g. 0.57 would indicate a 57% reciprocal overlap. Specified up to 5 decimal places (Not reported for Insertions).")))),Object(b.b)("p",null,Object(b.b)("strong",{parentName:"p"},"clinicalInterpretation")),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"benign"),Object(b.b)("li",{parentName:"ul"},"curated benign"),Object(b.b)("li",{parentName:"ul"},"curated pathogenic"),Object(b.b)("li",{parentName:"ul"},"likely benign"),Object(b.b)("li",{parentName:"ul"},"likely pathogenic"),Object(b.b)("li",{parentName:"ul"},"path gain"),Object(b.b)("li",{parentName:"ul"},"path loss"),Object(b.b)("li",{parentName:"ul"},"pathogenic"),Object(b.b)("li",{parentName:"ul"},"uncertain")))}j.isMDXComponent=!0},83:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/clingen-dosage-json",id:"data-sources/clingen-dosage-json",isDocsHomePage:!1,title:"clingen-dosage-json",description:"`json",source:"@site/docs/data-sources/clingen-dosage-json.md",slug:"/data-sources/clingen-dosage-json",permalink:"/NirvanaDocumentation/data-sources/clingen-dosage-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/clingen-dosage-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"clingenDosageSensitivityMap": [{\n    "chromosome": "15",\n    "begin": 30900686,\n    "end": 32153204,\n    "haploinsufficiency": "sufficient evidence suggesting dosage sensitivity is associated with clinical phenotype",\n    "triplosensitivity": "little evidence suggesting dosage sensitivity is associated with clinical phenotype",\n    "reciprocalOverlap": 0.00147,\n    "annotationOverlap": 0.33994\n},\n{\n    "chromosome": "15",\n    "begin": 31727418,\n    "end": 32153204,\n    "haploinsufficiency": "sufficient evidence suggesting dosage sensitivity is associated with clinical phenotype",\n    "triplosensitivity": "dosage sensitivity unlikely",\n    "reciprocalOverlap": 0.00147,\n    "annotationOverlap": 1\n}]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"clingenDosageSensitivityMap"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"object array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"chromosome"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"Ensembl-style chromosome names")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"begin"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"1-based position")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"end"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"integer"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"1-based position")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"haploinsufficiency"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"see possible values below")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"triplosensitivity"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"(same as haploinsufficiency)\xa0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"reciprocalOverlap"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"floating point"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"Range: 0 - 1. E.g. 0.57 would indicate a 57% reciprocal overlap. Specified up to 5 decimal places (Not reported for Insertions).")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"annotationOverlap"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"floating point"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"Range: 0 - 1. E.g. 0.57 would indicate a 57% reciprocal overlap. Specified up to 5 decimal places (Not reported for Insertions).")))),Object(b.b)("p",null,Object(b.b)("strong",{parentName:"p"},"haploinsufficiency and triplosensitivity")),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"no evidence to suggest that dosage sensitivity is associated with clinical phenotype"),Object(b.b)("li",{parentName:"ul"},"little evidence suggesting dosage sensitivity is associated with clinical phenotype"),Object(b.b)("li",{parentName:"ul"},"emerging evidence suggesting dosage sensitivity is associated with clinical phenotype"),Object(b.b)("li",{parentName:"ul"},"sufficient evidence suggesting dosage sensitivity is associated with clinical phenotype"),Object(b.b)("li",{parentName:"ul"},"gene associated with autosomal recessive phenotype"),Object(b.b)("li",{parentName:"ul"},"dosage sensitivity unlikely")))}j.isMDXComponent=!0},84:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/clingen-gene-validity-json",id:"data-sources/clingen-gene-validity-json",isDocsHomePage:!1,title:"clingen-gene-validity-json",description:"`json",source:"@site/docs/data-sources/clingen-gene-validity-json.md",slug:"/data-sources/clingen-gene-validity-json",permalink:"/NirvanaDocumentation/data-sources/clingen-gene-validity-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/clingen-gene-validity-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"clingenGeneValidity":[\n   {\n      "diseaseId":"MONDO_0007893",\n      "disease":"Noonan syndrome with multiple lentigines",\n      "classification":"no reported evidence",\n      "classificationDate":"2018-06-07"\n   },\n   {\n      "diseaseId":"MONDO_0015280",\n      "disease":"cardiofaciocutaneous syndrome",\n      "classification":"no reported evidence",\n      "classificationDate":"2018-06-07"\n   }\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"clingenGeneValidity"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"object"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"diseaseId"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"Monarch Disease Ontology ID (MONDO)")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"disease"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"disease label")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"classification"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"see below for possible values")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"classificationDate"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"yyyy-MM-dd")))),Object(b.b)("p",null,Object(b.b)("strong",{parentName:"p"},"classification")),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"no reported evidence"),Object(b.b)("li",{parentName:"ul"},"disputed"),Object(b.b)("li",{parentName:"ul"},"limited"),Object(b.b)("li",{parentName:"ul"},"moderate"),Object(b.b)("li",{parentName:"ul"},"definitive"),Object(b.b)("li",{parentName:"ul"},"strong"),Object(b.b)("li",{parentName:"ul"},"refuted")))}j.isMDXComponent=!0},85:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/amino-acid-conservation-json",id:"data-sources/amino-acid-conservation-json",isDocsHomePage:!1,title:"amino-acid-conservation-json",description:"`json",source:"@site/docs/data-sources/amino-acid-conservation-json.md",slug:"/data-sources/amino-acid-conservation-json",permalink:"/NirvanaDocumentation/data-sources/amino-acid-conservation-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/amino-acid-conservation-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"aminoAcidConservation": {\n    "scores": [0.34]\n}  \n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"aminoAcidConservation"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"object"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"scores"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"object array of doubles"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"percent conserved with respect to human amino acid residue. Range: 0.01 - 1.00")))))}j.isMDXComponent=!0},86:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/clinvar-json",id:"data-sources/clinvar-json",isDocsHomePage:!1,title:"clinvar-json",description:"`json",source:"@site/docs/data-sources/clinvar-json.md",slug:"/data-sources/clinvar-json",permalink:"/NirvanaDocumentation/data-sources/clinvar-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/clinvar-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"clinvar":[\n   {\n      "id":"VCV000036581.3",\n      "reviewStatus":"reviewed by expert panel",\n      "significance":[\n         "benign"\n      ],\n      "refAllele":"G",\n      "altAllele":"A",\n      "lastUpdatedDate":"2020-03-01",\n      "isAlleleSpecific":true\n   },\n   {\n      "id":"RCV000030258.4",\n      "variationId":"VCV000036581.3",\n      "reviewStatus":"reviewed by expert panel",\n      "alleleOrigins":[\n         "germline"\n      ],\n      "refAllele":"G",\n      "altAllele":"A",\n      "phenotypes":[\n         "Lynch syndrome"\n      ],\n      "medGenIds":[\n         "C1333990"\n      ],\n      "omimIds":[\n         "120435"\n      ],\n      "significance":[\n         "benign"\n      ],\n      "lastUpdatedDate":"2017-05-01",\n      "isAlleleSpecific":true\n   }\n]\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"id"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"ClinVar ID")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"variationId"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"ClinVar VCV ID")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"reviewStatus"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see possible values below")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"alleleOrigins"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see possible values below")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"refAllele"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"altAllele"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"phenotypes"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}))),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"medGenIds"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"MedGen IDs")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"omimIds"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"OMIM IDs")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"orphanetIds"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"Orphanet IDs")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"significance"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"see possible values below")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"lastUpdatedDate"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"yyyy-MM-dd")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"pubMedIds"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"string array"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"PubMed IDs")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"isAlleleSpecific"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"true when the current variant alternate allele matches the ClinVar alternate allele")))),Object(b.b)("p",null,Object(b.b)("strong",{parentName:"p"},"reviewStatus:")),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"no assertion provided"),Object(b.b)("li",{parentName:"ul"},"no assertion criteria provided"),Object(b.b)("li",{parentName:"ul"},"criteria provided, single submitter"),Object(b.b)("li",{parentName:"ul"},"practice guideline"),Object(b.b)("li",{parentName:"ul"},"classified by multiple submitters"),Object(b.b)("li",{parentName:"ul"},"criteria provided, conflicting interpretations"),Object(b.b)("li",{parentName:"ul"},"criteria provided, multiple submitters, no conflicts"),Object(b.b)("li",{parentName:"ul"},"no interpretation for the single variant")),Object(b.b)("p",null,Object(b.b)("strong",{parentName:"p"},"alleleOrigins:")),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"unknown"),Object(b.b)("li",{parentName:"ul"},"other"),Object(b.b)("li",{parentName:"ul"},"germline"),Object(b.b)("li",{parentName:"ul"},"somatic"),Object(b.b)("li",{parentName:"ul"},"inherited"),Object(b.b)("li",{parentName:"ul"},"paternal"),Object(b.b)("li",{parentName:"ul"},"maternal"),Object(b.b)("li",{parentName:"ul"},"de-novo"),Object(b.b)("li",{parentName:"ul"},"biparental"),Object(b.b)("li",{parentName:"ul"},"uniparental"),Object(b.b)("li",{parentName:"ul"},"not-tested"),Object(b.b)("li",{parentName:"ul"},"tested-inconclusive")),Object(b.b)("p",null,Object(b.b)("strong",{parentName:"p"},"significance:")),Object(b.b)("ul",null,Object(b.b)("li",{parentName:"ul"},"uncertain significance"),Object(b.b)("li",{parentName:"ul"},"not provided"),Object(b.b)("li",{parentName:"ul"},"benign"),Object(b.b)("li",{parentName:"ul"},"likely benign"),Object(b.b)("li",{parentName:"ul"},"likely pathogenic"),Object(b.b)("li",{parentName:"ul"},"pathogenic"),Object(b.b)("li",{parentName:"ul"},"drug response"),Object(b.b)("li",{parentName:"ul"},"histocompatibility"),Object(b.b)("li",{parentName:"ul"},"association"),Object(b.b)("li",{parentName:"ul"},"risk factor"),Object(b.b)("li",{parentName:"ul"},"protective"),Object(b.b)("li",{parentName:"ul"},"affects"),Object(b.b)("li",{parentName:"ul"},"conflicting data from submitters"),Object(b.b)("li",{parentName:"ul"},"other"),Object(b.b)("li",{parentName:"ul"},"no interpretation for the single variant"),Object(b.b)("li",{parentName:"ul"},"conflicting interpretations of pathogenicity")))}j.isMDXComponent=!0},87:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/gnomad-lof-json",id:"data-sources/gnomad-lof-json",isDocsHomePage:!1,title:"gnomad-lof-json",description:"`json",source:"@site/docs/data-sources/gnomad-lof-json.md",slug:"/data-sources/gnomad-lof-json",permalink:"/NirvanaDocumentation/data-sources/gnomad-lof-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/gnomad-lof-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"gnomAD":{ \n   "pLi":1.00e0,\n   "pNull":8.94e-40,\n   "pRec":1.84e-16,\n   "synZ":-8.44e-2,\n   "misZ":5.96e-1,\n   "loeuf":1.13e0\n}\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"pLi"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"probability of being intolerant of a single loss-of-function variant (like haploinsufficient genes, observed ~ 0.1*expected)")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"pNull"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"probability of being completely tolerant of loss of function variation (observed = expected)")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"pRec"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"probability of being intolerant of two loss of function variants (like recessive genes, observed ~ 0.5*expected)")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"synZ"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"corrected synonymous Z score")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"misZ"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"corrected missense Z score")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"loeuf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"loss of function observed/expected upper bound fraction (LOEUF)")))))}j.isMDXComponent=!0},88:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/gnomad-small-variants-json",id:"data-sources/gnomad-small-variants-json",isDocsHomePage:!1,title:"gnomad-small-variants-json",description:"`json",source:"@site/docs/data-sources/gnomad-small-variants-json.md",slug:"/data-sources/gnomad-small-variants-json",permalink:"/NirvanaDocumentation/data-sources/gnomad-small-variants-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/gnomad-small-variants-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"gnomad":{ \n   "coverage":20,\n   "allAf":0.190317,\n   "maleAf":0.193,\n   "femaleAf": 0.1935,\n   "afrAf":0.222876,\n   "amrAf":0.121394,\n   "easAf":0.239802,\n   "finAf":0.136833,\n   "nfeAf":0.181282,\n   "asjAf":0.258278,\n   "othAf":0.186094,\n   "allAn":30796,\n   "maleAn":15096,\n   "femaleAn":15700\n   "afrAn":8664,\n   "amrAn":832,\n   "easAn":1618,\n   "finAn":3486,\n   "nfeAn":14916,\n   "asjAn":302,\n   "othAn":978,\n   "allAc":5861,\n   "maleAc":2930,\n   "femaleAc": 2931,\n   "afrAc":1931,\n   "amrAc":101,\n   "easAc":388,\n   "finAc":477,\n   "nfeAc":2704,\n   "asjAc":78,\n   "othAc":182,\n   "allHc":561,\n   "afrHc":208,\n   "amrHc":6,\n   "easHc":42,\n   "finHc":31,\n   "nfeHc":242,\n   "asjHc":13,\n   "othHc":19,\n   "maleHc":280,\n   "femaleHc":281,\n   "controlsAllAf":0.190317,\n   "controlsAllAn":30796,\n   "controlsAllAc":5861,\n   "lowComplexityRegion":true,\n   "failedFilter":true\n}\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:null}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"coverage"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"average coverage (non-negative integer values)")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for all populations. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"maleAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for male population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"femaleAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for female population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"controlsAllAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the controls subset. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for all populations. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"maleAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for male population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"femaleAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for female population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"controlsAllAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for the controls subset. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for all populations. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"maleAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for male population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"femaleAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for female population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"controlsAllAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for the controls subset. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allHc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"count of homozygous individuals for all populations. Non-negative integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"maleHc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"count of homozygous individuals for male population. Non-negative integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"femaleHc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"count of homozygous individuals for female population. Non-negative integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"afrAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the African / African American population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"afrAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for the African / African American population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"afrAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for the African / African American population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"afrHc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"count of homozygous individuals for African / African American population. Non-negative integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"amrAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the Latino population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"amrAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for the Latino population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"amrAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for the Latino population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"amrHc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"count of homozygous individuals for Latino population. Non-negative integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"easAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the East Asian population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"easAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for the East Asian population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"easAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for the East Asian population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"easHc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"count of homozygous individuals for East Asian population. Non-negative integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"finAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the Finnish population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"finAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for the Finnish population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"finAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for the Finnish population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"finHc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"count of homozygous individuals for Finnish population. Non-negative integer")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"nfeAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the Non-Finnish European population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"nfeAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for the Non-Finnish European population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"nfeAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for the Non-Finnish European population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"nfeHc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"count of homozygous individuals for Non-Finnish European population. Non-negative integer")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"othAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the Other population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"othAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for the Other population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"othAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for the Other population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"othHc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"count of homozygous individuals for Other population. Non-negative integer")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"asjAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the Ashkenazi Jewish population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"asjAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for the Ashkenazi Jewish population Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"asjAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for the Ashkenazi Jewish population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"asjHc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"count of homozygous individuals for the Ashkenazi Jewish population. Non-negative integer")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"sasAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele frequency for the South Asian population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"sasAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele count for the South Asian population Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"sasAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"allele number for the South Asian population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"sasHc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"count of homozygous individuals for the South Asian population. Non-negative integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"failedFilter"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"True if this variant failed any filters (Note: we do not list the failed filters)")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"lowComplexityRegion"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"bool"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:null}),"True if this variant is located in a low complexity region.")))))}j.isMDXComponent=!0},89:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/phylop-json",id:"data-sources/phylop-json",isDocsHomePage:!1,title:"phylop-json",description:"`json",source:"@site/docs/data-sources/phylop-json.md",slug:"/data-sources/phylop-json",permalink:"/NirvanaDocumentation/data-sources/phylop-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/phylop-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"variants":[\n   {\n      "vid":"2:48010488:A",\n      "chromosome":"chr2",\n      "begin":48010488,\n      "end":48010488,\n      "refAllele":"G",\n      "altAllele":"A",\n      "variantType":"SNV",\n      "phylopScore":0.459\n  }\n]  \n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"phylopScore"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"range: -14.08 to 6.424")))))}j.isMDXComponent=!0},90:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return c})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return i})),a.d(t,"default",(function(){return j}));var n=a(3),r=a(7),b=(a(0),a(179)),c={},l={unversionedId:"data-sources/1000Genomes-snv-json",id:"data-sources/1000Genomes-snv-json",isDocsHomePage:!1,title:"1000Genomes-snv-json",description:"`json",source:"@site/docs/data-sources/1000Genomes-snv-json.md",slug:"/data-sources/1000Genomes-snv-json",permalink:"/NirvanaDocumentation/data-sources/1000Genomes-snv-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/1000Genomes-snv-json.md",version:"current"},i=[],o={toc:i};function j(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(b.b)("wrapper",Object(n.a)({},o,a,{components:t,mdxType:"MDXLayout"}),Object(b.b)("pre",null,Object(b.b)("code",Object(n.a)({parentName:"pre"},{className:"language-json"}),'"oneKg":{\n   "allAf":0.200879,\n   "afrAf":0.210287,\n   "amrAf":0.139769,\n   "easAf":0.275794,\n   "eurAf":0.181909,\n   "sasAf":0.173824,\n   "allAn":5008,\n   "afrAn":1322,\n   "amrAn":694,\n   "easAn":1008,\n   "eurAn":1006,\n   "sasAn":978,\n   "allAc":1006,\n   "afrAc":278,\n   "amrAc":97,\n   "easAc":278,\n   "eurAc":183,\n   "sasAc":170\n}\n')),Object(b.b)("table",null,Object(b.b)("thead",{parentName:"table"},Object(b.b)("tr",{parentName:"thead"},Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Field"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"center"}),"Type"),Object(b.b)("th",Object(n.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(b.b)("tbody",{parentName:"table"},Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele frequency for all populations. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele count for all populations. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele number for all populations. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"afrAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele frequency for the African super population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"afrAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele count for the African super population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"afrAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele number for the African super population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"amrAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele frequency for the Ad Mixed American super population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"amrAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele count for the Ad Mixed American super population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"amrAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele number for the Ad Mixed American super population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"easAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele frequency for the East Asian super population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"easAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele count for the East Asian super population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"easAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele number for the East Asian super population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"eurAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele frequency for the European super population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"eurAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele count for the European super population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"eurAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele number for the European super population. Non-zero integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"sasAf"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"float"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele frequency for the South Asian super population. Range: 0 - 1.0")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"sasAc"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele count for the South Asian super population. Integer.")),Object(b.b)("tr",{parentName:"tbody"},Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"sasAn"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"center"}),"int"),Object(b.b)("td",Object(n.a)({parentName:"tr"},{align:"left"}),"allele number for the South Asian super population. Non-zero integer.")))))}j.isMDXComponent=!0}}]);