(window.webpackJsonp=window.webpackJsonp||[]).push([[26,13],{125:function(e,t,n){"use strict";n.r(t),n.d(t,"frontMatter",(function(){return s})),n.d(t,"metadata",(function(){return c})),n.d(t,"rightToc",(function(){return o})),n.d(t,"default",(function(){return p}));var a=n(2),i=n(6),r=(n(0),n(132)),l=n(92),s={title:"ClinVar"},c={id:"data-sources/clinvar",title:"ClinVar",description:"RCV Files",source:"@site/docs/data-sources/clinvar.mdx",permalink:"/NirvanaDocumentation/data-sources/clinvar",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/clinvar.mdx",sidebar:"someSidebar",previous:{title:"Introduction",permalink:"/NirvanaDocumentation/introduction"},next:{title:"Nirvana JSON File Format",permalink:"/NirvanaDocumentation/file-formats/nirvana-json-file-format"}},o=[{value:"RCV Files",id:"rcv-files",children:[{value:"Example",id:"example",children:[]},{value:"Parsing",id:"parsing",children:[]}]},{value:"VCV Files",id:"vcv-files",children:[{value:"Example",id:"example-1",children:[]},{value:"Parsing",id:"parsing-1",children:[]}]},{value:"Parsing Significance",id:"parsing-significance",children:[]},{value:"Known Issues",id:"known-issues",children:[]},{value:"Download URL",id:"download-url",children:[]},{value:"JSON Output",id:"json-output",children:[]}],b={rightToc:o};function p(e){var t=e.components,n=Object(i.a)(e,["components"]);return Object(r.b)("wrapper",Object(a.a)({},b,n,{components:t,mdxType:"MDXLayout"}),Object(r.b)("h2",{id:"rcv-files"},"RCV Files"),Object(r.b)("h3",{id:"example"},"Example"),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml"}),'<?xml version="1.0" encoding="utf-8" standalone="yes"?>\n<ReleaseSet Dated="2016-07-04" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Type="full"\n            xsi:noNamespaceSchemaLocation="http://ftp.ncbi.nlm.nih.gov/pub/clinvar/xsd_public/clinvar_public_1.35.xsd">\n\n    <ClinVarSet ID="47064274">\n        <RecordStatus>current</RecordStatus>\n        <Title>NM_000235.4(LIPA):c.894+1G&gt;A AND Wolman disease</Title>\n        <ReferenceClinVarAssertion DateCreated="2012-08-13" DateLastUpdated="2019-11-02" ID="57604">\n            <ClinVarAccession Acc="RCV000000101" Version="3" Type="RCV" DateUpdated="2019-11-02"/>\n            <RecordStatus>current</RecordStatus>\n            <ClinicalSignificance DateLastEvaluated="1996-04-01">\n                <ReviewStatus>no assertion criteria provided</ReviewStatus>\n                <Description>Pathogenic</Description>\n            </ClinicalSignificance>\n            <Assertion Type="variation to disease"/>\n            <ObservedIn>\n                <Sample>\n                    <Origin>germline</Origin>\n                    <Species TaxonomyId="9606">human</Species>\n                    <AffectedStatus>not provided</AffectedStatus>\n                </Sample>\n                <Method>\n                    <MethodType>literature only</MethodType>\n                </Method>\n                <ObservedData ID="47897969">\n                    <Attribute Type="Description">In 2 sibs with Wolman disease (278000) from a consanguineous family,\n                        Aslanidis et al. (1996) detected homozygosity for a G-to-A mutation at position +1 of the splice\n                        donor site following exon 8 of the LIPA gene. Both children died within the first year of life.\n                        The parents, who were heterozygous for the mutation, had reduced enzymatic activity, while no\n                        enzymatic activity was detectable in fibroblasts from the affected children. Although the same\n                        donor splice site is involved as in the mutation reported in CESD (934G-A, 613497.0002), the\n                        nucleotide at position +1 was changed in the Wolman disease mutation while the nucleotide at\n                        position -1 was changed in the CESD mutation. Both mutations result in deletion of the same 24\n                        amino acids (exon 8), but the effects are dramatically different: the -1 mutation allowed some\n                        correct splicing (3% of total LIPA RNA), but the +1 splice site mutation, which affects one of\n                        the invariable nucleotides of the splice consensus sequences, permits no correct splicing.\n                        Aslanidis et al. (1996) suggested that the residual activity in CESD patients compared to Wolman\n                        patients may result either from a partially active enzyme with the internal deletion of 24 amino\n                        acids (skipping of exon 8) or from the production of low amounts of the full size of the protein\n                        due to inefficient exon exclusion from the mutated allele.\n                    </Attribute>\n                    <Citation Type="general">\n                        <ID Source="PubMed">8617513</ID>\n                    </Citation>\n                </ObservedData>\n            </ObservedIn>\n            <MeasureSet Type="Variant" ID="81" Acc="VCV000000081" Version="1">\n                <Measure Type="single nucleotide variant" ID="15120">\n                    <Name>\n                        <ElementValue Type="Preferred">NM_000235.4(LIPA):c.894+1G&gt;A</ElementValue>\n                    </Name>\n                    <AttributeSet>\n                        <Attribute Accession="NM_001288979" Version="1" Change="c.546+1G&gt;A"\n                                   Type="HGVS, coding, RefSeq">NM_001288979.1:c.546+1G&gt;A\n                        </Attribute>\n                    </AttributeSet>\n                    <AttributeSet>\n                        <Attribute Accession="NM_000235" Version="4" Change="c.894+1G&gt;A" Type="HGVS, coding, RefSeq">\n                            NM_000235.4:c.894+1G&gt;A\n                        </Attribute>\n                    </AttributeSet>\n                    <AttributeSet>\n                        <Attribute Accession="NM_001127605" Version="3" Change="c.894+1G&gt;A"\n                                   Type="HGVS, coding, RefSeq">NM_001127605.3:c.894+1G&gt;A\n                        </Attribute>\n                    </AttributeSet>\n                    <AttributeSet>\n                        <Attribute Accession="NG_008194" Version="1" Change="g.34394G&gt;A"\n                                   Type="HGVS, genomic, RefSeqGene">NG_008194.1:g.34394G&gt;A\n                        </Attribute>\n                    </AttributeSet>\n                    <AttributeSet>\n                        <Attribute Accession="NC_000010" Version="11" Change="g.89222510C&gt;T"\n                                   Type="HGVS, genomic, top level" integerValue="38">NC_000010.11:g.89222510C&gt;T\n                        </Attribute>\n                    </AttributeSet>\n                    <AttributeSet>\n                        <Attribute Accession="NC_000010" Version="10" Change="g.90982267C&gt;T"\n                                   Type="HGVS, genomic, top level, previous" integerValue="37">NC_000010.10:g.90982267C&gt;T\n                        </Attribute>\n                    </AttributeSet>\n                    <AttributeSet>\n                        <Attribute Accession="NM_000235" Version="3" Change="c.894+1G&gt;A" Type="HGVS, previous">\n                            NM_000235.3:c.894+1G&gt;A\n                        </Attribute>\n                    </AttributeSet>\n                    <AttributeSet>\n                        <Attribute Type="MolecularConsequence">splice donor variant</Attribute>\n                        <XRef ID="SO:0001575" DB="Sequence Ontology"/>\n                        <XRef ID="NM_000235.4:c.894+1G&gt;A" DB="RefSeq"/>\n                    </AttributeSet>\n                    <AttributeSet>\n                        <Attribute Type="MolecularConsequence">splice donor variant</Attribute>\n                        <XRef ID="SO:0001575" DB="Sequence Ontology"/>\n                        <XRef ID="NM_001127605.3:c.894+1G&gt;A" DB="RefSeq"/>\n                    </AttributeSet>\n                    <AttributeSet>\n                        <Attribute Type="MolecularConsequence">splice donor variant</Attribute>\n                        <XRef ID="SO:0001575" DB="Sequence Ontology"/>\n                        <XRef ID="NM_001288979.1:c.546+1G&gt;A" DB="RefSeq"/>\n                    </AttributeSet>\n                    <AttributeSet>\n                        <Attribute Type="nucleotide change">IVS8, G-A, +1</Attribute>\n                    </AttributeSet>\n                    <CytogeneticLocation>10q23.31</CytogeneticLocation>\n                    <SequenceLocation Assembly="GRCh38" AssemblyAccessionVersion="GCF_000001405.38"\n                                      AssemblyStatus="current" Chr="10" Accession="NC_000010.11" start="89222510"\n                                      stop="89222510" display_start="89222510" display_stop="89222510" variantLength="1"\n                                      positionVCF="89222510" referenceAlleleVCF="C" alternateAlleleVCF="T"/>\n                    <SequenceLocation Assembly="GRCh37" AssemblyAccessionVersion="GCF_000001405.25"\n                                      AssemblyStatus="previous" Chr="10" Accession="NC_000010.10" start="90982267"\n                                      stop="90982267" display_start="90982267" display_stop="90982267" variantLength="1"\n                                      positionVCF="90982267" referenceAlleleVCF="C" alternateAlleleVCF="T"/>\n                    <MeasureRelationship Type="within single gene">\n                        <Name>\n                            <ElementValue Type="Preferred">lipase A, lysosomal acid type</ElementValue>\n                        </Name>\n                        <Symbol>\n                            <ElementValue Type="Preferred">LIPA</ElementValue>\n                        </Symbol>\n                        <SequenceLocation Assembly="GRCh38" AssemblyAccessionVersion="GCF_000001405.38"\n                                          AssemblyStatus="current" Chr="10" Accession="NC_000010.11" start="89213569"\n                                          stop="89252039" display_start="89213569" display_stop="89252039" Strand="-"/>\n                        <SequenceLocation Assembly="GRCh37" AssemblyAccessionVersion="GCF_000001405.25"\n                                          AssemblyStatus="previous" Chr="10" Accession="NC_000010.10" start="90973325"\n                                          stop="91011659" display_start="90973325" display_stop="91011659"\n                                          variantLength="38335" Strand="-"/>\n                        <XRef ID="3988" DB="Gene"/>\n                        <XRef Type="MIM" ID="613497" DB="OMIM"/>\n                        <XRef ID="HGNC:6617" DB="HGNC"/>\n                    </MeasureRelationship>\n                    <XRef Type="Allelic variant" ID="613497.0005" DB="OMIM"/>\n                    <XRef Type="rs" ID="1564751995" DB="dbSNP"/>\n                    <Comment DataSource="ClinGen" Type="public">ClinGen staff contributed the HGVS expression for this\n                        variant.\n                    </Comment>\n                </Measure>\n                <Name>\n                    <ElementValue Type="Preferred">NM_000235.4(LIPA):c.894+1G&gt;A</ElementValue>\n                </Name>\n            </MeasureSet>\n            <TraitSet Type="Disease" ID="40">\n                <Trait ID="9048" Type="Disease">\n                    <Name>\n                        <ElementValue Type="Preferred">Wolman disease</ElementValue>\n                        <XRef ID="7899" DB="Office of Rare Diseases"/>\n                    </Name>\n                    <XRef ID="CN438428" DB="MedGen"/>\n                </Trait>\n            </TraitSet>\n        </ReferenceClinVarAssertion>\n        <ClinVarAssertion ID="20244">\n            <ClinVarSubmissionID localKey="613497.0005_WOLMAN DISEASE" submitter="OMIM" submitterDate="2017-12-22"\n                                 title="LIPA, IVS8, G-A, +1_WOLMAN DISEASE"/>\n            <ClinVarAccession Acc="SCV000020244" Version="2" Type="SCV" OrgID="3" OrganizationCategory="resource"\n                              OrgType="primary" DateUpdated="2019-03-31"/>\n            <RecordStatus>current</RecordStatus>\n            <ClinicalSignificance DateLastEvaluated="1996-04-01">\n                <ReviewStatus>no assertion criteria provided</ReviewStatus>\n                <Description>Pathogenic</Description>\n            </ClinicalSignificance>\n            <Assertion Type="variation to disease"/>\n            <ExternalID DB="OMIM" ID="613497.0005" Type="Allelic variant"/>\n            <ObservedIn>\n                <Sample>\n                    <Origin>germline</Origin>\n                    <Species>human</Species>\n                    <AffectedStatus>not provided</AffectedStatus>\n                </Sample>\n                <Method>\n                    <MethodType>literature only</MethodType>\n                </Method>\n                <ObservedData>\n                    <Attribute Type="Description">In 2 sibs with Wolman disease (278000) from a consanguineous family,\n                        Aslanidis et al. (1996) detected homozygosity for a G-to-A mutation at position +1 of the splice\n                        donor site following exon 8 of the LIPA gene. Both children died within the first year of life.\n                        The parents, who were heterozygous for the mutation, had reduced enzymatic activity, while no\n                        enzymatic activity was detectable in fibroblasts from the affected children. Although the same\n                        donor splice site is involved as in the mutation reported in CESD (934G-A, 613497.0002), the\n                        nucleotide at position +1 was changed in the Wolman disease mutation while the nucleotide at\n                        position -1 was changed in the CESD mutation. Both mutations result in deletion of the same 24\n                        amino acids (exon 8), but the effects are dramatically different: the -1 mutation allowed some\n                        correct splicing (3% of total LIPA RNA), but the +1 splice site mutation, which affects one of\n                        the invariable nucleotides of the splice consensus sequences, permits no correct splicing.\n                        Aslanidis et al. (1996) suggested that the residual activity in CESD patients compared to Wolman\n                        patients may result either from a partially active enzyme with the internal deletion of 24 amino\n                        acids (skipping of exon 8) or from the production of low amounts of the full size of the protein\n                        due to inefficient exon exclusion from the mutated allele.\n                    </Attribute>\n                    <Citation>\n                        <ID Source="PubMed">8617513</ID>\n                    </Citation>\n                    <XRef DB="OMIM" ID="278000" Type="MIM"/>\n                </ObservedData>\n            </ObservedIn>\n            <MeasureSet Type="Variant">\n                <Measure Type="Variation">\n                    <Name>\n                        <ElementValue Type="Preferred">LIPA, IVS8, G-A, +1</ElementValue>\n                    </Name>\n                    <AttributeSet>\n                        <Attribute Type="NonHGVS">IVS8, G-A, +1</Attribute>\n                    </AttributeSet>\n                    <MeasureRelationship Type="variant in gene">\n                        <Symbol>\n                            <ElementValue Type="Preferred">LIPA</ElementValue>\n                        </Symbol>\n                    </MeasureRelationship>\n                    <XRef DB="OMIM" ID="613497.0005" Type="Allelic variant"/>\n                </Measure>\n            </MeasureSet>\n            <TraitSet Type="Disease">\n                <Trait Type="Disease">\n                    <Name>\n                        <ElementValue Type="Preferred">WOLMAN DISEASE</ElementValue>\n                    </Name>\n                </Trait>\n            </TraitSet>\n        </ClinVarAssertion>\n    </ClinVarSet>\n\n</ReleaseSet>\n')),Object(r.b)("h3",{id:"parsing"},"Parsing"),Object(r.b)("p",null,"In the following section, we discuss which field of the XML was used to extract information that is presented in the JSON output."),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"ID")),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml",metastring:"{3}","{3}":!0}),'<ClinVarSet>\n   <ReferenceClinVarAssertion>    \n       <ClinVarAccession Acc="RCV000000001" Version="2">\n</ClinVarSet>\n')),Object(r.b)("p",null,"The Acc and Version fields are merged to form the ID (RCV000000001.2)"),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"LastUpdatedDate")),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml",metastring:"{2}","{2}":!0}),'<ClinVarSet>\n   <ReferenceClinVarAssertion DateCreated="2012-08-13" DateLastUpdated="2016-02-17" ID="57604" >\n</ClinVarSet>\n')),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"Significance")),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml",metastring:"{5}","{5}":!0}),'<ClinVarSet>\n   <ReferenceClinVarAssertion>\n       <ClinicalSignificance DateLastEvaluated="1996-04-01">        \n           <ReviewStatus>no assertion criteria provided</ReviewStatus>              \n           <Description>Pathogenic</Description>            \n       </ClinicalSignificance>\n</ClinVarSet>\n')),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"ReviewStatus")),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml",metastring:"{4}","{4}":!0}),'<ClinVarSet>\n   <ReferenceClinVarAssertion>    \n       <ClinicalSignificance DateLastEvaluated="1996-04-01">        \n           <ReviewStatus>no assertion criteria provided</ReviewStatus>             \n           <Description>Pathogenic</Description>            \n       </ClinicalSignificance>\n</ClinVarSet>\n')),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"Phenotypes")),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml",metastring:"{2-8}","{2-8}":!0}),'<ReferenceClinVarAssertion>\n   <TraitSet Type="Disease" ID="62">\n       <Trait Type="Disease">\n            <Name>\n               <ElementValue Type="Preferred">Joubert syndrome 9</ElementValue>\n            </Name>\n        </Trait>\n   </TraitSet>\n</ReferenceClinVarAssertion>\n')),Object(r.b)("p",null,'We only use the field with Type="Preferred". Multiple phenotypes may be reported'),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"Location and Variant Id")),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml",metastring:"{3,5-12}","{3,5-12}":!0}),'<ReferenceClinVarAssertion>\n<GenotypeSet Type="CompoundHeterozygote" ID="424709">\n   <MeasureSet Type="Variant" ID="81">\n       <Measure Type="single nucleotide variant" ID="15120">\n        <SequenceLocation Assembly="GRCh38" AssemblyAccessionVersion="GCF_000001405.38"  \n          AssemblyStatus="current" Chr="10" Accession="NC_000010.11" start="89222510"  \n          stop="89222510" display_start="89222510" display_stop="89222510" variantLength="1"  \n          positionVCF="89222510" referenceAlleleVCF="C" alternateAlleleVCF="T"/>  \n        <SequenceLocation Assembly="GRCh37" AssemblyAccessionVersion="GCF_000001405.25"  \n          AssemblyStatus="previous" Chr="10" Accession="NC_000010.10" start="90982267"  \n          stop="90982267" display_start="90982267" display_stop="90982267" variantLength="1"  \n          positionVCF="90982267" referenceAlleleVCF="C" alternateAlleleVCF="T"/>\n       </Measure>\n   </MeasureSet>\n </GenotypeSet>\n</ReferenceClinVarAssertion>\n')),Object(r.b)("ul",null,Object(r.b)("li",{parentName:"ul"},"The variant position is extracted from the fields for their respective assemblies."),Object(r.b)("li",{parentName:"ul"},"Updated records contain positionVCF, referenceAlleleVCF and alternateAlleleVCF fields and when present, we use them to create the variant."),Object(r.b)("li",{parentName:"ul"},'For older records, since "start\' and "stop" fields are not always available, we use the "display_start" and "display_end" fields.'),Object(r.b)("li",{parentName:"ul"},"If a required allele is not available, we extract it from the reference sequence."),Object(r.b)("li",{parentName:"ul"},"Only variants having a dbSNP id are extracted."),Object(r.b)("li",{parentName:"ul"},"Note that a ClinVar accession may have multiple variants associated with it (possible in different locations)"),Object(r.b)("li",{parentName:"ul"},"VariantId is extracted from the MeasureSet attributes.")),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"MedGen, OMIM, Orphanet IDs")),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml",metastring:"{4-7}","{4-7}":!0}),'<ReferenceClinVarAssertion>\n   <TraitSet Type="Disease" ID="175">\n       <Trait ID="3036" Type="Disease">\n           <XRef ID="C0086651" DB="MedGen"/>\n           <XRef ID="309297" DB="Orphanet"/>\n           <XRef ID="582" DB="Orphanet"/>\n           <XRef Type="MIM" ID="253000" DB="OMIM"/>\n       </Trait>\n   </TraitSet>\n</ReferenceClinVarAssertion>\n')),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"AlleleOrigins")),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml",metastring:"{2}","{2}":!0}),"<ClinVarAssertion>\n     <Origin>germline</Origin>\n</ClinVarAssertion>\n")),Object(r.b)("p",null,"We only extract all Allele Origins from Submissions (SCV) entries."),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"PubMedIds")),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml",metastring:"{4,10,16,21}","{4,10,16,21}":!0}),'<ClinVarAssertion>\n   <ClinicalSignificance DateLastEvaluated="1996-04-01">\n       <Citation Type="general">\n           <ID Source="PubMed">12114475</ID>\n       </Citation>\n   </ClinicalSignificance>\n   <AttributeSet>\n       <Attribute Type="AssertionMethod">LMM Criteria</Attribute>\n       <Citation>\n            <ID Source="PubMed">24033266</ID>\n       </Citation>\n   </AttributeSet>\n   <ObservedIn>\n       <ObservedData ID="9727445">\n           <Citation Type="general">\n               <ID Source="PubMed">9113933</ID>\n           </Citation>\n       </ObservedData>\n   </ObservedIn>\n   <Citation Type="general">\n       <ID Source="PubMed">23757202</ID>\n   </Citation>\n</ClinVarAssertion>\n')),Object(r.b)("p",null,"We only extract all Pubmed Ids from Submissions (SCV) entries."),Object(r.b)("h2",{id:"vcv-files"},"VCV Files"),Object(r.b)("h3",{id:"example-1"},"Example"),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml"}),'<?xml version="1.0" encoding="UTF-8" standalone="yes"?>\n<ClinVarVariationRelease xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://ftp.ncbi.nlm.nih.gov/pub/clinvar/xsd_public/clinvar_variation/variation_archive_1.4.xsd" ReleaseDate="2019-12-31">\n<VariationArchive VariationID="431749" VariationName="GRCh37/hg19 1p36.31(chr1:6051187-6158763)" VariationType="copy number gain" DateCreated="2017-08-12" DateLastUpdated="2019-09-10" Accession="VCV000431749" Version="1" RecordType="included" NumberOfSubmissions="0" NumberOfSubmitters="0">\n  <RecordStatus>current</RecordStatus>\n  <Species>Homo sapiens</Species>\n  <IncludedRecord>\n    <SimpleAllele AlleleID="425239" VariationID="431749">\n      <GeneList>\n        <Gene Symbol="KCNAB2" FullName="potassium voltage-gated channel subfamily A regulatory beta subunit 2" GeneID="8514" HGNC_ID="HGNC:6229" Source="calculated" RelationshipType="genes overlapped by variant">\n          <Location>\n            <CytogeneticLocation>1p36.31</CytogeneticLocation>\n            <SequenceLocation Assembly="GRCh38" AssemblyAccessionVersion="GCF_000001405.38" AssemblyStatus="current" Chr="1" Accession="NC_000001.11" start="5992639" stop="6101186" display_start="5992639" display_stop="6101186" Strand="+"/>\n            <SequenceLocation Assembly="GRCh37" AssemblyAccessionVersion="GCF_000001405.25" AssemblyStatus="previous" Chr="1" Accession="NC_000001.10" start="6052357" stop="6161252" display_start="6052357" display_stop="6161252" Strand="+"/>\n          </Location>\n          <OMIM>601142</OMIM>\n        </Gene>\n        <Gene Symbol="NPHP4" FullName="nephrocystin 4" GeneID="261734" HGNC_ID="HGNC:19104" Source="calculated" RelationshipType="genes overlapped by variant">\n          <Location>\n            <CytogeneticLocation>1p36.31</CytogeneticLocation>\n            <SequenceLocation Assembly="GRCh38" AssemblyAccessionVersion="GCF_000001405.38" AssemblyStatus="current" Chr="1" Accession="NC_000001.11" start="5862810" stop="5992425" display_start="5862810" display_stop="5992425" Strand="-"/>\n            <SequenceLocation Assembly="GRCh37" AssemblyAccessionVersion="GCF_000001405.25" AssemblyStatus="previous" Chr="1" Accession="NC_000001.10" start="5922869" stop="6052532" display_start="5922869" display_stop="6052532" Strand="-"/>\n          </Location>\n          <OMIM>607215</OMIM>\n        </Gene>\n      </GeneList>\n      <Name>GRCh37/hg19 1p36.31(chr1:6051187-6158763)</Name>\n      <VariantType>copy number gain</VariantType>\n      <Location>\n        <CytogeneticLocation>1p36.31</CytogeneticLocation>\n        <SequenceLocation Assembly="GRCh37" AssemblyAccessionVersion="GCF_000001405.25" forDisplay="true" AssemblyStatus="previous" Chr="1" Accession="NC_000001.10" start="6051187" stop="6158763" display_start="6051187" display_stop="6158763"/>      </Location>\n      <Interpretations>\n        <Interpretation NumberOfSubmissions="0" NumberOfSubmitters="0" Type="Clinical significance">\n          <Description>no interpretation for the single variant</Description>\n        </Interpretation>\n      </Interpretations>\n      <XRefList>\n        <XRef Type="Interpreted" ID="431733" DB="ClinVar"/>\n      </XRefList>\n    </SimpleAllele>\n    <ReviewStatus>no interpretation for the single variant</ReviewStatus>\n    <Interpretations>\n      <Interpretation NumberOfSubmissions="0" NumberOfSubmitters="0" Type="Clinical significance">\n        <Description>no interpretation for the single variant</Description>\n      </Interpretation>\n    </Interpretations>\n    <SubmittedInterpretationList>\n      <SCV Title="SUB1895145" Accession="SCV000296057" Version="1"/>\n    </SubmittedInterpretationList>\n    <InterpretedVariationList>\n      <InterpretedVariation VariationID="431733" Accession="VCV000431733" Version="1"/>\n    </InterpretedVariationList>\n  </IncludedRecord>\n</VariationArchive>\n</ClinVarVariationRelease>\n')),Object(r.b)("h3",{id:"parsing-1"},"Parsing"),Object(r.b)("p",null,"In the following section, we discuss which field of the XML was used to extract information that is presented in the JSON output."),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"id")),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml"}),'<VariationArchive VariationID="431749" VariationName="GRCh37/hg19 1p36.31(chr1:6051187-6158763)" VariationType="copy number gain" DateCreated="2017-08-12" DateLastUpdated="2019-09-10" Accession="VCV000431749" Version="1" RecordType="included" NumberOfSubmissions="0" NumberOfSubmitters="0">\n')),Object(r.b)("p",null,"The Acc and Version fields are merged to form the ID (RCV000000001.2)"),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"significance")),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml",metastring:"{7}","{7}":!0}),'<ClinVarVariationRelease>\n  <VariationArchive>\n    <IncludedRecord>\n      <SimpleAllele>\n        <Interpretations>\n          <Interpretation NumberOfSubmissions="0" NumberOfSubmitters="0" Type="Clinical significance">\n            <Description>no interpretation for the single variant</Description>\n          </Interpretation>\n        </Interpretations>\n      </SimpleAllele>\n    </IncludedRecord>\n  </VariationArchive>\n</ClinVarVariationRelease>\n')),Object(r.b)("p",null,"May have multiple significances listed."),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"reviewStatus")),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml",metastring:"{4}","{4}":!0}),"<ClinVarVariationRelease>\n  <VariationArchive>\n    <IncludedRecord>\n      <ReviewStatus>no interpretation for the single variant</ReviewStatus>\n    </IncludedRecord>\n  </VariationArchive>\n</ClinVarVariationRelease>\n")),Object(r.b)("h2",{id:"parsing-significance"},"Parsing Significance"),Object(r.b)("p",null,"Extracting significance(s) may involve parsing multiple fields. Take the following snippets into consideration."),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-xml",metastring:"{3,8,13-14}","{3,8,13-14}":!0}),'<ClinicalSignificance DateLastEvaluated="1996-04-01">\n    <ReviewStatus>no assertion criteria provided</ReviewStatus>\n    <Description>Pathogenic</Description>\n</ClinicalSignificance>\n\n<ClinicalSignificance DateLastEvaluated="2016-10-13">\n    <ReviewStatus>criteria provided, multiple submitters, no conflicts</ReviewStatus>\n    <Description>Pathogenic/Likely pathogenic</Description>\n</ClinicalSignificance>\n\n<ClinicalSignificance DateLastEvaluated="2012-06-07">\n    <ReviewStatus>no assertion criteria provided</ReviewStatus>\n    <Description>Conflicting interpretations of pathogenicity</Description>\n    <Explanation DataSource="ClinVar" Type="public">Pathogenic(1);Uncertain significance(1)</Explanation>\n</ClinicalSignificance>\n')),Object(r.b)("p",null,"Given the evidence, we converted significance field into an array of string which may be parsed out of the Descriptions or Explanation fields. Note that the delimiters in each field may vary. Currently, the delimiters for Description are ',' and '/'. For, Explanation we use ';' and '/'."),Object(r.b)("h2",{id:"known-issues"},"Known Issues"),Object(r.b)("div",{className:"admonition admonition-caution alert alert--warning"},Object(r.b)("div",Object(a.a)({parentName:"div"},{className:"admonition-heading"}),Object(r.b)("h5",{parentName:"div"},Object(r.b)("span",Object(a.a)({parentName:"h5"},{className:"admonition-icon"}),Object(r.b)("svg",Object(a.a)({parentName:"span"},{xmlns:"http://www.w3.org/2000/svg",width:"16",height:"16",viewBox:"0 0 16 16"}),Object(r.b)("path",Object(a.a)({parentName:"svg"},{fillRule:"evenodd",d:"M8.893 1.5c-.183-.31-.52-.5-.887-.5s-.703.19-.886.5L.138 13.499a.98.98 0 0 0 0 1.001c.193.31.53.501.886.501h13.964c.367 0 .704-.19.877-.5a1.03 1.03 0 0 0 .01-1.002L8.893 1.5zm.133 11.497H6.987v-2.003h2.039v2.003zm0-3.004H6.987V5.987h2.039v4.006z"})))),"Known Issues")),Object(r.b)("div",Object(a.a)({parentName:"div"},{className:"admonition-content"}))),Object(r.b)("ul",null,Object(r.b)("li",{parentName:"ul"},"XML contains ~1k entries more (out of 162K) compared to the VCF file."),Object(r.b)("li",{parentName:"ul"},"Web verification shows a 2.45% discrepancies in field values."),Object(r.b)("li",{parentName:"ul"},"XML had no field indicating a record to be associated with reference, something that was present in VCF."),Object(r.b)("li",{parentName:"ul"},'XML contains entries (e.g. RCV000016645 version=1) which have irregular bases ("R", "Y", "H", etc.) as their alternate alleles. These entries are not present in VCF.')),Object(r.b)("h2",{id:"download-url"},"Download URL"),Object(r.b)("p",null,Object(r.b)("a",Object(a.a)({parentName:"p"},{href:"ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/ClinVarFullRelease_00-latest.xml.gz"}),"ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/ClinVarFullRelease_00-latest.xml.gz")),Object(r.b)("h2",{id:"json-output"},"JSON Output"),Object(r.b)(l.default,{mdxType:"JSON"}))}p.isMDXComponent=!0},92:function(e,t,n){"use strict";n.r(t),n.d(t,"frontMatter",(function(){return l})),n.d(t,"metadata",(function(){return s})),n.d(t,"rightToc",(function(){return c})),n.d(t,"default",(function(){return b}));var a=n(2),i=n(6),r=(n(0),n(132)),l={},s={id:"data-sources/clinvar-json",title:"clinvar-json",description:"`json",source:"@site/docs/data-sources/clinvar-json.md",permalink:"/NirvanaDocumentation/data-sources/clinvar-json",editUrl:"https://github.com/Illumina/NirvanaDocumentation/edit/master/docs/data-sources/clinvar-json.md"},c=[],o={rightToc:c};function b(e){var t=e.components,n=Object(i.a)(e,["components"]);return Object(r.b)("wrapper",Object(a.a)({},o,n,{components:t,mdxType:"MDXLayout"}),Object(r.b)("pre",null,Object(r.b)("code",Object(a.a)({parentName:"pre"},{className:"language-json"}),'"clinvar":[\n   {\n      "id":"VCV000036581.3",\n      "reviewStatus":"reviewed by expert panel",\n      "significance":[\n         "benign"\n      ],\n      "refAllele":"G",\n      "altAllele":"A",\n      "lastUpdatedDate":"2020-03-01",\n      "isAlleleSpecific":true\n   },\n   {\n      "id":"RCV000030258.4",\n      "variationId":"VCV000036581.3",\n      "reviewStatus":"reviewed by expert panel",\n      "alleleOrigins":[\n         "germline"\n      ],\n      "refAllele":"G",\n      "altAllele":"A",\n      "phenotypes":[\n         "Lynch syndrome"\n      ],\n      "medGenIds":[\n         "C1333990"\n      ],\n      "omimIds":[\n         "120435"\n      ],\n      "significance":[\n         "benign"\n      ],\n      "lastUpdatedDate":"2017-05-01",\n      "isAlleleSpecific":true\n   },\n   {\n      "id":"RCV000144626.1",\n      "variationId":"VCV000036581.3",\n      "reviewStatus":"no assertion criteria provided",\n      "alleleOrigins":[\n         "germline"\n      ],\n      "refAllele":"G",\n      "altAllele":"A",\n      "phenotypes":[\n         "Lynch syndrome I"\n      ],\n      "medGenIds":[\n         "C2936783"\n      ],\n      "omimIds":[\n         "120435"\n      ],\n      "orphanetIds":[\n         "144"\n      ],\n      "significance":[\n         "benign"\n      ],\n      "lastUpdatedDate":"2017-04-05",\n      "pubMedIds":[\n         "18523027",\n         "22703879",\n         "22949387"\n      ],\n      "isAlleleSpecific":true\n   }\n]\n')),Object(r.b)("table",null,Object(r.b)("thead",{parentName:"table"},Object(r.b)("tr",{parentName:"thead"},Object(r.b)("th",Object(a.a)({parentName:"tr"},{align:"left"}),"Field"),Object(r.b)("th",Object(a.a)({parentName:"tr"},{align:"center"}),"Type"),Object(r.b)("th",Object(a.a)({parentName:"tr"},{align:"center"}),"Required"),Object(r.b)("th",Object(a.a)({parentName:"tr"},{align:"left"}),"Notes"))),Object(r.b)("tbody",{parentName:"table"},Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"id"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"Yes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"ClinVar ID")),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"variationId"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"Yes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"ClinVar VCV ID")),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"reviewStatus"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"Yes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"see possible values below")),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"alleleOrigins"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string array"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"Yes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"see possible values below")),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"refAllele"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"Yes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}))),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"altAllele"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"Yes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}))),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"phenotypes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string array"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"Yes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}))),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"medGenIds"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string array"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"Yes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"MedGen IDs")),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"omimIds"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string array"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"Yes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"OMIM IDs")),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"orphanetIds"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string array"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"Yes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"Orphanet IDs")),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"significance"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string array"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"Yes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"see possible values below")),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"lastUpdatedDate"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"No"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"yyyy-MM-dd")),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"pubMedIds"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"string array"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"Yes"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"PubMed IDs")),Object(r.b)("tr",{parentName:"tbody"},Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"isAlleleSpecific"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"bool"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"center"}),"No"),Object(r.b)("td",Object(a.a)({parentName:"tr"},{align:"left"}),"set to true if the current variant alternate allele matches the ClinVar alternate allele")))),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"reviewStatus:")),Object(r.b)("ul",null,Object(r.b)("li",{parentName:"ul"},"no assertion provided"),Object(r.b)("li",{parentName:"ul"},"no assertion criteria provided"),Object(r.b)("li",{parentName:"ul"},"criteria provided, single submitter"),Object(r.b)("li",{parentName:"ul"},"practice guideline"),Object(r.b)("li",{parentName:"ul"},"classified by multiple submitters"),Object(r.b)("li",{parentName:"ul"},"criteria provided, conflicting interpretations"),Object(r.b)("li",{parentName:"ul"},"criteria provided, multiple submitters, no conflicts"),Object(r.b)("li",{parentName:"ul"},"no interpretation for the single variant")),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"alleleOrigins:")),Object(r.b)("ul",null,Object(r.b)("li",{parentName:"ul"},"unknown"),Object(r.b)("li",{parentName:"ul"},"other"),Object(r.b)("li",{parentName:"ul"},"germline"),Object(r.b)("li",{parentName:"ul"},"somatic"),Object(r.b)("li",{parentName:"ul"},"inherited"),Object(r.b)("li",{parentName:"ul"},"paternal"),Object(r.b)("li",{parentName:"ul"},"maternal"),Object(r.b)("li",{parentName:"ul"},"de-novo"),Object(r.b)("li",{parentName:"ul"},"biparental"),Object(r.b)("li",{parentName:"ul"},"uniparental"),Object(r.b)("li",{parentName:"ul"},"not-tested"),Object(r.b)("li",{parentName:"ul"},"tested-inconclusive")),Object(r.b)("p",null,Object(r.b)("strong",{parentName:"p"},"significance:")),Object(r.b)("ul",null,Object(r.b)("li",{parentName:"ul"},"uncertain significance"),Object(r.b)("li",{parentName:"ul"},"not provided"),Object(r.b)("li",{parentName:"ul"},"benign"),Object(r.b)("li",{parentName:"ul"},"likely benign"),Object(r.b)("li",{parentName:"ul"},"likely pathogenic"),Object(r.b)("li",{parentName:"ul"},"pathogenic"),Object(r.b)("li",{parentName:"ul"},"drug response"),Object(r.b)("li",{parentName:"ul"},"histocompatibility"),Object(r.b)("li",{parentName:"ul"},"association"),Object(r.b)("li",{parentName:"ul"},"risk factor"),Object(r.b)("li",{parentName:"ul"},"protective"),Object(r.b)("li",{parentName:"ul"},"affects"),Object(r.b)("li",{parentName:"ul"},"conflicting data from submitters"),Object(r.b)("li",{parentName:"ul"},"other"),Object(r.b)("li",{parentName:"ul"},"no interpretation for the single variant"),Object(r.b)("li",{parentName:"ul"},"conflicting interpretations of pathogenicity")))}b.isMDXComponent=!0}}]);