{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Parse Nirvana JSON output in python\n",
    "\n",
    "Nirvana outputs a single JSON annotation file for a single input VCF file. The output file contains a single [JSON object](https://www.w3schools.com/js/js_json_objects.asp) to represent the annotations of all input VCF variants. The JSON file format can be found in the [documentation](https://illumina.github.io/NirvanaDocumentation/file-formats/nirvana-json-file-format).\n",
    "\n",
    "A [sample VCF file](https://github.com/Illumina/NirvanaDocumentation/blob/master/static/files/ceph_trio_test.json.gz) of the CEPH trio (NA12878, NA12891, and NA12892) can be downloaded from the [NirvanaDocumention](https://github.com/Illumina/NirvanaDocumentation/tree/master/static/files) git repo.\n",
    "\n",
    "This notebook demonstrates how you can parse this sample JSON file in python and retrieves the annotation data.\n",
    "\n",
    "## Read Nirvana JSON output by lines\n",
    "\n",
    "Even though the Nirvana JSON output is a single JSON object, different JSON object fields are written in different lines for memory efficient reading.\n",
    "\n",
    "The first line in the JSON file is the `header` line:\n",
    "\n",
    "```json\n",
    "{\"header\":{\"annotator\":\"Nirvana 3.13.0\",\"creationTime\":\"2020-12-18 19:43:53\",\"genomeAssembly\":\"GRCh38\",\n",
    "\"schemaVersion\":6,\"dataVersion\":\"91.27.61\",\"dataSources\":[{\"name\":\"VEP\",\"version\":\"91\",\"description\":\"B\n",
    "othRefSeqAndEnsembl\",\"releaseDate\":\"2017-12-18\"},{\"name\":\"MultiZ100Way\",\"version\":\"20171006\",\"descripti\n",
    "on\":\"Amino acid conservation scores calculated from MultiZ100Way alignments from UCSC.\",\"releaseDate\":\"\n",
    "2017-10-06\"},{\"name\":\"ClinVar\",\"version\":\"20200903\",\"description\":\"A freely accessible, public archive\n",
    "of reports of the relationships among human variations and phenotypes, with supporting evidence\",\"relea\n",
    "seDate\":\"2020-09-03\"},{\"name\":\"dbSNP\",\"version\":\"154\",\"description\":\"Identifiers for observed variants\"\n",
    ",\"releaseDate\":\"2020-05-01\"},{\"name\":\"dbSNP\",\"version\":\"151\",\"description\":\"Identifiers for observed va\n",
    "riants\",\"releaseDate\":\"2018-04-18\"},{\"name\":\"gnomAD\",\"version\":\"3.0\",\"description\":\"Allele frequencies\n",
    "from Genome Aggregation Database (gnomAD)\",\"releaseDate\":\"2019-10-16\"},{\"name\":\"MITOMAP\",\"version\":\"202\n",
    "00819\",\"description\":\"Small variants in the MITOMAP human mitochondrial genome database\",\"releaseDate\":\n",
    "\"2020-08-19\"},{\"name\":\"1000 Genomes Project\",\"version\":\"Phase 3 v3plus\",\"description\":\"A public catalog\n",
    "ue of human variation and genotype data\",\"releaseDate\":\"2013-05-27\"},{\"name\":\"PrimateAI\",\"version\":\"0.2\n",
    "\",\"description\":\"PrimateAI percentile scores.\",\"releaseDate\":\"2018-11-07\"},{\"name\":\"REVEL\",\"version\":\"2\n",
    "0200205\",\"description\":\"Pathogenicity scores of missense variants predicted by REVEL\",\"releaseDate\":\"20\n",
    "20-02-05\"},{\"name\":\"SpliceAI\",\"version\":\"1.3\",\"description\":\"SpliceAI scores, distances, etc.\",\"release\n",
    "Date\":\"2019-10-04\"},{\"name\":\"TOPMed\",\"version\":\"freeze_5\",\"description\":\"Allele frequencies from TOPMed\n",
    " data\",\"releaseDate\":\"2017-08-28\"},{\"name\":\"ClinGen\",\"version\":\"20160414\",\"releaseDate\":\"2016-04-14\"},{\n",
    "\"name\":\"ClinGen Dosage Sensitivity Map\",\"version\":\"20200913\",\"description\":\"Dosage sensitivity map from\n",
    " ClinGen (dbVar)\",\"releaseDate\":\"2020-09-13\"},{\"name\":\"MITOMAP_SV\",\"version\":\"20200819\",\"description\":\"\n",
    "Large structural variants in the MITOMAP human mitochondrial genome database\",\"releaseDate\":\"2020-08-19\n",
    "\"},{\"name\":\"1000 Genomes Project (SV)\",\"version\":\"Phase 3 v5a\",\"description\":\"A public catalogue of hum\n",
    "an variation and genotype data\",\"releaseDate\":\"2013-05-27\"},{\"name\":\"ClinGen disease validity curations\n",
    "\",\"version\":\"20200914\",\"description\":\"Disease validity curations from ClinGen (dbVar)\",\"releaseDate\":\"2\n",
    "020-09-14\"},{\"name\":\"gnomAD_gene_scores\",\"version\":\"2.1\",\"description\":\"Gene Loss of function scores fr\n",
    "om Genome Aggregation Database (gnomAD)\",\"releaseDate\":\"2018-10-17\"},{\"name\":\"OMIM\",\"version\":\"20200915\n",
    "\",\"description\":\"An Online Catalog of Human Genes and Genetic Disorders\",\"releaseDate\":\"2020-09-15\"},{\"\n",
    "name\":\"phyloP\",\"version\":\"hg38\",\"description\":\"20 way conservation score between humans and 19 other ve\n",
    "rtebrates\",\"releaseDate\":\"2015-04-17\"},{\"name\":\"gnomAD_LCR\",\"version\":\"2.1\",\"description\":\"Low complexi\n",
    "ty regions from gnomAD.\",\"releaseDate\":\"2019-04-10\"},{\"name\":\"MitochondrialHeteroplasmy\",\"version\":\"201\n",
    "80410\",\"description\":\"Variant read frequency percentiles for the Mitochondrial reference\",\"releaseDate\"\n",
    ":\"2020-05-21\"}],\"samples\":[\"Novaseq_TSPF450-NA12878-1-HFHWJDMXX_S1_L001\",\"Novaseq_TSPF450-NA12891-1-HFH\n",
    "WJDMXX_S3_L001\",\"Novaseq_TSPF450-NA12892-1-HFHWJDMXX_S2_L001\"]},\"positions\":[\n",
    "```\n",
    "Followed by the `positions` lines (two position lines in this example):\n",
    "\n",
    "```json\n",
    "{\"chromosome\":\"chr21\",\"position\":9975027,\"refAllele\":\"C\",\"altAlleles\":[\"G\"],\"quality\":102.47,\"filters\":\n",
    "[\"PASS\"],\"fisherStrandBias\":0.727,\"mappingQuality\":43.11,\"cytogeneticBand\":\"21p11.2\",\"samples\":[{\"genot\n",
    "ype\":\"0/1\",\"variantFrequencies\":[0.4474],\"totalDepth\":38,\"genotypeQuality\":32,\"alleleDepths\":[21,17]},{\n",
    "\"genotype\":\"0/1\",\"variantFrequencies\":[0.5147],\"totalDepth\":68,\"genotypeQuality\":36,\"alleleDepths\":[33,\n",
    "35]},{\"genotype\":\"0/0\",\"variantFrequencies\":[0.1667],\"totalDepth\":72,\"genotypeQuality\":0,\"alleleDepths\"\n",
    ":[60,12],\"failedFilter\":true}],\"variants\":[{\"vid\":\"21-9975027-C-G\",\"chromosome\":\"chr21\",\"begin\":9975027\n",
    ",\"end\":9975027,\"refAllele\":\"C\",\"altAllele\":\"G\",\"variantType\":\"SNV\",\"hgvsg\":\"NC_000021.9:g.9975027C>G\",\"\n",
    "phylopScore\":-1.7,\"dbsnp\":[\"rs75310125\"],\"gnomad\":{\"coverage\":102,\"failedFilter\":true,\"allAf\":0.464664,\n",
    "\"allAn\":125536,\"allAc\":58332,\"allHc\":34,\"afrAf\":0.396789,\"afrAn\":31886,\"afrAc\":12652,\"afrHc\":0,\"amrAf\":\n",
    "0.489333,\"amrAn\":12562,\"amrAc\":6147,\"amrHc\":9,\"easAf\":0.490489,\"easAn\":2944,\"easAc\":1444,\"easHc\":0,\"fin\n",
    "Af\":0.492007,\"finAn\":9758,\"finAc\":4801,\"finHc\":3,\"nfeAf\":0.486377,\"nfeAn\":59604,\"nfeAc\":28990,\"nfeHc\":1\n",
    "2,\"asjAf\":0.491335,\"asjAn\":3116,\"asjAc\":1531,\"asjHc\":4,\"sasAf\":0.494874,\"sasAn\":2926,\"sasAc\":1448,\"sasH\n",
    "c\":6,\"othAf\":0.477083,\"othAn\":1920,\"othAc\":916,\"othHc\":0,\"maleAf\":0.466374,\"maleAn\":61114,\"maleAc\":2850\n",
    "2,\"maleHc\":24,\"femaleAf\":0.463041,\"femaleAn\":64422,\"femaleAc\":29830,\"femaleHc\":10},\"topmed\":{\"allAf\":0.\n",
    "447447,\"allAn\":125568,\"allAc\":56185,\"allHc\":137,\"failedFilter\":true},\"transcripts\":[{\"transcript\":\"ENST\n",
    "00000625012.1\",\"source\":\"Ensembl\",\"bioType\":\"processed_transcript\",\"cdnaPos\":\"1475\",\"exons\":\"11/11\",\"ge\n",
    "neId\":\"ENSG00000270533\",\"hgnc\":\"CR382285.1\",\"consequence\":[\"non_coding_transcript_exon_variant\"],\"hgvsc\n",
    "\":\"ENST00000625012.1:n.1475G>C\",\"isCanonical\":true}]}]},\n",
    "{\"chromosome\":\"chr21\",\"position\":9975061,\"refAllele\":\"A\",\"altAlleles\":[\"G\"],\"quality\":190.73,\"filters\":\n",
    "[\"PASS\"],\"fisherStrandBias\":3.698,\"mappingQuality\":44.4,\"cytogeneticBand\":\"21p11.2\",\"samples\":[{\"genoty\n",
    "pe\":\"0/1\",\"variantFrequencies\":[0.5349],\"totalDepth\":43,\"genotypeQuality\":37,\"alleleDepths\":[20,23]},{\"\n",
    "genotype\":\"1/1\",\"variantFrequencies\":[0.7727],\"totalDepth\":66,\"genotypeQuality\":3,\"alleleDepths\":[15,51\n",
    "]},{\"genotype\":\"0/1\",\"variantFrequencies\":[0.5833],\"totalDepth\":72,\"genotypeQuality\":38,\"alleleDepths\":\n",
    "[30,42]}],\"variants\":[{\"vid\":\"21-9975061-A-G\",\"chromosome\":\"chr21\",\"begin\":9975061,\"end\":9975061,\"refAl\n",
    "lele\":\"A\",\"altAllele\":\"G\",\"variantType\":\"SNV\",\"hgvsg\":\"NC_000021.9:g.9975061A>G\",\"phylopScore\":-0.1,\"db\n",
    "snp\":[\"rs75428092\"],\"gnomad\":{\"coverage\":93,\"failedFilter\":true,\"allAf\":0.539865,\"allAn\":142030,\"allAc\"\n",
    ":76677,\"allHc\":5731,\"afrAf\":0.594262,\"afrAn\":41586,\"afrAc\":24713,\"afrHc\":3944,\"amrAf\":0.518277,\"amrAn\":\n",
    "13514,\"amrAc\":7004,\"amrHc\":254,\"easAf\":0.581298,\"easAn\":3112,\"easAc\":1809,\"easHc\":253,\"finAf\":0.517912,\n",
    "\"finAn\":10384,\"finAc\":5378,\"finHc\":188,\"nfeAf\":0.511456,\"nfeAn\":64074,\"nfeAc\":32771,\"nfeHc\":769,\"asjAf\"\n",
    ":0.524864,\"asjAn\":3298,\"asjAc\":1731,\"asjHc\":83,\"sasAf\":0.549736,\"sasAn\":3036,\"sasAc\":1669,\"sasHc\":151,\"\n",
    "othAf\":0.539869,\"othAn\":2132,\"othAc\":1151,\"othHc\":85,\"maleAf\":0.539736,\"maleAn\":68804,\"maleAc\":37136,\"m\n",
    "aleHc\":2762,\"femaleAf\":0.539986,\"femaleAn\":73226,\"femaleAc\":39541,\"femaleHc\":2969},\"topmed\":{\"allAf\":0.\n",
    "567087,\"allAn\":125568,\"allAc\":71208,\"allHc\":8558,\"failedFilter\":true},\"transcripts\":[{\"transcript\":\"ENS\n",
    "T00000625012.1\",\"source\":\"Ensembl\",\"bioType\":\"processed_transcript\",\"cdnaPos\":\"1441\",\"exons\":\"11/11\",\"g\n",
    "eneId\":\"ENSG00000270533\",\"hgnc\":\"CR382285.1\",\"consequence\":[\"non_coding_transcript_exon_variant\"],\"hgvs\n",
    "c\":\"ENST00000625012.1:n.1441T>C\",\"isCanonical\":true}]}]},\n",
    "```\n",
    "\n",
    "Finally, after the `positions` lines are the `genes` lines (3 lines for 2 genes in the example):\n",
    "\n",
    "```json\n",
    "],\"genes\":[\n",
    "{\"name\":\"ABCC13\",\"omim\":[{\"mimNumber\":608835,\"geneName\":\"ATP-binding cassette, subfamily C, member 13\",\n",
    "\"description\":\"ABCC13 belongs to a large family of ATP-binding cassette (ABC) transporters that play important roles as membrane transporters or ion channel modulators. However, ABCC13 is a truncated protein that lacks critical ATP-binding motifs and is unlikely to be a functional transporter (Yabuuchi et al., 2002).\"}]},\n",
    "{\"name\":\"USP25\",\"gnomAD\":{\"pLi\":7.52e-1,\"pRec\":2.48e-1,\"pNull\":1.85e-12,\"synZ\":2.56e-1,\"misZ\":1.24e0,\"loeuf\":3.30e-1},\"omim\":[{\"mimNumber\":604736,\"description\":\"Ubiquitin is a highly conserved 76-amino acid protein involved in regulation of intracellular protein breakdown, cell cycle regulation, and stress response. Ubiquitin is released from degraded proteins by disassembly of the polyubiquitin chains, which is mediated by ubiquitin-specific proteases (USPs), such as USP25 (Valero et al., 1999).\"}]}\n",
    "]}\n",
    "```\n",
    "\n",
    "Therefore, parsing the file requires some trimming and handling for each line to make them correctly formatted JSON objects or arrays."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
