---
title: Getting Started
---

Nirvana is written in C# using [.NET Core](https://www.microsoft.com/net/download/core) (an amazing runtime environment that currently runs on Windows, Linux, Mac OS X, and in Docker images). Once .NET Core has been downloaded, all you need to do is grab the source, compile it, and grab the data files.

:::tip


Nirvana currently uses .NET Core 2.1 or later. Please make sure that you have the most current runtime from the [.NET Core downloads](https://www.microsoft.com/net/download/core) page.

:::

## Quick Start

If you want to get started right away, we've created a script that downloads Nirvana, compiles it, and starts annotating a test file:

```bash
curl -O https://illumina.github.io/NirvanaDocumentation/scripts/TestNirvana.sh
sh ./TestNirvana.sh
```

We have verified that this script works on Windows (using Git Bash or WSL), Linux, and Mac OS X.

## Compiling Nirvana

The following will grab the latest version of Nirvana from GitHub and compile it using the .NET Core compiler:

```Bash
git clone https://github.com/Illumina/Nirvana.git
cd Nirvana
dotnet build -c Release
```

## Downloading the data files

To download the latest data sources (or update the ones that you already have), use the following command to automate the download from S3:

```Bash
dotnet bin/Release/netcoreapp2.1/Downloader.dll \
     --ga GRCh37 \
     -o Data
```

* the `--ga` argument specifies the genome assembly which can be `GRCh37`, `GRCh38`, or `both`.
* the `-o` argument specifies the output directory

## Download a test VCF file

Here's a toy VCF file you can play around with:

```Bash
curl -O https://illumina.github.io/NirvanaDocumentation/VCF/HiSeq.10000.vcf
```

## Running Nirvana

Once you have downloaded the data sets, use the following command to annotate your VCF:

```Bash
dotnet bin/Release/netcoreapp2.1/Nirvana.dll \
     -c Data/Cache/GRCh37/Both \
     --sd Data/SupplementaryAnnotation/GRCh37 \
     -r Data/References/Homo_sapiens.GRCh37.Nirvana.dat \
     -i HiSeq.10000.vcf \
     -o HiSeq.10000.annotated
```

* the `-c` argument specifies the cache prefix
* the `--sd` argument specifies the supplementary annotation directory
* the `-r` argument specifies the compressed reference path
* the `-i` argument specifies the input VCF path
* the `-o` argument specifies the output filename prefix

When running Nirvana, performance metrics are shown as it evaluates each chromosome in the input VCF file:

![](NirvanaOutput.png)
