---
title: Nirvana JSON File Format
---

import ClinVar from '../data-sources/clinvar-json.md'

## Overview

### Conventions
In the Nirvana JSON representation, we try to maximize the amount of useful information that is relayed in the output file. As such, we have several conventions that are useful to know about:

- With boolean key/value pairs, we only output the keys that have a true value. I.e. there's no reason to display `"isStructuralVariant":false` a few million times when annotating a small variant VCF.
- When transferring data from the VCF file to the JSON (e.g. for allele depths (AD)), it is common to use a period (.) as a placeholder for missing data in the VCF file. Nirvana treats periods like empty or null strings and therefore will not output those entries.

### JSON Layout

The JSON file produced by Nirvana divided into three main sections:

![JSON Layout](/img/JSON-Layout.svg)

| Section    | Contents                                              |
|:-----------|:------------------------------------------------------|
| header     | configuration, data source versions, and sample names | 
| positions  | variant-level annotation                              | 
| genes      | gene-level annotation                                 | 

In general, each position corresponds to a row in the original VCF file.

For each gene that was referenced in the transcripts found in the positions section, there will be additional gene-level annotation in the gene section.

## Header
```json
{ 
   "header":{ 
      "annotator":"Nirvana 3.0.0-alpha.5+g6c52e247",
      "creationTime":"2017-06-14 15:53:13",
      "genomeAssembly":"GRCh37",
      "dataSources":[ 
         { 
            "name":"OMIM",
            "version":"unknown",
            "description":"An Online Catalog of Human Genes and Genetic Disorders",
            "releaseDate":"2017-05-03"
         },
         { 
            "name":"VEP",
            "version":"84",
            "description":"BothRefSeqAndEnsembl",
            "releaseDate":"2017-01-16"
         },
         { 
            "name":"ClinVar",
            "version":"20170503",
            "description":"A freely accessible, public archive of reports of the relationships among human variations and phenotypes, with supporting evidence",
            "releaseDate":"2017-05-03"
         },
         { 
            "name":"phyloP",
            "version":"hg19",
            "description":"46 way conservation score between humans and 45 other vertebrates",
            "releaseDate":"2009-11-10"
         }
      ],
      "samples":[ 
         "NA12878",
         "NA12891",
         "NA12892"
      ]
   },
```

| Field          | Type         | Required | Notes |
|:---------------|:------------:|:--------:|:------|
| annotator      | string       | Yes      | the name of the annotator and the current version                |
| creationTime   | string       | Yes      | yyyy-MM-dd hh:mm:ss |
| genomeAssembly | string       | Yes      | see possible values below |
| schemaVersion  | integer      | Yes      | Incremented whenever the core structure of the JSON file introduces breaking changes |
| dataVersion    | string       | Yes      | |
| dataSources    | object array | Yes      | |
| - name         | string       | Yes      | |
| - version      | string       | Yes      | |
| - description  | string       | No       | Optional description of the data source. e.g. Ensembl/RefSeq when describing the transcript source |
| - releaseDate  | string       | No       | yyyy-MM-dd |
| samples        | string array | No       | the order of these sample names will be used throughout the JSON file when enumerating samples |

**genomeAssembly:**
- GRCh37
- GRCh38
- hg19
- SARSCoV2

## Positions
```json
"positions":[ 
   { 
      "chromosome":"chr2",
      "position":48010488,
      "repeatUnit":"GGCCCC",
      "refRepeatCount":3,
      "svEnd":48020488,
      "refAllele":"G",
      "altAlleles":[ 
         "A",
         "GT"
      ],
      "quality":461,
      "filters":[ 
         "PASS"
      ],
      "ciPos":[ 
         -170,
         170
      ],
      "ciEnd":[ 
         -175,
         175
      ],
      "svLength":1000,
      "strandBias":1.23,
      "jointSomaticNormalQuality":29,
      "cytogeneticBand":"2p16.3",
```

| Field                     | Type          | Required | Variant Type  | Notes |
|:--------------------------|:-------------:|:--------:|:-------------:|:------|
| chromosome                | string        | Yes      | all           | exactly as displayed in the vcf |
| postion                   | integer       | Yes      | all           | exactly as displayed in the vcf (1-based notation). Range: 1 - 250 million |
| repeatUnit                | string        | No       | STR           | provided by ExpansionHunter |
| refRepeatCount            | integer       | No       | STR           | provided by ExpansionHunter |
| svEnd                     | integer       | No       | SV            | |
| refAllele                 | string        | Yes      | all           | exactly as displayed in the vcf|
| altAllele                 | string array  | Yes      | all           | exactly as displayed in the vcf|
| quality                   | float         | Yes      | all           | exactly as displayed in the vcf (Normally an integer, but some variant callers using floating point. Has been observed as high as 500k)|
| filters                   | string array  | No       | all           | exactly as displayed in the vcf |
| ciPos                     | integer array | No       | SV            | |
| ciEnd                     | integer array | No       | SV            | |
| svLength                  | integer       | No       | SV            | |
| strandBias                | float         | No       | small variant | provided by GATK (from SB) |
| jointSomaticNormalQuality | integer       | No       | SV            | provided by the Manta variant caller (SOMATICSCORE) |
| cytogeneticBand           | string        | No       | all           | e.g. 17p13.1 |

## Samples
```json
"samples":[
   {
      "genotype":"0/1",
      "variantFrequencies":[
         0.333,
         0.5
      ],
      "totalDepth":57,
      "genotypeQuality":12,
      "copyNumber":3,
      "repeatUnitCounts":[
         10,
         20
      ],
      "alleleDepths":[
         10,
         20,
         30
      ],
      "failedFilter":true,
      "splitReadCounts":[
         10,
         20
      ],
      "pairedEndReadCounts":[
         10,
         20
      ],
      "isDeNovo":true,
      "diseaseAffectedStatuses":[
         "-"
      ],
      "artifactAdjustedQualityScore":89.3,
      "likelihoodRatioQualityScore":78.2,
      "heteroplasmyPercentile":[
         23.13,
         12.65
      ]
   }
]
```

| Field                        | Type          | Required | Notes |
|:-----------------------------|:-------------:|:--------:|:------|
| genotype                     | string        | Yes      | |
| variantFrequencies           | float array   | Yes      | Range: 0 - 1.0. One value per alternate allele |
| totalDepth                   | integer       | Yes      | non-negative integer values |
| genotypeQuality              | integer       | Yes      | non-negative integer values. Typically maxes out at 99 |
| copyNumber                   | integer       | Yes      | non-negative integer values |
| repeatUnitCounts             | integer array | Yes      | ExpansionHunter-specific |
| alleleDepths                 | integer array | Yes      | non-negative integer values |
| failedFilter                 | bool          | Yes      | |
| splitReadCounts              | integer array | No       | Manta-specific |
| pairedEndReadCounts          | integer array | No       | Manta-specific |
| isDeNovo                     | bool          | No       | |
| diseaseAffectedStatuses      | string array  | No       | ExpansionHunter-specific |
| artifactAdjustedQualityScore | float         | No       | PEPE-specific. Range: 0 - 100.0 |
| likelihoodRatioQualityScore  | float         | No       | PEPE-specific. Range: 0 - 100.0 |
| heteroplasmyPercentile       | float         | No       | Range: 0 - 100. 2 decimal places. One value per alternate allele |

**Empty samples**

If a sample does not contain any entries, we will create a sample object that contains the `isEmpty` key. This ensures that sample ordering is preserved while indicating that a sample is intentionally empty.

```json
"samples":[ 
   { 
      "isEmpty":true
   }
],
```

## Variants
```json
"variants":[ 
   { 
      "vid":"2:48010488:A",
      "chromosome":"chr2",
      "begin":48010488,
      "end":48010488,
      "isReferenceMinorAllele":true,
      "isStructuralVariant":true,
      "refAllele":"G",
      "altAllele":"A",
      "variantType":"SNV",
      "isDecomposedVariant":true,
      "isRecomposedVariant":true,
      "linkedVids":["2:48010488:GTA:ATC"],
      "hgvsg":"NC_000002.11:g.48010488G>A",
      "phylopScore":0.459
```
### Regulatory Regions
```json
"regulatoryRegions":[ 
   { 
      "id":"ENSR00001542175",
      "type":"promoter",
      "consequence":[ 
         "regulatory_region_variant"
      ]
   }
]
```

### ClinVar
<ClinVar />

### Transcripts
```json
"transcripts":[ 
   { 
      "transcript":"ENST00000445503.1",
      "source":"Ensembl",
      "bioType":"nonsense_mediated_decay",
      "codons":"gGg/gAg",
      "aminoAcids":"G/E",
      "cdnaPos":"268",
      "cdsPos":"116",
      "exons":"1/9",
      "introns":"1/8",
      "proteinPos":"39",
      "geneId":"ENSG00000116062",
      "hgnc":"MSH6",
      "consequence":[ 
         "missense_variant",
         "NMD_transcript_variant"
      ],
      "hgvsc":"ENST00000445503.1:c.116G>A",
      "hgvsp":"ENSP00000405294.1:p.(Gly39Glu)",
      "geneFusion":{ 
         "exon":6,
         "intron":5,
         "fusions":[ 
            { 
               "hgvsc":"ETV6{ENST00000396373.4}:c.1_1009+3402_RUNX1{ENST00000437180.1}:c.58+568_1443",
               "exon":3,
               "intron":2
            },
            { 
               "hgvsc":"ETV6{ENST00000396373.4}:c.1_1009+3402_RUNX1{ENST00000300305.3}:c.58+568_1443",
               "exon":2,
               "intron":1
            }
         ]
      },
      "isCanonical":true,
      "polyPhenScore":0.95,
      "polyPhenPrediction":"probably damaging",
      "proteinId":"ENSP00000405294.1",
      "siftScore":0.61,
      "siftPrediction":"tolerated",
      "completeOverlap":true
   }
]
```

## Genes
```json
"genes":[ 
   { 
      "name":"MSH6",
      "hgncId":7329,
      "summary":"This gene encodes a member of the DNA mismatch repair MutS family. In E. coli, the MutS protein helps in the recognition of mismatched nucleotides prior to their repair. A highly conserved region of approximately 150 aa, called the Walker-A adenine nucleotide binding motif, exists in MutS homologs. The encoded protein heterodimerizes with MSH2 to form a mismatch recognition complex that functions as a bidirectional molecular switch that exchanges ADP and ATP as DNA mismatches are bound and dissociated. Mutations in this gene may be associated with hereditary nonpolyposis colon cancer, colorectal cancer, and endometrial cancer. Transcripts variants encoding different isoforms have been described. [provided by RefSeq, Jul 2013]",
      "acmgActionableGene":true,
      /* this is where gene-level data sources can be found */
      /* e.g. OMIM and ExAC                                 */
   }
]
```
